<div class="WordSection1" style="page: WordSection1;" id="contentDivNode"><div><div class="front" data-edited="true"><p class="jrnlDOI">10.7554/eLife.15545</p><p class="jrnlArtType">Research Article</p><h1 class="jrnlArtTitle">Dual functions of a small regulatory subunit in the mitochondrial calcium uniporter complex</h1><div data-editable="true" construct="true" wrapelement="div" class="jrnlAbsGroup"><h1 class="jrnlAbsHead">Abstract</h1><p class="jrnlAbsPara">Mitochondrial Ca<sup>2+</sup> uptake, a process crucial for bioenergetics and Ca<sup>2+</sup> signaling, is catalyzed by the mitochondrial calcium uniporter. The uniporter is a multi-subunit Ca<sup>2+</sup>-activated Ca<sup>2+</sup> channel, with the Ca<sup>2+</sup> pore formed by the MCU protein and Ca<sup>2+</sup>-dependent activation mediated by MICU subunits. Recently, a mitochondrial inner membrane protein EMRE was identified as a uniporter subunit absolutely required for Ca<sup>2+</sup> permeation. However, the molecular mechanism and regulatory purpose of EMRE remain largely unexplored. Here, we determine the transmembrane orientation of EMRE, and show that its known MCU-activating function is mediated by the interaction of transmembrane helices from both proteins. We also reveal a second function of EMRE: to maintain tight MICU regulation of the MCU pore, a role that requires EMRE to bind MICU1 using its conserved C-terminal polyaspartate tail. This dual functionality of EMRE ensures that all transport-competent uniporters are tightly regulated, responding appropriately to a dynamic intracellular Ca<sup>2+</sup> landscape.</p></div><div class="jrnlBoxBlock" box-type="elife-digest" data-editable="true"><h1 class="jrnlBoxCaption">eLife digest</h1><p class="jrnlBoxText">Like all power plants, mitochondria &ndash; the compartments inside our cells that supply energy &ndash; must adjust their energy output to match fluctuations in demand. Inside cells, the levels of calcium ions in the cytoplasm often signal such demands. Mitochondria therefore control their calcium ion levels with tightly regulated, membrane-embedded proteins that move calcium ions into and out of the mitochondria. One of these membrane machines, the mitochondrial calcium uniporter (MCU) complex, is a "smart channel" that admits calcium ions into the mitochondria only when their cytoplasmic levels exceed a threshold.</p><p class="jrnlBoxText">The MCU complex contains four essential proteins: MCU, which forms the pore through which the calcium ions enter the mitochondrion; MICU1 and MICU2, which act as &ldquo;gatekeepers&rdquo;, opening the pore only when the cell contains high levels of calcium ions; and EMRE, a small, mysterious protein. Why is EMRE required for the channel's operation, and how does it fit into the four-protein complex?</p><p class="jrnlBoxText">By comparing EMRE proteins from different species, constructing mutant forms of EMRE, and recording calcium ion transport in mitochondria from cultured human cells, Tsai, Phillips et al. show that EMRE has two key roles. First, it snuggles up against the MCU protein and forms an essential part of the calcium ion-selective pore. Second, it acts as molecular glue to fix the calcium ion-sensing MICU gatekeepers to the pore. These two linked functions ensure that the MCU complex switches on only when the cell contains high levels of calcium ions, preventing the cell becoming catastrophically overloaded with calcium ions and cell death.</p><p class="jrnlBoxText">Challenges for the future are to purify the MCU complex and reconstitute its ability to transport calcium ions from its component parts. This will help to determine the structure of the channel.</p></div></div><div class="body" data-edited="true"><h1 class="jrnlHead1" bold="true" patternhead="true">Introduction</h1><p class="jrnlSecPara">Ca<sup>2+</sup> regulation of key mitochondrial processes such as ATP production and initiation of apoptosis is controlled by precise balance of Ca<sup>2+</sup> influx and efflux across the mitochondrial inner membrane (<span class="jrnlBibRef" data-citation-string=" R12 ">Gunter et al., 2000</span>; <span class="jrnlBibRef" data-citation-string=" R28 ">Rizzuto et al., 2012</span>). Studies in the 1960s and '70s established that mitochondria from most eukaryotes, except for certain yeast species, can take up large quantities of Ca<sup>2+</sup> from the cytosol into the matrix through a mechanism that is membrane potential dependent and strongly inhibited by ruthenium compounds such as Ru360 (<span class="jrnlBibRef" data-citation-string=" R4 ">Carafoli and Lehninger, 1971</span>; <span class="jrnlBibRef" data-citation-string=" R8 ">Deluca and Engstrom, 1961</span>; <span class="jrnlBibRef" data-citation-string=" R35 ">Ying et al., 1991</span>). A few years ago, the field witnessed a groundbreaking achievement &mdash; identification of the <i>MCU</i> gene (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R6 ">De Stefani et al., 2011</span>). The 35-kDa MCU protein oliogomerizes with unknown stoichiometry to form a Ca<sup>2+</sup>-selective pore (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>). MCU possesses two transmembrane helices (TMHs) connected by a short loop that hosts a signature sequence (DIME) thought to contribute to a Ca<sup>2+</sup>-selective permeation site. The N- and C-terminal regions of MCU are exposed to the mitochondrial matrix, each with a coiled-coil sequence of unknown function.</p><p class="jrnlSecPara">It was subsequently found that MCU forms a complex with the mitochondrial Ca<sup>2+</sup> uptake protein 1 (MICU1), which has co-evolved with MCU since early eukaryotic evolution (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R3 ">Bick et al., 2012</span>). In humans, MICU1 has two additional homologues, MICU2 and the neuron-specific MICU3 (<span class="jrnlBibRef" data-citation-string=" R26 ">Plovanich et al., 2013</span>). The MICUs serve as the Ca<sup>2+</sup>-sensing gate that confers Ca<sup>2+</sup>-dependence to opening of the Ca<sup>2+</sup>-selective pore (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R23 ">Mallilankaraman et al., 2012</span>). In resting cellular conditions, where cytoplasmic Ca<sup>2+</sup> is low, MICUs shut the pore to prevent excessive Ca<sup>2+</sup> influx into the matrix, a dangerous process that could diminish inner membrane potential and trigger apoptotic cell death. Transient elevation of Ca<sup>2+</sup> to the low &mu;M range, detected by EF-hands in MICUs, releases this inhibition to open the channel (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R16 ">Kamer and Mootha, 2014</span>). To avoid confusion on nomenclature, we henceforth refer to the Ru-360 sensitive mitochondrial Ca<sup>2+</sup> channel complex as the 'uniporter complex,' a molecular assembly of the pore-forming MCU protein along with associated regulatory subunits.</p><p class="jrnlSecPara">Recently, using quantitative mass spectroscopy, Mootha and colleagues discovered yet another component of the uniporter complex: the essential MCU regulator (EMRE), a small (~10&thinsp;kDa) inner membrane protein found only in metazoa (<span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>). EMRE possesses a single TMH and a highly conserved C-terminal polyaspartate tail, typically composed of one glutamate followed by 5&ndash;7 aspartates. In humans, MCU-EMRE interaction is absolutely required for Ca<sup>2+</sup> permeation via MCU (<span class="jrnlBibRef" data-citation-string=" R20 ">Kovacs-Bogdan et al., 2014</span>; <span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>). However, an MCU homologue in <i>D. discoideum, </i>a species belonging to the EMRE-lacking Amoebazoa group in protists, is fully capable of conducting Ca<sup>2+ </sup>(<span class="jrnlBibRef" data-citation-string=" R20 ">Kovacs-Bogdan et al., 2014</span>). The question naturally arises: what might be the physiological importance for MCU to become strictly dependent on EMRE for function in humans? What would be the consequence if human MCU could transport Ca<sup>2+</sup> without EMRE?</p><p class="jrnlSecPara">We address these questions by mounting an extensive investigation of EMRE. We first sought to determine the protein's transmembrane topology, a problem that cannot be definitively resolved by standard protease digestion assays (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R32 ">Vais et al., 2016</span>) due to the small size of the protein&rsquo;s extra-membrane regions. Two alternative strategies &ndash; directed mass-tagging and MCU-EMRE fusion construction - establish that EMRE exposes its N-terminal region to the matrix and C-terminus to the intermembrane space (IMS). Mutagenesis screening and domain-interaction analysis further demonstrate that EMRE supports Ca<sup>2+</sup> transport by using its TMH to bind to MCU through its first TMH (TMH1). Moreover, EMRE also interacts with MICU1 via its C-terminal polyaspartate tail, a molecular contact that turns out to be crucial to retain MICUs in the uniporter complex to gate the MCU pore. These results lead to a molecular model wherein the dual 'MCU-activating' and 'MICU-retaining' functionalities of EMRE together play a crucial role in orchestrating uniporter responses to intracellular Ca<sup>2+</sup> signaling.</p><h1 class="jrnlHead1" bold="true" patternhead="true">Results</h1><h2 class="jrnlHead2" italic="true" headweightage="19">Functional dependence of MCU on EMRE in various species</h2><p class="jrnlSecPara">To study uniporter subunits without interference from native mitochondrial proteins, we employed CRISPR/Cas9 to produce MCU-knockout (KO), EMRE-KO, or MCU/EMRE double KO (ME-KO) HEK 293 cell lines. A standard Ca<sup>2+</sup> flux assay was used to evaluate uniporter activity. In a typical experiment, HEK cells were permeabilized with digitonin in the presence of a Ca<sup>2+</sup>-sensing fluorophore (CG-5N) and then treated with 10&thinsp;&mu;M extracellular Ca<sup>2+</sup> (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1A</span>). In WT cells, Ca<sup>2+</sup> is rapidly sequestered by mitochondria, and Ru360, a potent MCU inhibitor, arrests the uptake immediately (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1A</span>). (Henceforth for clearer data presentation, only the response of permeabilized cells to Ca<sup>2+</sup> will be presented, as in the red box in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1A</span>, with arrowheads indicating Ru360 addition.) Consistent with previous reports (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R6 ">De Stefani et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>), EMRE-, MCU-, or ME-KO mitochondria are completely devoid of uniporter activity, a deficiency rescued by supplying the deleted genes (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1B</span>).</p><div class="jrnlFigBlock" id="BLK_F1" data-stream-name="a_F1" data-page-num="3" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F1"><span class="label">Figure 1.</span> Functional analysis of uniporter in various species.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) A representative fluorescence-based Ca<sup fontsize="11.0pt">2+</sup> flux experiment. (<span class="partLabel">B</span>) Characterization of ME-KO HEK 293 cells. <i fontsize="11.0pt">Left</i>: western analysis comparing EMRE, MCU, and actin expression in WT or ME-KO cells. <i fontsize="11.0pt">Right</i>: Loss of MCU-mediated Ca<sup fontsize="11.0pt">2+</sup> uptake in ME-KO cells, and rescue by delivering both MCU and EMRE genes. (<span class="partLabel">C</span>) Activity of uniporters in species indicated. Ca<sup fontsize="11.0pt">2+</sup> flux experiments were performed using ME-KO cells expressing MCU alone or MCU and EMRE from the same species.</p><img src="/ecs/article/82842/resources/elife15545f001.jpg" data-mce-src="/ecs/article/82842/resources/elife15545f001.jpg" data-primary-extension="tif" data-width="1421" data-height="1173" data-id="1985832" alt="ecs/article/82842/resources/elife15545f001.jpg" height="677" width="819"/></div><p class="jrnlSecPara">Genome sequences imply that MCU and MICU proteins are ancient in eukaryotic evolution, while EMRE sequences appear only among metazoa. Indeed, MCU homologues from an amoeba, <i>D. discoideum,</i> and a green plant, <i>A. thaliana</i>, both of which lack EMRE, can alone mediate rapid, Ru360-sensitive mitochondrial Ca<sup>2+</sup> uptake in ME-KO cells (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>). In contrast, metazoan MCU homologues from <i>C. elegans</i> and <i>D. melanogaster</i> require EMRE to transport Ca<sup>2+</sup> (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>), as in humans. This striking difference raises questions regarding the biological purpose of EMRE in animals and offers opportunities for attacking questions of molecular mechanism.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Transmembrane orientation of EMRE</h2><p class="jrnlSecPara">To approach the physiological importance of EMRE&rsquo;s regulatory function, we first ask how it assembles with other uniporter subunits into a channel complex. The orientation of EMRE in the inner membrane was determined by a cysteine-modification, mass-tagging method. In a typical experiment, mitoplasts (outside-out submitochondrial vesicles lacking the outer membrane) prepared from HEK 293 cells are incubated with a 5-kDa, membrane impermeant thiol-reactive reagent, polyethylene glycol maleimide (PEGM). If the protein of interest has a cysteine exposed to the external solution, PEGM would react with this cysteine and thus increase the protein&rsquo;s molecular weight.</p><p class="jrnlSecPara">We first validated the assay on the known orientation of MCU (<span class="jrnlBibRef" data-citation-string=" R17 ">Kamer and Mootha, 2015</span>; <span class="jrnlBibRef" data-citation-string=" R24 ">Murgia and Rizzuto, 2015</span>). Human MCU possesses five cysteines, all in the N-terminal domain. If this region faces outward towards the IMS, PEGM would increase MCU&rsquo;s mass by 5&thinsp;kDa per residue modified. Experiments (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2A</span>), however, show that MCU mobility on SDS-PAGE is not altered by PEGM treatment unless the mitoplast membrane is first disrupted by the mild detergent dodecyl maltoside (DDM). The results thus confirm MCU&rsquo;s N<sub>in</sub>-C<sub>in</sub> orientation, with both the N- and C-termini residing in the matrix.</p><div class="jrnlFigBlock" id="BLK_F2" data-stream-name="a_F2" data-page-num="4" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F2"><span class="label">Figure 2.</span> Transmembrane orientation of MCU and EMRE.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Western blot analysis of WT-MCU response to PEGM, in the absence or presence of DDM detergent, with molecular weight marker positions indicated on left. (<span class="partLabel">B</span>) PEGM treatment of WT, I49C, or I97C EMRE. EMRE&rsquo;s molecular weight is ~10 kDa. (<span class="partLabel">C</span>) Cartoon illustrating the proposed membrane orientation of MCU and EMRE. The N-terminus of EMRE is fused to the C-terminus of MCU (dashed line). Blue circles: native cysteines. Green boxes: soluble region. (<span class="partLabel">D</span>) WT-MCU or MCU-EMRE fusion protein (Fus) probed with anti-MCU (left) or anti-EMRE (right) antibodies. (<span class="partLabel">E</span>) MCU-EMRE fusion treated with PEGM. In the presence of DDM, PEGM treatment produces 4 bands, representing fusion proteins with various numbers of cysteines modified. (<span class="partLabel">F</span>) Mitochondrial Ca<sup fontsize="11.0pt">2+</sup> uptake in ME-KO cells with (red) or without (black) expression of the MCU-EMRE fusion protein. See also <span class="jrnlFigRef" data-citation-string=" F2-S1 ">Figure 2&mdash;figure supplement 1</span>.</p><img src="/ecs/article/82842/resources/elife15545f002.jpg" data-mce-src="/ecs/article/82842/resources/elife15545f002.jpg" data-primary-extension="tif" data-width="1799" data-height="1319" data-id="1985992" alt="ecs/article/82842/resources/elife15545f002.jpg" height="601" width="819"/></div><div class="jrnlFigBlock" id="BLK_F2-S1" data-stream-name="a_F2-S1"><p class="jrnlFigCaption" id="F2-S1"><span class="label">Figure 2&mdash;figure supplement 1.</span> Uniporter function supported by indicated EMRE mutants.</p><img src="/ecs/article/82842/resources/elife15545fs001.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs001.jpg" data-primary-extension="tif" data-width="1373" data-height="441" data-id="1985892" alt="ecs/article/82842/resources/elife15545fs001.jpg" height="263" width="819"/></div><p class="jrnlSecPara">To determine EMRE topology, we constructed mutants with unique cysteines (I49C or I97C) engineered on either side of the TMH of the naturally cysteineless EMRE. Like WT, these mutants support MCU-dependent Ca<sup>2+</sup> flux in EMRE-KO cells (<span class="jrnlFigRef" data-citation-string=" F2-S1 ">Figure 2&mdash;figure supplement 1</span>). In intact mitoplasts free of native EMRE protein, PEGM readily labels I97C-EMRE but fails to react with I49C or the cysteineless WT protein, while after detergent pretreatment both mutants are labeled (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B</span>). EMRE thus adopts a N<sub>in</sub>-C<sub>out</sub> orientation, with its C-terminal tail facing the IMS.</p><p class="jrnlSecPara"><span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2C</span> summarizes the inner-membrane topology of MCU and EMRE inferred here. This was further verified by fusing EMRE onto the C-terminus of MCU, thus forcing the orientation of the two proteins in tandem to conform to the above molecular picture (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2C</span>). The fusion construct was tested in ME-KO cells, where it was expressed as a full-length protein detectable by both MCU and EMRE antibodies (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2D</span>). As with WT MCU, PEGM fails to modify any of the five native cysteines in the N-terminus without detergent pretreatment (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2E</span>), thus implying that the fusion protein is inserted homogeneously into the inner membrane in a proper N<sub>in</sub>-C<sub>out</sub> orientation. Moreover, the MCU-EMRE fusion protein mediates robust, Ru360-sensitive mitochondrial Ca<sup>2+</sup> uptake (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2F</span>), a powerful result corroborating the transmembrane orientation cartooned in <span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2C</span>.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Mapping the EMRE transmembrane helix</h2><p class="jrnlSecPara">We next investigate how EMRE interacts with MCU to support Ca<sup>2+</sup> permeation. This issue is addressed using three EMRE variants (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3A</span>), with N- or C-termini largely deleted by replacing it with a foreign 'C8' epitope (PRGPDRPEGIEE) (<span class="jrnlBibRef" data-citation-string=" R1 ">Abacioglu et al., 1994</span>) into either region, or with the TMH substituted by an artificial transmembrane 'WALP' helix (GWWLALALALALALALWWA) (<span class="jrnlBibRef" data-citation-string=" R18 ">Killian et al., 1996</span>). These mutants, named &Delta;N-, &Delta;C-, or WALP-EMRE, are all properly targeted to EMRE-KO mitochondria (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B</span>). Both &Delta;N- and &Delta;C-EMRE fully support uniporter function, but cells expressing WALP-EMRE exhibit no uniporter activity (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3C</span>). The results are surprising, as the strict conservation of EMRE&rsquo;s C-terminal polyaspartate tail implies that it should carry out some sort of crucial function.</p><div class="jrnlFigBlock" id="BLK_F3" data-stream-name="a_F3" data-page-num="5" data-float-placement="BottomMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F3"><span class="label">Figure 3.</span> MCU-interacting residues in EMRE.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) EMRE constructs with indicated regions substituted by either the C8 epitope or the WALP helix. MTS: mitochondrial targeting sequence. Green boxes: soluble regions. Pink ovals: polyaspartate tail. (<span class="partLabel">B</span>) The presence of these mutants in whole cell lysate (W) or isolated mitochondria (M). &Delta;C-EMRE is not detectable by the EMRE antibody because the C-terminal truncation removes the epitope. (<span class="partLabel">C</span>) Mitochondrial Ca<sup fontsize="11.0pt">2+</sup> uptake in EMRE-KO cells expressing WALP-, &Delta;N-, or &Delta;C-EMRE. (<span class="partLabel">D</span>) Diagram summarizing Trp scan of the EMRE TM helix. Red shows positions where Trp substitution reduces the rate of Ca<sup fontsize="11.0pt">2+</sup> uptake by &gt;50%. (<span class="partLabel">E</span>) Sequence alignment of EMRE TM helix. Yellow indicates residues intolerant to Trp substitutions in human EMRE. (<span class="partLabel">F</span>) Co-IP experiments using 1D4-tagged MCU immobilized in 1D4 affinity columns to pull down indicated EMRE mutants. IP: elution, analyzed using indicated antibody. CL: whole cell lysate input. Upper panel: proteins being expressed in ME-KO cells. Leftmost lane: MCU-free control to rule out non-specific binding of EMRE in the 1D4 column. See also <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1</span>.</p><img src="/ecs/article/82842/resources/elife15545f003.jpg" data-mce-src="/ecs/article/82842/resources/elife15545f003.jpg" data-primary-extension="tif" data-width="1833" data-height="1065" data-id="1985852" alt="ecs/article/82842/resources/elife15545f003.jpg" height="476" width="819"/></div><div class="jrnlFigBlock" id="BLK_F3-S1" data-stream-name="a_F3-S1"><p class="jrnlFigCaption" id="F3-S1"><span class="label">Figure 3&mdash;figure supplement 1.</span> Functional impact of mutations in EMRE&rsquo;s TM helix.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Expression of indicated EMRE mutants. (<span class="partLabel">B</span>) Ca<sup fontsize="11.0pt">2+</sup> flux traces in EMRE-KO cells expressing EMRE mutants. (<span class="partLabel">C</span>) Bar chart summarizing Ca<sup fontsize="11.0pt">2+</sup> uptake activity in cells expressing various EMRE mutants. <i fontsize="11.0pt">Dashed red line</i>: 50% WT activity.</p><img src="/ecs/article/82842/resources/elife15545fs002.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs002.jpg" data-primary-extension="tif" data-width="1852" data-height="1119" data-id="1986082" alt="ecs/article/82842/resources/elife15545fs002.jpg" height="496" width="819"/></div><p class="jrnlSecPara">To locate EMRE residues critical for MCU-mediated Ca<sup>2+</sup> transport, we performed tryptophan scanning mutagenesis to cover the entire transmembrane region (S64 to A92). This classical strategy posits that the large tryptophan side chain introduced at a protein interface is more likely to disrupt helical packing and hence function than when projecting towards lipid (<span class="jrnlBibRef" data-citation-string=" R14 ">Hong and Miller, 2000</span>; <span class="jrnlBibRef" data-citation-string=" R31 ">Sharp et al., 1995</span>). The results (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3D</span>, <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1</span>) highlight a rather clean segregation of Trp-sensitive vs insensitive positions on a helical-wheel diagram. In particular, Trp substitution at G81 or S85 completely eliminates Ca<sup>2+</sup> uptake via the uniporter complex. These two residues belong to a conserved Gxxx[G/A/S] motif (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3E</span>), frequently found to mediate packing of TMHs in membrane protein structures (<span class="jrnlBibRef" data-citation-string=" R29 ">Russ and Engelman, 2000</span>).</p><p class="jrnlSecPara">Co-immunoprecipitation (co-IP) experiments further confirm the GxxxS sequence as crucial to MCU-EMRE complex formation. EMRE variants were co-expressed with MCU carrying a C-terminal '1D4' epitope (TETSQVAPA) (<span class="jrnlBibRef" data-citation-string=" R22 ">MacKenzie et al., 1984</span>) in ME-KO cells, and the MCU-EMRE complex was immobilized on a 1D4 affinity column for downstream analysis. WT EMRE is captured by MCU, but disruption of the GxxxS region by the G81W or S85W mutation prevents this association, while Trp substitution of residues elsewhere on the TMH does not (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3F</span>). These results taken together show that EMRE binds to MCU using a Gxxx[G/A/S] motif in the C-terminal half of its TMH, and that this binding is necessary to activate the Ca<sup>2+</sup>-conducting pore in the uniporter complex.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">MCU recognition of EMRE</h2><p class="jrnlSecPara">To examine the MCU side of the interaction with EMRE, we launched Trp-perturbation mutagenesis of both TMHs (residues L234 &ndash; T254 in TMH1; Y268 &ndash; V283 in TMH2). All mutants were expressed to near WT levels in MCU-KO cells (<span class="jrnlFigRef" data-citation-string=" F4-S1 ">Figure 4&mdash;figure supplement 1</span>), and were classified as either low- or high-impact on Ca<sup>2+</sup> transport function (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4A</span>, <span class="jrnlFigRef" data-citation-string=" F4-S1 ">Figure 4&mdash;figure supplement 1</span>). Of the 16 Trp mutations in TMH2, only two (F269W, T271W, located near the N-terminal end of the helix) induce severe functional defects, as if most TMH2 residues project either to the lipid bilayer or an aqueous cavity. In contrast, the six Trp-sensitive positions in TMH1 tend toward one side of a helical wheel diagram, suggesting that this high-impact face might pack against other TMHs in the MCU-EMRE complex.</p><div class="jrnlFigBlock" id="BLK_F4" data-stream-name="a_F4" data-page-num="7" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F4"><span class="label">Figure 4.</span> Interactions between EMRE and MCU&rsquo;s TMH1.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Helical projection diagram summarizing functional impact of Trp substitutions in TMHs of MCU. Trp mutation that reduces Ca<sup fontsize="11.0pt">2+</sup> uptake by &gt;70% is defined as high impact (red), and &lt;30% as low impact (black). Red arc highlights proposed helical surface sensitive to Trp substitutions. (<span class="partLabel">B</span>) Residue swap showing an impaired MCU mutant (A241F) forming a highly functional uniporter complex with an EMRE mutant (F77A). <i fontsize="11.0pt">Left</i>: Ca<sup fontsize="11.0pt">2+</sup> uptake in ME-KO cells expressing indicated MCU and EMRE mutants. <i fontsize="11.0pt">Right</i>: Ca<sup fontsize="11.0pt">2+</sup> uptake (upper) and expression of key uniporter proteins (lower) in cells transfected with A241F-MCU and WT- or F77A-EMRE. (<span class="partLabel">C</span>) Co-IP experiments comparing complex formation of A241F-MCU with WT- or F77A-EMRE. (<span class="partLabel">D</span>) Ca<sup fontsize="11.0pt">2+</sup> flux in a hMCU-ceMCU chimera (human portion: yellow, <i fontsize="11.0pt">C. elegans</i> portion: blue), coexpressed with either hEMRE or ceEMRE in ME-KO cells. See also <span class="jrnlFigRef" data-citation-string=" F4-S1 F4-S2 F4-S3 ">Figure 4&mdash;figure&nbsp;supplements 1&ndash;3</span>.</p><img src="/ecs/article/82842/resources/elife15545f004.jpg" data-mce-src="/ecs/article/82842/resources/elife15545f004.jpg" data-primary-extension="tif" data-width="2066" data-height="1124" data-id="1985912" alt="ecs/article/82842/resources/elife15545f004.jpg" height="446" width="819"/></div><div class="jrnlFigBlock" id="BLK_F4-S1" data-stream-name="a_F4-S1"><p class="jrnlFigCaption" id="F4-S1"><span class="label">Figure 4&mdash;figure&nbsp;supplement 1.</span> Functional impact of mutations in MCU&rsquo;s TM helices.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Expression of selected MCU mutants. (<span class="partLabel">B&ndash;C</span>) The effect of mutations in MCU&rsquo;s TMH1 (<span class="partLabel">B</span>) or TMH2 (<span class="partLabel">C</span>) on Ca<sup fontsize="11.0pt">2+</sup> flux via the uniporter. <i fontsize="11.0pt">Dashed red line</i>: 30% WT activity.</p><img src="/ecs/article/82842/resources/elife15545fs003.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs003.jpg" data-primary-extension="tif" data-width="1383" data-height="1486" data-id="1985792" alt="ecs/article/82842/resources/elife15545fs003.jpg" height="880" width="819"/></div><div class="jrnlFigBlock" id="BLK_F4-S2" data-stream-name="a_F4-S2"><p class="jrnlFigCaption" id="F4-S2"><span class="label">Figure 4&mdash;figure&nbsp;supplement 2.</span> Trp mutations in D. discoideum MCU.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Sequence alignment of TMH1 and TMH2 of human and <i fontsize="11.0pt">D. discoideum</i> MCU homologues. Red indicates residues selected for Trp substitution. (<span class="partLabel">B</span>) Characterization of <i fontsize="11.0pt">D. discoideum</i> MCU mutants. <i fontsize="11.0pt">Left</i>: Ca<sup fontsize="11.0pt">2+</sup> flux in ME-KO cells. <i fontsize="11.0pt">Right</i>: expression of the mutants. (<span class="partLabel">C</span>) The activity of Trp mutants compared with WT <i fontsize="11.0pt">D. discoideum</i> MCU.</p><img src="/ecs/article/82842/resources/elife15545fs004.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs004.jpg" data-primary-extension="tif" data-width="1918" data-height="1222" data-id="1985752" alt="ecs/article/82842/resources/elife15545fs004.jpg" height="522" width="819"/></div><div class="jrnlFigBlock" id="BLK_F4-S3" data-stream-name="a_F4-S3"><p class="jrnlFigCaption" id="F4-S3"><span class="label">Figure 4&mdash;figure&nbsp;supplement 3.</span> Uniporter formation by F77A EMRE and MCU mutants.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Uniporter activity in EMRE-KO cells expressing F77A EMRE and MCU mutants as indicated. Activity is normalized to that produced by WT MCU and WT EMRE. (<span class="partLabel">B</span>) Co-IP test of complex formation by A241W MCU and WT- or F77A-EMRE.</p><img src="/ecs/article/82842/resources/elife15545fs005.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs005.jpg" data-primary-extension="tif" data-width="2645" data-height="928" data-id="1985772" alt="ecs/article/82842/resources/elife15545fs005.jpg" height="288" width="819"/></div><p class="jrnlSecPara">If a Trp mutation in MCU perturbs the function of human uniporter solely by interrupting EMRE binding, it should cause negligible functional impact when introduced at corresponding positions in <i>D. discoideum</i> MCU, because this homologue transports Ca<sup>2+</sup> without EMRE (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>). We therefore introduced each of the eight high-impact Trp substitutions (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4A</span>) into <i>D. discoideum</i> MCU to test the mutational effect in ME-KO cells. The results (<span class="jrnlFigRef" data-citation-string=" F4-S2 ">Figure 4&mdash;figure supplement 2</span>) show that six of these mutations are functionally disruptive while two are fully active; these two active mutants correspond to L240W and A241W in TMH1 of human MCU, suggesting that these residues in human MCU contact EMRE.</p><p class="jrnlSecPara">A steric clash between a substituted Trp in MCU and a native residue in EMRE could in principle be alleviated by reducing the side-chain volume of that particular EMRE residue. We chose F77 in EMRE to test this idea, as it is on the same helical face as the GxxxS sequence identified above, and since the large Phe residue enables substantial shortening of the side chain. Accordingly, F77A EMRE was coexpressed with each of the 6 functionally defective MCU Trp mutants of TMH1. This EMRE mutant, which forms a functional channel with WT MCU, rescues Ca<sup>2+</sup> transport with A241W but not with any of the other mutants (<span class="jrnlFigRef" data-citation-string=" F4-S3 ">Figure 4&mdash;figure supplement 3</span>). Similarly, the impaired uniporter function induced by A241F in MCU is rescued by F77A in EMRE. Co-IP experiments further show that A241F (or A241W) MCU pulls down F77A but not WT EMRE (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4C</span>, <span class="jrnlFigRef" data-citation-string=" F4-S3 ">Figure 4&mdash;figure supplement 3</span>). These results demonstrate that the combination of a large and a small side chain (F or W, A) on EMRE position 77 and MCU position 241 leads to proper transport function regardless of which protein the residues occupy. This 'side chain swap' experiment argues strongly that A241 in MCU&rsquo;s TMH1 is in close proximity to F77 in the TMH of EMRE in the uniporter complex.</p><p class="jrnlSecPara">While analyzing MCU/EMRE from various species (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>), we noticed that human MCU (hMCU) forms functional Ca<sup>2+</sup> channels with human EMRE (hEMRE) or with <i>C. elegans</i> EMRE (ceEMRE), but <i>C. elegans</i> MCU (ceMCU) supports mitochondrial Ca<sup>2+</sup> uptake only with ceEMRE (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4D</span>). These results present an opportunity to test which of ceMCU&rsquo;s two TMHs is responsible for discriminating against hEMRE. Accordingly, we produced two MCU chimeras, with TMH1 or TMH2 of ceMCU substituted by the corresponding region in hMCU. The chimera containing hMCU TMH1, though well expressed, is transport-inactive in the presence of either hEMRE or ceEMRE. The chimera containing hMCU TMH2, however, is activated by ceEMRE, while remaining unresponsive to hEMRE (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4D</span>). This result independently supports the proposal that TMH1 of MCU contains the interaction region for EMRE.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Localization and inner membrane association of MICUs</h2><p class="jrnlSecPara">With the membrane disposition of the MCU-EMRE complex in hand, we now ask how EMRE interacts with MICU proteins (<span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>), which contain no apparent transmembrane sequences. To tackle this problem, we began by testing whether MICUs bind to EMRE from the matrix- or IMS-side of the inner membrane. Currently, the submitochondrial localization of MICU1 is unsettled (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R11 ">Foskett and Madesh, 2014</span>; <span class="jrnlBibRef" data-citation-string=" R13 ">Hoffman et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R15 ">Hung et al., 2014</span>; <span class="jrnlBibRef" data-citation-string=" R33 ">Waldeck-Weiermair et al., 2015</span>), and that of MICU2 has not been approached with rigorous methods (<span class="jrnlBibRef" data-citation-string=" R32 ">Vais et al., 2016</span>). Moreover, although MICU1 is known to be a peripheral membrane protein (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>), it remains unclear if MICU2 is similarly attached to the inner membrane.</p><p class="jrnlSecPara">Inner membrane association of MICUs was probed by stripping mitoplasts of peripheral membrane proteins using alkaline Na<sub>2</sub>CO<sub>3</sub> treatments. <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5A</span> shows that MICU1 and cytochrome C (Cyt-C), but not the integral membrane protein Letm1, are extracted into Na<sub>2</sub>CO<sub>3 </sub>solution, a result consistent with a previous report (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>) that MICU1 is a peripheral membrane protein. Furthermore, nearly all MICU1 is membrane-bound, virtually none appearing in the IMS without Na<sub>2</sub>CO<sub>3</sub> treatment, in contrast to Cyt-C, which is found in both the IMS and the membrane (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5A</span>). Similar experiments demonstrate that MICU2 remains membrane-associated even after harsher Na<sub>2</sub>CO<sub>3</sub> extraction conditions (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5B</span>). The results thus establish that MICU1 and MICU2 are both confined to the inner-membrane surface under physiological conditions.</p><div class="jrnlFigBlock" id="BLK_F5" data-stream-name="a_F5" data-page-num="8" data-float-placement="BottomMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F5"><span class="label">Figure 5.</span> Localization of MICUs and interaction with the pore region.</p><p class="jrnlFigCaption">(<span class="partLabel">A&ndash;B</span>) Carbonate extraction (pH 10.5 or 11.5) of MICU1 at 4&deg;C (<span class="partLabel">A</span>) or MICU2 at room temperature (<span class="partLabel">B</span>) for 1 h, showing membrane pellet (M), proteins extracted into supernatant (S), and control (con) with mitoplasts treated at pH 7.0. (<span class="partLabel">C&ndash; D</span>) PEGM modification of MICU1 or MICU2. Both MICUs are detected at monomer (~64 kDa) or dimer (~115 kDa) positions. (<span class="partLabel">E</span>) Co-IP experiments using immobilized Flag-tagged MICU1 to pull down MCU or EMRE. For all experiments shown in this figure, MICU1 and MICU2 were Flag- and V5-tagged, respectively, and were detected using corresponding Flag and V5 antibodies. See also <span class="jrnlFigRef" data-citation-string=" F5-S1 ">Figure 5&mdash;figure supplement 1</span>.</p><img src="/ecs/article/82842/resources/elife15545f005.jpg" data-mce-src="ecs/article/82842/resources/elife15545f005.jpg" height="491" width="819"/></div><div class="jrnlFigBlock" id="BLK_F5-S1" data-stream-name="a_F5-S1"><p class="jrnlFigCaption" id="F5-S1"><span class="label">Figure 5&mdash;figure supplement 1.</span> MICU2 interaction with other subunits in the uniporter complex.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Co-IP experiment showing complex formation of MICU1 and MICU2. MICU2 was used as bait. (<span class="partLabel">B</span>) Co-IP experiments using immobilized MICU2 to pull down MCU or EMRE. (<span class="partLabel">C</span>) Immobilized MICU2 was used to pull down EMRE with MICU1 also expressed. For all Co-IP experiments, ME-KO cells were used, and proteins being expressed are indicated at the upper panel above the Western results. MICU1 and MICU2 were Flag and V5 tagged, respectively, and were detected with corresponding antibodies. IP: elution. CL: cell lysate.</p><img src="/ecs/article/82842/resources/elife15545fs006.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs006.jpg" data-primary-extension="tif" data-width="2533" data-height="886" data-id="1985812" alt="ecs/article/82842/resources/elife15545fs006.jpg" height="287" width="819"/></div><p class="jrnlSecPara">To test if MICU1 is exposed to the mitochondrial matrix or the IMS, we incubated mitoplasts with PEGM, relying on MICU1's native cysteines to report accessibility to the reagent from the mitoplast exterior. PEGM readily reacts with MICU1 (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5C</span>), showing that this subunit resides on the outer, IMS side of the inner membrane. Two issues regarding this experiment require comment. First, the PEGM-treated sample appears as a smear instead of a defined band, consistent with heterogeneous modification of the protein's 7 native cysteines. Second, a significant western-blot signal is observed roughly corresponding to MICU1 dimers, stable in SDS-PAGE conditions, as reported previously (<span class="jrnlBibRef" data-citation-string=" R25 ">Patron et al., 2014</span>). This signal also shifts upward after PEGM treatment, suggesting that MICU1 dimers or higher oligomers are also localized to the IMS. Similar results in parallel experiments with MICU2 (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5D</span>) argue that both MICU1 and MICU2 are associated with the outer leaflet of the mitochondrial inner membrane, a location in harmony with the cytoplasmic Ca<sup>2+</sup>-sensing function of these proteins.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Interaction of MICUs with the pore-forming region</h2><p class="jrnlSecPara">It has been established, as we also confirm here (<span class="jrnlFigRef" data-citation-string=" F5-S1 ">Figure 5&mdash;figure supplement 1</span>), that MICU1 forms a stable complex with MICU2 (<span class="jrnlBibRef" data-citation-string=" R16 ">Kamer and Mootha, 2014</span>; <span class="jrnlBibRef" data-citation-string=" R25 ">Patron et al., 2014</span>), and that MICU1, but not MICU2, is tightly associated with the MCU-EMRE complex (<span class="jrnlBibRef" data-citation-string=" R16 ">Kamer and Mootha, 2014</span>; <span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>). It however remains uncertain if MICU1 interacts with MCU or EMRE (<span class="jrnlBibRef" data-citation-string=" R13 ">Hoffman et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R30 ">Sancak et al., 2013</span>), a problem addressed using co-IP to examine association of FLAG-tagged MICU1 with MCU or EMRE expressed individually in ME-KO cells. As illustrated in <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span>, MICU1 can precipitate EMRE without MCU present and MCU without EMRE present. This observation rules out a required MICU1-interacting surface contributed by both MCU and EMRE. It also invites us to search for the molecular determinants mediating MICU1-EMRE interaction in the absence of MCU.</p><p class="jrnlSecPara">The IMS localization of MICU1 (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5C</span>) implies that EMRE binds to MICU1 via its C-terminal, IMS-exposed region containing the polyaspartate tail (EDDDDDD). This highly charged tail alerts us to a complementary polybasic sequence (KKKKR), which though conserved in MICU1, is absent in MICU2 (<span class="jrnlBibRef" data-citation-string=" R13 ">Hoffman et al., 2013</span>). Indeed, co-IP experiments demonstrate that MICU1 pulls down &Delta;N- but not &Delta;C-EMRE (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6A</span>). Moreover, a MICU1 mutant carrying an electrostatically neutered sequence (KKKKR =&gt;EQEQR) readily complexes with MICU2, but not with EMRE (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6B</span>). These results strongly argue that the EMRE-MICU1 interaction is mediated by this electrostatic pair. The strict conservation of these charged sequences further suggests that the EMRE-MICU1 interaction plays an important, previously unappreciated physiological role.</p><div class="jrnlFigBlock" id="BLK_F6" data-stream-name="a_F6" data-page-num="10" data-float-placement="TopDoubleColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F6"><span class="label">Figure 6.</span> Functional importance of the MICU1-EMRE interaction.</p><p class="jrnlFigCaption">(<span class="partLabel">A&ndash;B</span>) Co-IP experiments with WT- or EQEQ-MICU1 (Flag-tagged) used to pull down WT or mutant EMRE proteins in ME-KO cells. MICU1 was detected using anti-Flag, MICU2 by anti-V5, and &Delta;N- or &Delta;C-EMRE by anti-C8. (<span class="partLabel">C&ndash;D</span>) The effect of MICU1 knockdown on mitochondrial Ca<sup fontsize="11.0pt">2+</sup> uptake in WT HEK 293 cells at high (<span class="partLabel">C</span>) or low (<span class="partLabel">D</span>) Ca<sup fontsize="11.0pt">2+</sup> conditions. Con: cells with no MICU KD. sh1-3 indicates three stable cell lines expressing distinct shRNAs against <i fontsize="11.0pt">MICU1</i> mRNA. (<span class="partLabel">E</span>) Mitochondrial Ca<sup fontsize="11.0pt">2+</sup> uptake (30 &micro;M Ca<sup fontsize="11.0pt">2+</sup>) using untransfected EMRE-KO cells, or cells expressing WT-, &Delta;N-, or &Delta;C-EMRE as indicated. (<span class="partLabel">F&ndash;G</span>) Ca<sup fontsize="11.0pt">2+</sup> flux (0.5 &micro;M Ca<sup fontsize="11.0pt">2+</sup>) via MCU complexed with WT- or &Delta;N-EMRE (<span class="partLabel">F</span>), or &Delta;C-EMRE (<span class="partLabel">G</span>) in the presence or absence of stable MICU1 KD by shRNA2. Data shown in <span class="partLabel">C&ndash;</span><span class="partLabel">G</span> represent mean &plusmn; s.e.m. of 3&ndash;4 independent measurements. See also <span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1</span>.</p><img src="/ecs/article/82842/resources/elife15545f006.jpg" data-mce-src="ecs/article/82842/resources/elife15545f006.jpg" height="469" width="819"/></div><div class="jrnlFigBlock" id="BLK_F6-S1" data-stream-name="a_F6-S1"><p class="jrnlFigCaption" id="F6-S1"><span class="label">Figure 6&mdash;figure supplement 1.</span> Biochemical characterization of MICU1 knockdown cells.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) the mRNA level of MICU1 in MICU1 knockdown cells compared with WT control, as determined by qPCR. (<span class="partLabel">B</span>) MCU and EMRE expression in these cells.</p><img src="/ecs/article/82842/resources/elife15545fs007.jpg" data-mce-src="/ecs/article/82842/resources/elife15545fs007.jpg" data-primary-extension="tif" data-width="1665" data-height="510" data-id="1985732" alt="ecs/article/82842/resources/elife15545fs007.jpg" height="251" width="819"/></div><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Functional role of the MICU1-EMRE interaction</h2><p class="jrnlSecPara">The MICU proteins act as the Ca<sup>2+</sup>-sensing gate in the uniporter complex, shutting the pore at resting cellular Ca<sup>2+</sup> concentrations and opening it when cytoplasmic Ca<sup>2+</sup> exceeds ~1 &micro;M. By binding to both MCU and MICU1, EMRE might serve as an anchor to retain the MICU1-MICU2 pair near the Ca<sup>2+</sup>-conducting pore. If so, disrupting the MICU1-EMRE interaction would yield a population of channels free of MICUs, allowing unregulated, constitutive Ca<sup>2+ </sup>permeation from the cytosol into the matrix. To examine this idea, we quantify MCU-dependent Ca<sup>2+</sup> uptake by following accumulation of the <sup>45</sup>Ca<sup>2+</sup> radioisotope into mitochondria in digitonin-permeabilized cells, an approach that allows free extramitochondrial Ca<sup>2+</sup> to be buffered at well-defined submicromolar concentrations without sacrificing sensitivity.</p><p class="jrnlSecPara">We first examine the effect of MICU1 knockdown (KD) on mitochondrial Ca<sup>2+</sup> uptake. Three HEK293 cell lines stably expressing distinct short hairpin RNAs (sh1 &ndash; 3) were generated, all exhibiting at least 70% decrease of <i>MICU1</i> mRNA and normal levels of MCU or EMRE protein (<span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1</span>). Although insufficiently sensitive antibodies frustrate quantification of MICU1, the MICU1-KD cell lines show a profound functional alteration in <sup>45</sup>Ca<sup>2+</sup> uptake. At high Ca<sup>2+ </sup>(30&thinsp;&mu;M), rates of Ca<sup>2+</sup> transport into WT and MICU1-KD mitochondria are similar, while as expected, uptake is virtually undetectable in ME-KO cells (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6C</span>). At low Ca<sup>2+ </sup>(0.5&thinsp;&mu;M), however, MICU1-KD mitochondria show massive Ca<sup>2+</sup> accumulation in the matrix (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6D</span>), in dramatic contrast to WT, where very little Ca<sup>2+</sup> uptake occurs. These observations confirm previous studies that identified MICU1 as a molecular element controlling Ca<sup>2+</sup>-dependent activation of the uniporter complex (<span class="jrnlBibRef" data-citation-string=" R5 ">Csordas et al., 2013</span>; <span class="jrnlBibRef" data-citation-string=" R23 ">Mallilankaraman et al., 2012</span>).</p><p class="jrnlSecPara">Functional manifestations of the EMRE-MICU1 interaction were further examined by comparing <sup>45</sup>Ca<sup>2+</sup> uptake supported by EMRE variants expressed in EMRE-KO cells. As above, WT-, &Delta;N-, or &Delta;C-EMRE all activate MCU-dependent Ca<sup>2+</sup> uptake to a similar degree at high Ca<sup>2+</sup> (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6E</span>). At low Ca<sup>2+</sup>, uptake is suppressed in mitochondria hosting WT or &Delta;N-EMRE, but is enhanced over 50-fold by MICU1 knockdown (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6F</span>). In contrast, Ca<sup>2+</sup> rapidly enters mitochondria containing &Delta;C-EMRE, which cannot bind MICU1, and the rate is only trivially increased after MICU1 KD (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6G</span>). We thus conclude that MICU1 must bind EMRE to maintain uninterrupted engagement with the MCU pore, thus conferring Ca<sup>2+</sup>-dependent gating upon what would otherwise be constitutive Ca<sup>2+</sup> leakage into the mitochondrial matrix.</p><h1 class="jrnlHead1" bold="true" patternhead="true">Discussion</h1><p class="jrnlSecPara">In the past few years, the mitochondrial Ca<sup>2+</sup> transport field has witnessed a molecular dawn following a half-century of functional phenomenology. It is now firmly established that the human mitochondrial Ca<sup>2+</sup> uniporter is a Ca<sup>2+</sup>-activated Ca<sup>2+</sup> channel composed of at least four proteins: MCU, EMRE, MICU1, and MICU2. The present work focuses mainly on EMRE, the least understood component of the channel complex. Impressed by the functional dependence on EMRE arising as metazoan uniporters evolved, we have endeavored to enrich our current view of the uniporter subunits and the physiological purposes of the domain interactions mediating their assembly. Results here establish (1) EMRE orientation in the inner membrane, (2) the molecular contacts governing EMRE interactions with MCU and MICU1, (3) the disposition of the MICU1-MICU2 complex on the outer surface of the inner-membrane, and (4) the functional purpose of the EMRE-MICU1 interaction. These findings lead to a molecular model (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7</span>) featuring a central role of EMRE in orchestrating uniporter responses to intracellular Ca<sup>2+</sup> signaling.</p><div class="jrnlFigBlock" id="BLK_F7" data-stream-name="a_F7" data-page-num="11" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F7"><span class="label">Figure 7.</span> The physiological role of EMRE.</p><p class="jrnlFigCaption">Cartoon summarizing main findings here, illustrating how the two functions of EMRE, MCU activation and MICU1 retention, together prevent Ca<sup fontsize="11.0pt">2+</sup> leakage through the uniporter complex in resting cellular conditions.</p><img src="/ecs/article/82842/resources/elife15545f007.jpg" data-mce-src="ecs/article/82842/resources/elife15545f007.jpg" height="319" width="819"/></div><p class="jrnlSecPara">When a Ca<sup>2+</sup> channel in the plasma membrane opens, or when an intracellular store releases Ca<sup>2+</sup>, a cytoplasmic Ca<sup>2+</sup> wave is generated. Once the wave hits mitochondria, Ca<sup>2+</sup> can rise above ~1&thinsp;&mu;M, activating the uniporter to catalyze rapid Ca<sup>2+</sup> entry into the mitochondrial matrix. Mitochondria can therefore serve as a buffer to shape intracellular Ca<sup>2+</sup> signals (<span class="jrnlBibRef" data-citation-string=" R9 ">Demaurex et al., 2009</span>). Moreover, Ca<sup>2+</sup> entry boosts ATP output by accelerating the citric acid cycle, but excessive, sustained Ca<sup>2+</sup> accumulation in the matrix triggers caspase-dependent apoptosis (<span class="jrnlBibRef" data-citation-string=" R28 ">Rizzuto et al., 2012</span>). Thus, mitochondria can also decode Ca<sup>2+</sup> stimulation as either metabolic or death signals. Failure of the uniporter to appropriately respond to Ca<sup>2+</sup> signals would perturb these crucial physiological processes and could also produce other serious problems. For instance, if the uniporter fails to stay inactive under resting conditions, the large negative inner membrane potential of energized mitochondria would drive continual Ca<sup>2+</sup> influx. Removing these Ca<sup>2+</sup> ions requires the action of Na<sup>+</sup>/Ca<sup>2+</sup> and Na<sup>+</sup>/H<sup>+</sup> exchangers at a cost of 3&thinsp;H<sup>+</sup> entering into the matrix for each Ca<sup>2+</sup> expelled. Unregulated, &lsquo;leaky&rsquo; uniporters would therefore divert protons away from the F<sub>o</sub>F<sub>1</sub>-ATPase, partially uncoupling electron transport from ATP synthesis.</p><p class="jrnlSecPara">The recent discovery of MICU proteins (<span class="jrnlBibRef" data-citation-string=" R2 ">Baughman et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R23 ">Mallilankaraman et al., 2012</span>) has enhanced our mechanistic understanding of how cellular Ca<sup>2+</sup> signals control uniporter activity. It is now clear that the mitochondrial response to physiological Ca<sup>2+ </sup>is mediated by the Ca<sup>2+</sup>-sensing gate formed by the MICU subunits that engage the pore-lining MCU proteins at the external face of the inner membrane. This modular design comes with a potential danger: that the gating and ion-conducing subunits might dissociate sporadically, producing unregulated Ca<sup>2+ </sup>channels. How could this problem be prevented? It is likely that MICU1 has a high intrinsic affinity to MCU, since at least part of the MICU1-MCU complex survives lengthy co-IP experiments. Moreover, diffusion of MICUs confined to two dimensions within the inner membrane raises the local density of MICUs near the MCU pore. These factors combined could in principle reduce the population of MICU-free, unregulated uniporters &mdash; a condition that might be particularly helpful to protist and plant mitochondria, where MCU and MICUs are the only components of the uniporter complex.</p><p class="jrnlSecPara">The small, single-pass membrane protein EMRE emerges in animals as a new subunit of the uniporter complex. A crucial finding here is that the presence of EMRE in the uniporter complex ensures that the channel conducts Ca<sup>2+</sup> only when cytoplasmic Ca<sup>2+</sup> rises above resting levels. This requires EMRE to use its polyaspartate tail to bind MICU1, as if it functions as 'molecular glue' to prevent dissociation of MICU1 from the MCU pore, a circumstance that would produce catastrophic Ca<sup>2+</sup> leakage (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7</span>). Alternatively, EMRE might allosterically transmit the Ca<sup>2+</sup> signal from MICUs to the pore; in this case, disrupting EMRE-MICU1 interaction would also prevent MICUs from properly gating the Ca<sup>2+ </sup>pathway. We consider this allosteric scenario unlikely, as the mechanism by which MICU1 gates MCU probably evolved in early eukaryotic evolution when EMRE was absent.</p><p class="jrnlSecPara">The understanding that EMRE safeguards mitochondria against inappropriate Ca<sup>2+ </sup>uptake helps us appreciate the physiological importance of the strict EMRE-dependence of uniporter function appearing in animals. As in any multisubunit protein, it is inevitable that EMRE might occasionally dissociate from MCU, and some tissues under natural or pathological conditions might express MCU in excess of EMRE. Under these situations, a population of EMRE-free uniporters could arise. These channels would also lack MICUs, which would no longer be EMRE-linked to the pore. But the MCU-activation function of EMRE would ensure that these channels would become inactive, preventing them from wreaking havoc on normal cell physiology (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7</span>).</p><p class="jrnlSecPara">We should point out that our results clash with several published assertions regarding the uniporter complex. First, single-channel recordings in planar lipid bilayers have been used to argue that the human MCU protein alone is sufficient to reconstitute a Ca<sup>2+</sup> channel without EMRE (<span class="jrnlBibRef" data-citation-string=" R6 ">De Stefani et al., 2011</span>; <span class="jrnlBibRef" data-citation-string=" R25 ">Patron et al., 2014</span>). These recordings, however, obtained with in vitro-expressed protein of uncharacterized purity, show channel properties vastly different from uniporter currents directly patch-recorded from intact mitoplasts (<span class="jrnlBibRef" data-citation-string=" R19 ">Kirichok et al., 2004</span>). Second, a recent study (<span class="jrnlBibRef" data-citation-string=" R32 ">Vais et al., 2016</span>) using protease digestion argues that EMRE adopts a N<sub>out</sub>-C<sub>in</sub> orientation, opposite to that deduced here. Interpretation of the assay, however, is based on an unjustified assumption that the N- and C- termini of EMRE are digested at similar rates. In contrast, our results are supported by two lines of direct and independent evidence &ndash; mass tagging of substituted cysteines and the functional competence of the MCU-EMRE fusion protein (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1</span>). This same study also claims that EMRE uses its C-terminal tail to sense matrix Ca<sup>2+</sup>, producing a biphasic response of uniporter activity to matrix Ca<sup>2+</sup>. However, this phenomenon, to our best knowledge, has not been observed in any mitochondrial Ca<sup>2+</sup> uptake experiments in the literature or in previous patch recordings (<span class="jrnlBibRef" data-citation-string=" R19 ">Kirichok et al., 2004</span>). We also note a recent study appearing when our work was under review (<span class="jrnlBibRef" data-citation-string=" R34 ">Yamamoto et al., 2016</span>) that deduced, using an epitope-tagging method, a N<sub>in</sub>-C<sub>out</sub> EMRE orientation fully consistent with our results. That study also shows that a Pro-to-Ala substitution 3 residues N-terminal to the predicted TMH - a region left unperturbed in our 22-residue deletion &Delta;N-EMRE construct - abolishes EMRE-MCU interaction. Thus, a small portion in the N-terminus of EMRE near the TMH might also be involved in binding MCU.</p><p class="jrnlSecPara">In summary, our results argue that EMRE mediates two distinct functions &ndash; MCU activation and MICU retention - through two distinct types of subunit-subunit interactions. These functions conspire to achieve a single physiological outcome: obligatory linkage of the Ca<sup>2+</sup>-conducting and Ca<sup>2+</sup>-sensing machineries, a necessary condition for the uniporter complex to respond rapidly and accurately to the elaborate Ca<sup>2+</sup>-signaling network that has evolved in animal cells.</p><h1 class="jrnlHead1" bold="true" patternhead="true">Materials and methods</h1><h2 class="jrnlHead2" italic="true" headweightage="19">Molecular biology, cell culture, and transient expression</h2><p class="jrnlSecPara">Site-directed mutagenesis was performed using the QuickChange mutagenesis kit (Agilent). HEK 293 cells were grown in Dulbecco&rsquo;s modified Eagle&rsquo;s medium supplemented with 10% FBS, and were incubated at 37&deg;C, 5% CO<sub>2</sub>. The HEK-293&nbsp;cell line was supplied by Dr. D.E. Clapham and&nbsp;authenticated by short tandem repeat profiling conducted by ATCC, and was free of mycoplasma as determined by PCR based detection using a kit supplied by ATCC (30-1012K).&nbsp;Transient transfection was performed using Lipofectamine 3000 (Life Technologies), following manufacturer&rsquo;s instructions. Cells were used for downstream analysis 1-2 days after <span class="authorQueryNode Author" id="1463561630901" rid="QRY_1463561630901" data-username="Alistair (eLife Sciences Publications, Ltd)" data-time="05/18/2016 9:53am">transfection.</span>
</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">RNA interference</h2><p class="jrnlSecPara">Stable knockdown was achieved by lentivirus, using the transfer vector pLKO.1 puro (Sigma) for U6-driven shRNA expression. The viral titer was determined with a p24 ELISA kit (<span class="authorQueryNode Author" id="1461489068502" rid="QRY_1461489068502" data-username="Sarumathi (eLife Sciences Publications, Ltd)" data-time="04/24/2016 2:41pm">Clontech, Mountain View, CA)</span>. WT HEK293 cells were exposed to the virus for 12&thinsp;hr, using a multiplicity of infection of 5&ndash;10. Afterwards, the culture was incubated with 2&thinsp;&mu;g/mL puromycin for 2 days to eliminate untransduced cells. The efficiency of knockdown was evaluated by quantitative PCR (qPCR). Detailed qPCR procedure and the shRNA sequences are reported in Extended Experimental Procedures.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Gene knockout by CRISPR/Cas9</h2><p class="jrnlSecPara">Gene knockout by CRISPR/Cas9 was performed using the published protocol (<span class="jrnlBibRef" data-citation-string=" R27 ">Ran et al., 2013</span>). In brief, the pSpCas9(BB) vector containing the 20-nucleotide guide sequence was transfected into HEK 293 cells. After two days of incubation, single cells were isolated by serial dilutions, and expanded for 2&ndash;4 weeks. Gene KO was assessed by sequencing and Western blot. Two sets of guide sequences (see <span class="jrnlSupplRef" data-citation-string=" SP1 ">Supplementary file 1</span>) were used to rule out off-target effects.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Western blot and co-immunoprecipitation</h2><p class="jrnlSecPara">For Western blot, proteins on an SDS gel were transferred onto nitrocellulose membranes, which were blocked by 5% milk in TBS, and then incubated with the primary antibody diluted in TBST (TBS + 0.1% Tween-20). Signal development was done using alkaline phosphatase conjugated secondary antibodies (Pierce) and the NBT/BCIP substrate (Life Technologies). The primary antibody and dilution used: &alpha;-MCU (Sigma, HPA016480, 1:2000), &alpha;-EMRE (Santa Cruz, 86337, 1:400), &alpha;-FLAG (Sigma, F1804, 1:4000), &alpha;-V5 (Life Technologies, 46&ndash;0705, 1:5000), &alpha;-Cyt-C (Santa Cruz, 13156, 1:1000), &alpha;-&beta;-actin (Santa Cruz, 69879, 1:500), &alpha;-Letm1 (Abcam, 55434, 1:2000). Monoclonal anti-1D4 and -C8 antibodies were produced in house.</p><p class="jrnlSecPara">All co-IP experiments were performed at 4 &deg;C. Transfected HEK 293 cells were grown in a 10-cm dish to confluency, were harvested, and then lysed in 1-mL solubilization buffer (SB, 100&thinsp;mM NaCl, 20&thinsp;mM Tris, 1&thinsp;mM EGTA, 25&thinsp;mM DDM, pH 7.5-HCl), supplemented with an EDTA-free protease inhibitor cocktail (cOmplete Ultra, Roche). The cell lysate was clarified by centrifugation, and a small portion of the sample was taken for whole cell lysate analysis. Antibody-conjugated Sepharose beads (25&thinsp;&mu;L) were added, and after 1&thinsp;h, the beads were collected on a mini column, washed with 2-mL SB, and eluted with 200-&mu;L SDS-gel loading buffer for Western blot. Antibody affinity gel used: FLAG (Sigma, A2220), V5 (Sigma, A7345). 1D4 and C8 affinity gels were produced using 20-mg 1D4 or C8 antibody per 1-g Sepharose 4B (GE Healthcare).</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Mitoplast production</h2><p class="jrnlSecPara">Mitoplasts were formed at 4&deg;C by standard procedures that yield outside-out, stable transport vesicles. Protease inhibitor (cOmplete Ultra, Roche) was present in all steps. HEK 293 cells from a 15-cm dish were pelleted, resuspended in 2-mL mitochondria resuspension buffer (MRB, 250&thinsp;mM sucrose, 5&thinsp;mM HEPES, 1&thinsp;mM EGTA, pH 7.2-KOH), and lysed by passing through a 27.5&thinsp;g needle 15 &ndash; 20 times. Nuclei and cell debris were removed by spinning the cell lysate at 1000&thinsp;g for 10&thinsp;min. The supernatant was spun down at 10,000&thinsp;g for 10&thinsp;min, resuspended in 2-mL MRB, and then spun down again to pellet crude mitochondria. To obtain mitoplasts, mitochondria were resuspended in 800-&mu;L hypotonic shock buffer (5&thinsp;mM sucrose, 5&thinsp;mM HEPES, 1mM EGTA, pH 7.2-KOH), and subjected to osmotic shock for 10&thinsp;min. Then 200&thinsp;&mu;L of high-salt storage buffer (750&thinsp;mM KCl, 100&thinsp;mM HEPES, 2.5&thinsp;mM EGTA, pH 7.2-KOH) was added, and mitoplasts were subsequently sedimented by centrifugation at 20,000&thinsp;g for 10&thinsp;min. The supernatant, which contains proteins in the outer membrane and the intermembrane space, was collected if further analysis is required.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Thiol modification</h2><p class="jrnlSecPara">Mitoplasts were resuspended in the thiol-modification buffer (100&thinsp;mM NaCl, 50&thinsp;mM MOPS, pH 7.0-NaOH), to which 1&thinsp;mM PEGM (Sigma) in the presence or absence of 1&thinsp;mM DDM (Anatrace) was added. The samples were incubated for 1&ndash;4&thinsp;hr at RT before the reaction was quenched with 5&thinsp;mM cysteine. All reagents were prepared fresh before experiments.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Carbonate extraction</h2><p class="jrnlSecPara">Mitoplasts were resuspended either in carbonate extraction buffer (120&thinsp;mM NaCO<sub>3</sub>, pH 10.5- or 11.5-NaOH) or in a control solution (250&thinsp;mM sucrose, 25&thinsp;mM Tris, pH 7.0-HCl). The samples were incubated at 4&deg;C or RT for 1&thinsp;hr, and then spun down with ultracentrifugation at 200,000&thinsp;g for 1&thinsp;hr. The supernatant contains proteins extracted by carbonate, while the membrane pellet containing integral membrane proteins.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Mitochondrial Ca<sup>2+</sup> uptake assays</h2><p class="jrnlSecPara">All Ca<sup>2+</sup> uptake assays were repeated at least 3 times on multiple preparations, and traces in figures show typical responses. For the fluorescence-based assay, 10<sup>7</sup> HEK 293 cells were suspended in 10-mL Ca<sup>2+</sup> flux wash buffer (CWB, 120&thinsp;mM KCl, 25&thinsp;mM HEPES, 2&thinsp;mM KH<sub>2</sub>PO<sub>4</sub>, 1&thinsp;mM MgCl<sub>2</sub>, 50&thinsp;&mu;M EGTA, pH 7.2-KOH), pelleted at 1000&thinsp;g for 5&thinsp;min, and resuspended in 2.5-mL recording buffer (RB, 120&thinsp;mM KCl, 25&thinsp;mM HEPES, 2&thinsp;mM KH<sub>2</sub>PO<sub>4</sub>, 1&thinsp;mM MgCl<sub>2</sub>, 5&thinsp;mM succinate, pH 7.2-KOH). 2&thinsp;mL of the cell suspension was loaded into a stirred quartz cuvette in a Hitachi F-2500 spectrophotometer (ex: 506 nm, ex-slit: 2.5 nm, em: 532 nm, em-slit: 2.5 nm, sampling frequency: 2&thinsp;Hz), with the temperature maintained at 37&deg;C by a circulating bath. In a typical experiment, reagents were added into the cuvette in the following order: 0.5-&mu;M calcium green 5N (Life Technologies), 30-&mu;M digitonin (Sigma), 10-&mu;M CaCl<sub>2</sub>, and 2-&mu;M Ru360 (Santa Cruz). Under these conditions, peak free Ca<sup>2+</sup> concentrations were close to 10&thinsp;&mu;M (11 <u>+</u> 6 SD, N=40, as determined by calibration). Because of uncertainties in protein concentration, Ca<sup>2+ </sup>uptake activity is reported as a linear fit to the fluorescent signal obtained in the first 10&thinsp;s after addition of 10-&mu;M CaCl<sub>2.</sub> Activity was not altered by 5 &micro;M thapsigargin.</p><p class="jrnlSecPara">For the <sup>45</sup>Ca<sup>2+</sup> based uptake assay, 2 x 10<sup>6</sup> cells were suspended in 1.5-mL CWB, spun down at 2,000&thinsp;g for 1&thinsp;min, and resuspended again in 200-&mu;L CWB, supplemented with 5&thinsp;&mu;M thapsigargin and 30&thinsp;&mu;M digitonin. To initiate Ca<sup>2+</sup> flux, 100&thinsp;&mu;L of the cell suspension was transferred to either 400-&mu;L high-Ca<sup>2+</sup> flux buffer (RB + 10&thinsp;&mu;M EGTA and 40&thinsp;&mu;M <sup>45</sup>CaCl<sub>2</sub>) or 400-&mu;L low-Ca<sup>2+ </sup>flux buffer (RB + 0.69&thinsp;mM EGTA, 0.5&thinsp;mM CaCl<sub>2</sub>, and 20 uM <sup>45</sup>CaCl<sub>2</sub>, pH 7.2-KOH), with both solution containing 5-&mu;M thapsigargin and 30-&mu;M digitonin. At desired time points, 100&thinsp;&mu;L of the reaction mixture was added into 5-mL ice-cold CWB, and then filtered through 0.45-&mu;m nitrocellulose membranes (EMD-Millipore) on a vacuum filtration manifold (Millipore model 1225). The membrane was washed immediately with 5-mL ice cold CWB, and later transferred into scintillation vials for counting. <sup>45</sup>Ca<sup>2+</sup> radioisotope was purchased from Perkin Elmer, with a specific activity of 12.5 mCi/mg.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Bioinformatics</h2><p class="jrnlSecPara">Sequences of MCU or EMRE homologues were collected using PSI-BLAST search of ~100 species. EMRE was identified by the presence of the polyaspartic tail, and MCU by the conserved DIME loop. Multiple sequence alignment was performed using the ClustalW2 online server (<span class="jrnlBibRef" data-citation-string=" R21 ">Larkin et al., 2007</span>). The helical wheels were plotted using Antheprot v 6.4 (<span class="jrnlBibRef" data-citation-string=" R7 ">Deleage et al., 2001</span>). Mitochondrial targeting sequence prediction was carried out using the TargetP 1.1 online server (<span class="jrnlBibRef" data-citation-string=" R10 ">Emanuelsson et al., 2007</span>).</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Quantitative PCR</h2><p class="jrnlSecPara">Whole cell RNA was extracted from HEK 293 cells grown in 6-well plates using TRIzol. Residual DNA was removed using the TURBO DNA-free kit (Life Technologies). First strand cDNA synthesis was performed with 1&thinsp;&mu;g RNA using M-MuLV reverse transcriptase (NEB), following manufacturer&rsquo;s instructions. The sample was subsequently digested with RNaseH (NEB). qPCR was performed with SsoFast EvaGreen Supermixes (BIO-RAD), using 0.5&thinsp;&mu;M &beta;-actin or MICU1 primers, and 0.5, 2.5, 5, or 10 ng RNA for producing a standard curve. Detection of the PCR product was done with a CFX96 real-time PCR detection system (BIO-RAD), using the following protocol: 95&deg;C for 30&thinsp;s, 50 cycles of 95&deg;C for 5&thinsp;s and 57&deg;C for 5&thinsp;s.&nbsp;The sequence of the primers is provided in <span class="jrnlSupplRef" data-citation-string=" SP1 ">Supplementary file 1</span>.</p><p class="jrnlSecPara">&Delta;Ct was calculated by subtracting the Ct for &beta;-actin from the Ct for MICU1, with 3 independent RNA extractions and qPCR measurements using 2.5 ng whole RNA. &Delta;&Delta;Ct was calculated by subtracting the mean &Delta;Ct for control WT cells from the mean &Delta;Ct for each stable MICU1 knockdown cells. The results were presented as the percentage of MICU1 mRNA in MICU1 knockdown cells relative to MICU1 mRNA in WT control, using the equation% mRNA = 1/2<sup>|&Delta;&Delta;Ct|</sup>.</p></div><div class="back" data-edited="true"><h1 class="jrnlAckHead">Acknowledgements</h1><p class="jrnlAckPara">We thank Dr. Vamsi Mootha for plasmids and valuable advice, Daniel Turman, Dr. Nicolas Last, Dr. Joel Meyerson, Dr. Randy Stockbridge, and Dr. Rocio Finol-Urdaneta for insightful comments of the manuscript. We are particularly grateful to Dr. Daniel Oprian for assistance for raising antibodies and producing the associated affinity media, and for supporting M.R. to participate in this project. This work is supported in part by NIH Grant R01-GM107023. The authors declare no conflict of interest.</p><h1 class="jrnlRefHead">References</h1><p class="jrnlRefText" id="R1" data-seq-id="R1" sortby="Abacioglu 1994" data-old-id="R1" data-cited="true" seqid="R1" data-doi="10.1089/aid.1994.10.371" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/8068416"><span class="RefAuthor"><span class="RefSurName">Abacioglu</span> <span class="RefGivenName">YH</span></span>, <span class="RefAuthor"><span class="RefSurName">Fouts</span> <span class="RefGivenName">TR</span></span>, <span class="RefAuthor"><span class="RefSurName">Laman</span> <span class="RefGivenName">JD</span></span>, <span class="RefAuthor"><span class="RefSurName">Claassen</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Pincus</span> <span class="RefGivenName">SH</span></span>, <span class="RefAuthor"><span class="RefSurName">Moore</span> <span class="RefGivenName">JP</span></span>, <span class="RefAuthor"><span class="RefSurName">Roby</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Kamin-Lewis</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Lewis</span> <span class="RefGivenName">GK</span></span>. <span class="RefYear">1994</span>. <span class="RefArticleTitle">Epitope mapping and topology of baculovirus-expressed HIV-1 gp160 determined with a panel of murine monoclonal antibodies</span>. <span class="RefJournalTitle"><em>AIDS Research and Human Retroviruses</em></span> <span class="RefVolume"><strong>10</strong></span>:<span class="RefFPage">371</span>&ndash;<span class="RefLPage">381</span>.</p><p class="jrnlRefText" id="R2" data-seq-id="R2" sortby="Baughman 2011" data-old-id="R2" data-cited="true" seqid="R2" data-doi="10.1038/nature10234" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/21685886"><span class="RefAuthor"><span class="RefSurName">Baughman</span> <span class="RefGivenName">JM</span></span>, <span class="RefAuthor"><span class="RefSurName">Perocchi</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Girgis</span> <span class="RefGivenName">HS</span></span>, <span class="RefAuthor"><span class="RefSurName">Plovanich</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Belcher-Timme</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Sancak</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Bao</span> <span class="RefGivenName">XR</span></span>, <span class="RefAuthor"><span class="RefSurName">Strittmatter</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Goldberger</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Bogorad</span> <span class="RefGivenName">RL</span></span>, <span class="RefAuthor"><span class="RefSurName">Koteliansky</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2011</span>. <span class="RefArticleTitle">Integrative genomics identifies MCU as an essential component of the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>Nature</em></span> <span class="RefVolume"><strong>476</strong></span>:<span class="RefFPage">341</span>&ndash;<span class="RefLPage">345</span>.</p><p class="jrnlRefText" id="R3" data-seq-id="R3" sortby="Bick 2012" data-old-id="R3" data-cited="true" seqid="R3" movetopanel="yes" data-doi="10.1126/science.1214977" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22605770"><span class="RefAuthor"><span class="RefSurName">Bick</span> <span class="RefGivenName">AG</span></span>, <span class="RefAuthor"><span class="RefSurName">Calvo</span> <span class="RefGivenName">SE</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Evolutionary diversity of the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>Science</em></span> <span class="RefVolume"><strong>336</strong></span>.</p><p class="jrnlRefText" id="R4" data-seq-id="R4" sortby="Carafoli 1971" data-old-id="R4" data-cited="true" seqid="R4" data-doi="10.1042/bj1220681" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/5129264"><span class="RefAuthor"><span class="RefSurName">Carafoli</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Lehninger</span> <span class="RefGivenName">AL</span></span>. <span class="RefYear">1971</span>. <span class="RefArticleTitle">A survey of the interaction of calcium ions with mitochondria from different tissues and species</span>. <span class="RefJournalTitle"><em>The Biochemical Journal</em></span> <span class="RefVolume"><strong>122</strong></span>:<span class="RefFPage">681</span>&ndash;<span class="RefLPage">690</span>.</p><p class="jrnlRefText" id="R5" data-seq-id="R5" sortby="Csordas 2013" data-old-id="R5" data-cited="true" seqid="R5" data-doi="10.1016/j.cmet.2013.04.020" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/23747253" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Csord&aacute;s</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Golen&aacute;r</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Seifert</span> <span class="RefGivenName">EL</span></span>, <span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Sancak</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Perocchi</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Moffat</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Weaver</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">de la Fuente Perez</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Bogorad</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Koteliansky</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Adijanto</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>, <span class="RefAuthor"><span class="RefSurName">Hajn&oacute;czky</span> <span class="RefGivenName">G</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">MICU1 controls both the threshold and cooperative activation of the mitochondrial Ca&sup2;&#8314; uniporter</span>. <span class="RefJournalTitle"><em>Cell Metabolism</em></span> <span class="RefVolume"><strong>17</strong></span>:<span class="RefFPage">976</span>&ndash;<span class="RefLPage">987</span>.</p><p class="jrnlRefText" id="R6" data-seq-id="R6" sortby="De Stefani 2011" data-old-id="R6" data-cited="true" seqid="R6" data-doi="10.1038/nature10230" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/21685888"><span class="RefAuthor"><span class="RefSurName">De Stefani</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Raffaello</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Teardo</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Szab&ograve;</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">Rizzuto</span> <span class="RefGivenName">R</span></span>. <span class="RefYear">2011</span>. <span class="RefArticleTitle">A forty-kilodalton protein of the inner membrane is the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>Nature</em></span> <span class="RefVolume"><strong>476</strong></span>:<span class="RefFPage">336</span>&ndash;<span class="RefLPage">340</span>.</p><p class="jrnlRefText" id="R7" data-seq-id="R7" sortby="Deleage 2001" data-old-id="R7" data-cited="true" seqid="R7" data-doi="10.1016/S0010-4825(01)00008-7" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/11334635"><span class="RefAuthor"><span class="RefSurName">Del&eacute;age</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Combet</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Blanchet</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Geourjon</span> <span class="RefGivenName">C</span></span>. <span class="RefYear">2001</span>. <span class="RefArticleTitle">ANTHEPROT: An integrated protein sequence analysis software with client/server capabilities</span>. <span class="RefJournalTitle"><em>Computers in Biology <em>and Medicine</em></em></span> <span class="RefVolume"><em><strong>31</strong></em></span>:<span class="RefFPage">259</span>&ndash;<span class="RefLPage">267</span>.</p><p class="jrnlRefText" id="R8" data-seq-id="R8" sortby="Deluca 1961" data-old-id="R8" data-cited="true" seqid="R8" data-doi="10.1073/pnas.47.11.1744" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/13885269" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">DeLuca</span> <span class="RefGivenName">HF</span></span>, <span class="RefAuthor"><span class="RefSurName">Engstrom</span> <span class="RefGivenName">GW</span></span>. <span class="RefYear">1961</span>. <span class="RefArticleTitle">Calcium uptake by rat kidney mitochondria</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>47</strong></span>:<span class="RefFPage">1744</span>&ndash;<span class="RefLPage">1750</span>.</p><p class="jrnlRefText noPMID" id="R9" data-seq-id="R9" sortby="Demaurex 1787" data-old-id="R9" seqid="R9" movetopanel="yes" data-doi="10.1016/j.bbabio.2008.12.012" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19161976"><span class="RefAuthor"><span class="RefSurName">Demaurex</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Poburko</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Frieden</span> <span class="RefGivenName">M</span></span>. <span class="RefYear">2009</span>. <span class="RefArticleTitle">Regulation of plasma membrane calcium fluxes by mitochondria</span>. <span class="RefJournalTitle"><em>Biochimica Et Biophysica Acta</em></span> <span class="RefVolume"><strong>1787</strong></span>:<span class="RefFPage">1383</span>&ndash;<span class="RefLPage">1394</span>.</p><p class="jrnlRefText" id="R10" data-seq-id="R10" sortby="Emanuelsson 2007" data-old-id="R10" data-cited="true" seqid="R10" data-doi="10.1038/nprot.2007.131" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17446895"><span class="RefAuthor"><span class="RefSurName">Emanuelsson</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Brunak</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">von Heijne</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Nielsen</span> <span class="RefGivenName">H</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Locating proteins in the cell using targetp, signalp and related tools</span>. <span class="RefJournalTitle"><em>Nature Protocols</em></span> <span class="RefVolume"><strong>2</strong></span>:<span class="RefFPage">953</span>&ndash;<span class="RefLPage">971</span>.</p><p class="jrnlRefText" id="R11" data-seq-id="R11" sortby="Foskett 2014" data-old-id="R11" data-cited="true" seqid="R11" data-doi="10.1016/j.bbrc.2014.04.146" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24792178" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kevin Foskett</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Madesh</span> <span class="RefGivenName">M</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Regulation of the mitochondrial Ca(2+) uniporter by MICU1 and MICU2</span>. <span class="RefJournalTitle"><em>Biochemical and Biophysical Research Communications</em></span> <span class="RefVolume"><strong>449</strong></span>:<span class="RefFPage">377</span>&ndash;<span class="RefLPage">383</span>.</p><p class="jrnlRefText" id="R12" data-seq-id="R12" sortby="Gunter 2000" data-old-id="R12" data-cited="true" seqid="R12" data-doi="10.1054/ceca.2000.0168" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/11115368"><span class="RefAuthor"><span class="RefSurName">Gunter</span> <span class="RefGivenName">TE</span></span>, <span class="RefAuthor"><span class="RefSurName">Buntinas</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Sparagna</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Eliseev</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Gunter</span> <span class="RefGivenName">K</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">Mitochondrial calcium transport: Mechanisms and functions</span>. <span class="RefJournalTitle"><em>Cell Calcium</em></span> <span class="RefVolume"><strong>28</strong></span>:<span class="RefFPage">285</span>&ndash;<span class="RefLPage">296</span>.</p><p class="jrnlRefText" id="R13" data-seq-id="R13" sortby="Hoffman 2013" data-old-id="R13" data-cited="true" seqid="R13" data-doi="10.1016/j.celrep.2013.11.026" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24332854"><span class="RefAuthor"><span class="RefSurName">Hoffman</span> <span class="RefGivenName">NE</span></span>, <span class="RefAuthor"><span class="RefSurName">Chandramoorthy</span> <span class="RefGivenName">HC</span></span>, <span class="RefAuthor"><span class="RefSurName">Shamugapriya</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Rajan</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Mallilankaraman</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Gandhirajan</span> <span class="RefGivenName">RK</span></span>, <span class="RefAuthor"><span class="RefSurName">Vagnozzi</span> <span class="RefGivenName">RJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Ferrer</span> <span class="RefGivenName">LM</span></span>, <span class="RefAuthor"><span class="RefSurName">Sreekrishnanilayam</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Natarajaseenivasan</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Vallem</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Force</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Choi</span> <span class="RefGivenName">ET</span></span>, <span class="RefAuthor"><span class="RefSurName">Cheung</span> <span class="RefGivenName">JY</span></span>, <span class="RefAuthor"><span class="RefSurName">Madesh</span> <span class="RefGivenName">M</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">MICU1 motifs define mitochondrial calcium uniporter binding and activity</span>. <span class="RefJournalTitle"><em>Cell Reports</em></span> <span class="RefVolume"><strong>5</strong></span>:<span class="RefFPage">1576</span>&ndash;<span class="RefLPage">1588</span>.</p><p class="jrnlRefText" id="R14" data-seq-id="R14" sortby="Hong 2000" data-old-id="R14" data-cited="true" seqid="R14" data-doi="10.1085/jgp.115.1.51" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10613918"><span class="RefAuthor"><span class="RefSurName">Hong</span> <span class="RefGivenName">KH</span></span>, <span class="RefAuthor"><span class="RefSurName">Miller</span> <span class="RefGivenName">C</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">The lipid-protein interface of a shaker K(+) channel</span>. <span class="RefJournalTitle"><em>The Journal of General Physiology</em></span> <span class="RefVolume"><strong>115</strong></span>:<span class="RefFPage">51</span>&ndash;<span class="RefLPage">58</span>.</p><p class="jrnlRefText" id="R15" data-seq-id="R15" sortby="Hung 2014" data-old-id="R15" data-cited="true" seqid="R15" data-doi="10.1016/j.molcel.2014.06.003" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25002142"><span class="RefAuthor"><span class="RefSurName">Hung</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Zou</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Rhee</span> <span class="RefGivenName">HW</span></span>, <span class="RefAuthor"><span class="RefSurName">Udeshi</span> <span class="RefGivenName">ND</span></span>, <span class="RefAuthor"><span class="RefSurName">Cracan</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Svinkina</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Carr</span> <span class="RefGivenName">SA</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>, <span class="RefAuthor"><span class="RefSurName">Ting</span> <span class="RefGivenName">AY</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Proteomic mapping of the human mitochondrial intermembrane space in live cells via ratiometric APEX tagging</span>. <span class="RefJournalTitle"><em>Molecular Cell</em></span> <span class="RefVolume"><strong>55</strong></span>:<span class="RefFPage">332</span>&ndash;<span class="RefLPage">341</span>.</p><p class="jrnlRefText" id="R16" data-seq-id="R16" sortby="Kamer 2014" data-old-id="R16" data-cited="true" seqid="R16" data-doi="10.1002/embr.201337946" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24503055"><span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">MICU1 and MICU2 play nonredundant roles in the regulation of the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>EMBO Reports</em></span> <span class="RefVolume"><strong>15</strong></span>:<span class="RefFPage">299</span>&ndash;<span class="RefLPage">307</span>.</p><p class="jrnlRefText" id="R17" data-seq-id="R17" sortby="Kamer 2015" data-old-id="R17" data-cited="true" seqid="R17" data-doi="10.1038/nrm4039" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/26285678"><span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">The molecular era of the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>Nature Reviews. Molecular Cell Biology</em></span> <span class="RefVolume"><strong>16</strong></span>:<span class="RefFPage">545</span>&ndash;<span class="RefLPage">553</span>.</p><p class="jrnlRefText" id="R18" data-seq-id="R18" sortby="Killian 1996" data-old-id="R18" data-cited="true" seqid="R18" data-doi="10.1021/bi9519258" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/8547239"><span class="RefAuthor"><span class="RefSurName">Killian</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Salemink</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">de Planque</span> <span class="RefGivenName">MR</span></span>, <span class="RefAuthor"><span class="RefSurName">Lindblom</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Koeppe</span> <span class="RefGivenName">RE</span></span>, <span class="RefAuthor"><span class="RefSurName">Greathouse</span> <span class="RefGivenName">DV</span></span>. <span class="RefYear">1996</span>. <span class="RefArticleTitle">Induction of nonbilayer structures in diacylphosphatidylcholine model membranes by transmembrane alpha-helical peptides: Importance of hydrophobic mismatch and proposed role of tryptophans</span>. <span class="RefJournalTitle"><em>Biochemistry</em></span> <span class="RefVolume"><strong>35</strong></span>:<span class="RefFPage">1037</span>&ndash;<span class="RefLPage">1045</span>.</p><p class="jrnlRefText" id="R19" data-seq-id="R19" sortby="Kirichok 2004" data-old-id="R19" data-cited="true" seqid="R19" data-doi="10.1038/nature02246" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/14737170"><span class="RefAuthor"><span class="RefSurName">Kirichok</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Krapivinsky</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Clapham</span> <span class="RefGivenName">DE</span></span>. <span class="RefYear">2004</span>. <span class="RefArticleTitle">The mitochondrial calcium uniporter is a highly selective ion channel</span>. <span class="RefJournalTitle"><em>Nature</em></span> <span class="RefVolume"><strong>427</strong></span>:<span class="RefFPage">360</span>&ndash;<span class="RefLPage">364</span>.</p><p class="jrnlRefText" id="R20" data-seq-id="R20" sortby="Kovacs-Bogdan 2014" data-old-id="R20" data-cited="true" seqid="R20" data-doi="10.1073/pnas.1400514111" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24889638"><span class="RefAuthor"><span class="RefSurName">Kov&aacute;cs-Bogd&aacute;n</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Sancak</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Plovanich</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Jambhekar</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Huber</span> <span class="RefGivenName">RJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Myre</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Blower</span> <span class="RefGivenName">MD</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Reconstitution of the mitochondrial calcium uniporter in yeast</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>111</strong></span>:<span class="RefFPage">8985</span>&ndash;<span class="RefLPage">8990</span>.</p><p class="jrnlRefText" id="R21" data-seq-id="R21" sortby="Larkin 2007" data-old-id="R21" data-cited="true" seqid="R21" movetopanel="yes" data-doi="10.1093/bioinformatics/btm404" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17846036"><span class="RefAuthor"><span class="RefSurName">Larkin</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Blackshields</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Brown</span> <span class="RefGivenName">NP</span></span>, <span class="RefAuthor"><span class="RefSurName">Chenna</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">McGettigan</span> <span class="RefGivenName">PA</span></span>, <span class="RefAuthor"><span class="RefSurName">McWilliam</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Valentin</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Wallace</span> <span class="RefGivenName">IM</span></span>, <span class="RefAuthor"><span class="RefSurName">Wilm</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Lopez</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Thompson</span> <span class="RefGivenName">JD</span></span>, <span class="RefAuthor"><span class="RefSurName">Gibson</span> <span class="RefGivenName">TJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Higgins</span> <span class="RefGivenName">DG</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Clustal W and clustal X version 2.0</span>. <span class="RefJournalTitle"><em>Bioinformatics</em></span> <span class="RefVolume"><strong>23</strong></span>:<span class="RefFPage">2947</span>&ndash;<span class="RefLPage">2948</span>.</p><p class="jrnlRefText" id="R22" data-seq-id="R22" sortby="MacKenzie 1984" data-old-id="R22" data-cited="true" seqid="R22" data-doi="10.1021/bi00321a041" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/6529569"><span class="RefAuthor"><span class="RefSurName">MacKenzie</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Arendt</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Hargrave</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">McDowell</span> <span class="RefGivenName">JH</span></span>, <span class="RefAuthor"><span class="RefSurName">Molday</span> <span class="RefGivenName">RS</span></span>. <span class="RefYear">1984</span>. <span class="RefArticleTitle">Localization of binding sites for carboxyl terminal specific anti-rhodopsin monoclonal antibodies using synthetic peptides</span>. <span class="RefJournalTitle"><em>Biochemistry</em></span> <span class="RefVolume"><strong>23</strong></span>:<span class="RefFPage">6544</span>&ndash;<span class="RefLPage">6549</span>.</p><p class="jrnlRefText" id="R23" data-seq-id="R23" sortby="Mallilankaraman 2012" data-old-id="R23" data-cited="true" seqid="R23" data-doi="10.1016/j.cell.2012.10.011" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/23101630" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Mallilankaraman</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Doonan</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">C&aacute;rdenas</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Chandramoorthy</span> <span class="RefGivenName">HC</span></span>, <span class="RefAuthor"><span class="RefSurName">M&uuml;ller</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Miller</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hoffman</span> <span class="RefGivenName">NE</span></span>, <span class="RefAuthor"><span class="RefSurName">Gandhirajan</span> <span class="RefGivenName">RK</span></span>, <span class="RefAuthor"><span class="RefSurName">Molg&oacute;</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Birnbaum</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Rothberg</span> <span class="RefGivenName">BS</span></span>, <span class="RefAuthor"><span class="RefSurName">Mak</span> <span class="RefGivenName">DO</span></span>, <span class="RefAuthor"><span class="RefSurName">Foskett</span> <span class="RefGivenName">JK</span></span>, <span class="RefAuthor"><span class="RefSurName">Madesh</span> <span class="RefGivenName">M</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">MICU1 is an essential gatekeeper for mcu-mediated mitochondrial Ca(2+) uptake that regulates cell survival</span>. <span class="RefJournalTitle"><em>Cell</em></span> <span class="RefVolume"><strong>151</strong></span>:<span class="RefFPage">630</span>&ndash;<span class="RefLPage">644</span>.</p><p class="jrnlRefText" id="R24" data-seq-id="R24" sortby="Murgia 2015" data-old-id="R24" data-cited="true" seqid="R24" data-doi="10.1016/j.ceca.2014.11.001" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/26048007"><span class="RefAuthor"><span class="RefSurName">Murgia</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Rizzuto</span> <span class="RefGivenName">R</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">Molecular diversity and pleiotropic role of the mitochondrial calcium uniporter</span>. <span class="RefJournalTitle"><em>Cell Calcium</em></span> <span class="RefVolume"><strong>58</strong></span>:<span class="RefFPage">11</span>&ndash;<span class="RefLPage">17</span>.</p><p class="jrnlRefText" id="R25" data-seq-id="R25" sortby="Patron 2014" data-old-id="R25" data-cited="true" seqid="R25" data-doi="10.1016/j.molcel.2014.01.013" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24560927"><span class="RefAuthor"><span class="RefSurName">Patron</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Checchetto</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Raffaello</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Teardo</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Vecellio Reane</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Mantoan</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Granatiero</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Szab&ograve;</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">De Stefani</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Rizzuto</span> <span class="RefGivenName">R</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">MICU1 and MICU2 finely tune the mitochondrial ca2+ uniporter by exerting opposite effects on MCU activity</span>. <span class="RefJournalTitle"><em>Molecular Cell</em></span> <span class="RefVolume"><strong>53</strong></span>:<span class="RefFPage">726</span>&ndash;<span class="RefLPage">737</span>.</p><p class="jrnlRefText" id="R26" data-seq-id="R26" sortby="Plovanich 2013" data-old-id="R26" data-cited="true" seqid="R26" data-doi="10.1371/journal.pone.0055785" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/23409044"><span class="RefAuthor"><span class="RefSurName">Plovanich</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Bogorad</span> <span class="RefGivenName">RL</span></span>, <span class="RefAuthor"><span class="RefSurName">Sancak</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Strittmatter</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">AA</span></span>, <span class="RefAuthor"><span class="RefSurName">Girgis</span> <span class="RefGivenName">HS</span></span>, <span class="RefAuthor"><span class="RefSurName">Kuchimanchi</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">De Groot</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Speciner</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Taneja</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Oshea</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Koteliansky</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">MICU2, a paralog of MICU1, resides within the mitochondrial uniporter complex to regulate calcium handling</span>. <span class="RefJournalTitle"><em>PloS One</em></span> <span class="RefVolume"><strong>8</strong></span>:<span class="RefFPage">e55785</span>.</p><p class="jrnlRefText" id="R27" data-seq-id="R27" sortby="Ran 2013" data-old-id="R27" data-cited="true" seqid="R27" data-doi="10.1038/nprot.2013.143" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24157548"><span class="RefAuthor"><span class="RefSurName">Ran</span> <span class="RefGivenName">FA</span></span>, <span class="RefAuthor"><span class="RefSurName">Hsu</span> <span class="RefGivenName">PD</span></span>, <span class="RefAuthor"><span class="RefSurName">Wright</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Agarwala</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Scott</span> <span class="RefGivenName">DA</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">F</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">Genome engineering using the crispr-cas9 system</span>. <span class="RefJournalTitle"><em>Nature Protocols</em></span> <span class="RefVolume"><strong>8</strong></span>:<span class="RefFPage">2281</span>&ndash;<span class="RefLPage">2308</span>.</p><p class="jrnlRefText" id="R28" data-seq-id="R28" sortby="Rizzuto 2012" data-old-id="R28" data-cited="true" seqid="R28" data-doi="10.1038/nrm3412" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22850819"><span class="RefAuthor"><span class="RefSurName">Rizzuto</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">De Stefani</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Raffaello</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Mammucari</span> <span class="RefGivenName">C</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Mitochondria as sensors and regulators of calcium signalling</span>. <span class="RefJournalTitle"><em>Nature Reviews. Molecular Cell Biology</em></span> <span class="RefVolume"><strong>13</strong></span>:<span class="RefFPage">566</span>&ndash;<span class="RefLPage">578</span>.</p><p class="jrnlRefText" id="R29" data-seq-id="R29" sortby="Russ 2000" data-old-id="R29" data-cited="true" seqid="R29" data-doi="10.1006/jmbi.1999.3489" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10677291"><span class="RefAuthor"><span class="RefSurName">Russ</span> <span class="RefGivenName">WP</span></span>, <span class="RefAuthor"><span class="RefSurName">Engelman</span> <span class="RefGivenName">DM</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">The gxxxg motif: A framework for transmembrane helix-helix association</span>. <span class="RefJournalTitle"><em>Journal of Molecular Biology</em></span> <span class="RefVolume"><strong>296</strong></span>:<span class="RefFPage">911</span>&ndash;<span class="RefLPage">919</span>.</p><p class="jrnlRefText" id="R30" data-seq-id="R30" sortby="Sancak 2013" data-old-id="R30" data-cited="true" seqid="R30" movetopanel="yes" data-doi="10.1126/science.1242993" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24231807"><span class="RefAuthor"><span class="RefSurName">Sancak</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Markhard</span> <span class="RefGivenName">AL</span></span>, <span class="RefAuthor"><span class="RefSurName">Kitami</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Kov&aacute;cs-Bogd&aacute;n</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Kamer</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Udeshi</span> <span class="RefGivenName">ND</span></span>, <span class="RefAuthor"><span class="RefSurName">Carr</span> <span class="RefGivenName">SA</span></span>, <span class="RefAuthor"><span class="RefSurName">Chaudhuri</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Clapham</span> <span class="RefGivenName">DE</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">AA</span></span>, <span class="RefAuthor"><span class="RefSurName">Calvo</span> <span class="RefGivenName">SE</span></span>, <span class="RefAuthor"><span class="RefSurName">Goldberger</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">EMRE is an essential component of the mitochondrial calcium uniporter complex</span>. <span class="RefJournalTitle"><em>Science</em></span> <span class="RefVolume"><strong>342</strong></span>:<span class="RefFPage">1379</span>&ndash;<span class="RefLPage">1382</span>.</p><p class="jrnlRefText" id="R31" data-seq-id="R31" sortby="Sharp 1995" data-old-id="R31" data-cited="true" seqid="R31" data-doi="10.1073/pnas.92.17.7946" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/7644518"><span class="RefAuthor"><span class="RefSurName">Sharp</span> <span class="RefGivenName">LL</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhou</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Blair</span> <span class="RefGivenName">DF</span></span>. <span class="RefYear">1995</span>. <span class="RefArticleTitle">Features of mota proton channel structure revealed by tryptophan-scanning mutagenesis</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>92</strong></span>:<span class="RefFPage">7946</span>&ndash;<span class="RefLPage">7950</span>.</p><p class="jrnlRefText" id="R32" data-seq-id="R32" sortby="Vais 2016" data-old-id="R32" data-cited="true" seqid="R32" data-doi="10.1016/j.celrep.2015.12.054" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/26774479" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Vais</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Mallilankaraman</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Mak</span> <span class="RefGivenName">DO</span></span>, <span class="RefAuthor"><span class="RefSurName">Hoff</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Payne</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Tanis</span> <span class="RefGivenName">JE</span></span>, <span class="RefAuthor"><span class="RefSurName">Foskett</span> <span class="RefGivenName">JK</span></span>. <span class="RefYear">2016</span>. <span class="RefArticleTitle">EMRE is a matrix Ca(2+) sensor that governs gatekeeping of the mitochondrial Ca(2+) uniporter</span>. <span class="RefJournalTitle"><em>Cell Reports</em></span> <span class="RefVolume"><strong>14</strong></span>.</p><p class="jrnlRefText" id="R33" data-seq-id="R33" sortby="Waldeck-Weiermair 2015" data-old-id="R33" data-cited="true" seqid="R33" data-doi="10.1038/srep15602" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/26489515" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Waldeck-Weiermair</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Malli</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Parichatikanond</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Gottschalk</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Madreiter-Sokolowski</span> <span class="RefGivenName">CT</span></span>, <span class="RefAuthor"><span class="RefSurName">Klec</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Rost</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Graier</span> <span class="RefGivenName">WF</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">Rearrangement of MICU1 multimers for activation of MCU is solely controlled by cytosolic Ca(2.)</span>. <span class="RefJournalTitle"><em>Scientific Reports</em></span> <span class="RefVolume"><strong>5</strong></span>.</p><p class="jrnlRefText noPMID" id="R34" data-seq-id="R34" sortby="Yamamoto 2016" data-old-id="R34" data-cited="true" seqid="R34" data-doi="10.1016/j.bbabio.2016.03.019"><span class="RefAuthor"><span class="RefSurName">Yamamoto</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Yamagoshi</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Harada</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Kawano</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Minami</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Ido</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Kuwahara</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Fujita</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Ozono</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Watanabe</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Yamada</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Terada</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Shinohara</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear">2016</span>. <span class="RefArticleTitle">Analysis of the structure and function of EMRE in a yeast expression system</span>. <span class="RefJournalTitle"><em>Biochimica Et Biophysica Acta (BBA) - Bioenergetics</em></span>.</p><p class="jrnlRefText" id="R35" data-seq-id="R35" sortby="Ying 1991" data-old-id="R35" data-cited="true" seqid="R35" data-doi="10.1021/bi00234a016" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/2036363"><span class="RefAuthor"><span class="RefSurName">Ying</span> <span class="RefGivenName">WL</span></span>, <span class="RefAuthor"><span class="RefSurName">Emerson</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Clarke</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Sanadi</span> <span class="RefGivenName">DR</span></span>. <span class="RefYear">1991</span>. <span class="RefArticleTitle">Inhibition of mitochondrial calcium ion transport by an oxo-bridged dinuclear ruthenium ammine complex</span>. <span class="RefJournalTitle"><em>Biochemistry</em></span> <span class="RefVolume"><strong>30</strong></span>:<span class="RefFPage">4949</span>&ndash;<span class="RefLPage">4952</span>.</p><div class="jrnlSupplBlock" id="BLK_SP1" data-stream-name="a_SP1"><p class="jrnlSupplCaption" id="SP1"><span class="label"><span class="authorQueryNode Author" id="1462617320310" rid="QRY_1462617320310" data-username="Sarumathi (eLife Sciences Publications, Ltd)" data-time="05/7/2016 4:05pm">Supplementary file 1.</span></span>&nbsp;Supplementary&nbsp;experimental&nbsp;procedure.</p><p><a download="Supplementary_file_1" href="/ecs/article/82842/resources/Supplementary_file_1.docx" data-href="/ecs/article/82842/resources/Supplementary_file_1.docx" data-mce-href="ecs/article/82842/resources/Supplementary_file_1.docx">Supplementary_file_1</a>
</p></div><div class="sub-article" data-editable="true" article-type="article-commentary" id="sa1"><h1 class="sub-article-title">Decision letter</h1><div class="sub-body"><div class="jrnlBoxBlock"><p class="jrnlBoxText">In the interests of transparency, eLife includes the editorial decision letter and accompanying author responses. A lightly edited version of the letter sent to the authors after peer review is shown, indicating the most substantive concerns; minor comments are not usually included.</p></div><p>Thank you for submitting your article "A Central Role of EMRE in Regulation of the Mitochondrial Uniporter Complex" for consideration by <i>eLife</i>. Your article has been favorably evaluated by Gary Westbrook (Senior Editor) and three reviewers, one of whom, David Clapham, is a member of our Board of Reviewing Editors. The reviewers discussed the reviews with one another and the Senior Editor has drafted this decision to help you prepare a revised submission.</p><p>As you will see, all reviewers were enthusiastic about the quality and relevance of the work, and its publication after revision in <i>eLife</i>. Reviewers 2 and 3 had very minor points as listed below. Reviewer 2 made some extensive comments, but note these were considered "suggestions". We ask that you consider these points in your revised manuscript.</p><p>Minor points from Reviewers 2 and 3:</p><p>1) A new paper in BBA reports functional mutagenesis of EMRE, showing that deletion of amino acids at EMRE's N terminus abolishes function, in conflict with the current paper. The discrepancy ought to be reconciled or at least discussed.</p><p>2) <span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B</span>: Western blot: what was the molecular weight for EMRE?</p><p>3) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5C and D</span>: The predicted mol. wt. for MICU1 and MICU2 are approximately 54 and 50&thinsp;KDa respectively. The observed bands in Western blots are near 64&thinsp;KDa for both. How do you explain the difference? You also mentioned that MICU1 antibody was not good enough to quantify the MICU1 expression level in MICU1 knockdown cell lines. A comment on reliability of your MICU1 antibodies may be needed.</p><p>4) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5B</span> indicates that MICU2 remains membrane-bound even after harsher Na2CO3 treatment where as MICU1 comes off. This result seems to contradict the reports indicating that MICU2 binds MCU-complex via MICU1 either by forming a disulfide bond or other (Patron et al, 2014 as well as Kamer et al, 2014). This result was not discussed at all considering its importance.</p><p>5) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span> shows that MICU1 can precipitate MCU in the absence of EMRE. Doesn't this observation contradict the strict requirement of EMRE C-terminal for MICU1 engagement with the pore-forming subunit (MCU)?</p><p>Reviewer #1 general assessment:</p><p>Tsai et al. investigate EMRE, a non- pore-forming subunit of the mitochondrial uniporter, (the MCU, or MiCa channel). They approach the relevant orientation and functional questions with a large number of experimental methods. First, the authors use gene editing to develop assays in permeabilized HEK cells. MCU homologs from amoeba and plants mediate Ca uptake alone, while worms, flies, and human MCU requires EMRE. Cysteine and tryptophan scanning/substitution mutagenesis then enables them to verify that MCU's N and C termini are inside the mitoplast, while EMRE's N terminus is inside the mitoplast and its C terminus outside. Chimeras and other experiments then suggest that TMH1 of MCU is the interaction region for EMRE. Other subunits, the relatively soluble subunits MICU1 and MICU2, are shown to associate with the outer leaflet of the inner mitochondrial membrane, and that EMRE associates with MICU1 by electrostatic interactions. MICU function is verified as regulating Ca entry, and the authors propose that MICU1 must bind EMRE to engage the MCU pore. Thus, MICUs regulate what would otherwise be an unregulated Ca pore. An important finding is that EMRE polyaspartate tail ensures that the channel conducts Ca only when cytoplasmic Ca rises above its resting levels. In short, EMRE mediates MICU retention and MCU retention. Based on the lack of EMRE in lower species, the authors conclude that EMRE arose as a safety mechanism, allowing MICU1 gating of MCU only if EMRE is present.</p><p>This work clarifies many of the issues that have plagued MCU subunit function and orientation. The methods are robust and the conclusions are likely sound. The only additional experiments the field needs to settle remaining conformational questions is the high resolution structure of the MCU/EMRE/MICU1/2/MCUR1 complex.</p><p><i>Reviewer #2 general assessment and major comments:</i>
</p><p>This is a new paper from a premiere ion channel biochemistry lab. The paper focuses on the macromolecular organization of the mitochondrial calcium uniporter complex, which in humans consists of five proteins: MCU (the pore), MICU1/2 (calcium sensors in the IMS), and EMRE (a small membrane protein that relays MICU1/2 calcium sensing to the open state of MCU). The study combines clever biochemical methods (e.g., artificial transmembrane domains, tryptophan mutagenesis, and protein fusions) to shed insight into the complex, with a focus on EMRE and its interactions with MCU and MICU1. The topic is timely. Some important suggestions designed to help the authors improve what could be a very important paper:</p><p>1) This paper reports a mix of novel and not so novel results &ndash; hence novel findings are buried. I would encourage the authors to emphasize the novel and de-emphasize results that are confirmatory. For example, the topology of MCU, IMS localization of MICU1/2, gatekeeping activity by MICU1, and dual functions of EMRE (keeping MCU open and linking MICU1/2 to MCU) are all well-established now. The novel findings include the topology of EMRE, the MCU-EMRE fusion (which could be a very important tool for the field), EMRE interaction with TM1, and interaction between MICU1 and EMRE.</p><p>2) In MCU knockout cell lines and mouse tissues, loss of MCU causes loss of EMRE expression. In many figures, the authors claim loss of interaction (by IP) between EMRE and MCU, but EMRE protein expression seems normal. This raises the possibility that the IP conditions used here do not reflect the in vivo binding status of these two proteins. For example, in <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3F</span>, the authors mutate EMRE and evaluate which variants are able to interact with MCU. The S85W mutant evidently expresses very well, yet does not interact with MCU. Can the authors provide additional evidence, e.g., BN-PAGE, demonstrating a lack of a shift of the MCU holocomplex?</p><p>3) The authors claim that the &delta;-N, &delta;-C and WALP EMRE localize to the mitochondria. However, it's not clear how pure their mitochondrial preps are without markers for other organelles. Alternatively, they could consider simple microscopy experiments to look for co-localization with mitochondrial markers. Again, it is surprising to see that WALP EMRE expresses so well, even though it does not bind to MCU. How do the activities of &delta;-N and &delta;-C compare to WT? Please add WT EMRE traces and expression to <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B and C</span>.</p><p>4) One of the important new claims relates to the nature of the interaction between MCU and EMRE. The authors have performed tryptophan scanning mutagenesis of both transmembranes of MCU and identify six in TM1 versus two in TM2 that disrupt function. Based on this difference (six versus two), the authors prioritize TM1 as being more important for EMRE binding. They then show that two of these mutations can be introduced into the DdMCU without impairing function &ndash; and since DdMCU operates independently of EMRE, prioritize these two as important for human MCU binding to EMRE. Finally, the authors perform an amino acid swap between MCU and EMRE to support the conclusion that A241 in MCU and F77 in EMRE directly interact.</p><p>I have some concerns about the logic and inference. First, it's not clear how the authors can exclude the possibility that the other four residues on TM1, or the other two residues on TM2, are important for EMRE binding. Second, if this model is correct, then the MCU chimera with TM1 from human, and TM2 from <i>C. elegans</i>, ought in principle be activated by human EMRE, yet it is not. Third, in these functional studies shown in <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1</span>, IPs are not performed, so we do not know if EMRE is actually bound or not bound to MCU. The authors ought to review these data, add the necessary controls, and then state their conclusions carefully. Also, the authors cannot claim direct interaction &ndash; this is nuanced in the text, but in the figure legends, direct interaction is claimed.</p><p>5) The co-IP data presented in <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span> suggests that MCU can interact with MICU1 independent of EMRE, and that MICU1 can interact with EMRE independent of MCU. These experiments lack proper controls. The authors could express another mitochondrial IM protein as a negative control.</p><p><i>Reviewer #3 general assessment:</i>
</p><p>This is an extremely important paper that clarifies the general architecture of the MCU complex and sheds light on the physiological function of EMRE and MICU1. The experiments are carefully planned and conducted. The paper is essentially ready for publication.</p></div></div><div class="sub-article" data-editable="true" article-type="reply" id="sa2"><h1 class="sub-article-title">Author response</h1><div class="sub-body"><p><i>Reviewer #2 major comments:</i>
</p><p><i>[&hellip;] Some important suggestions designed to help the authors improve what could be a very important paper:</i>
</p><p><i>1) This paper reports a mix of novel and not so novel results &ndash; hence novel findings are buried. I would encourage the authors to emphasize the novel and de-emphasize results that are confirmatory. For example, the topology of MCU, IMS localization of MICU1/2, gatekeeping activity by MICU1, and dual functions of EMRE (keeping MCU open and linking MICU1/2 to MCU) are all well-established now. The novel findings include the topology of EMRE, the MCU-EMRE fusion (which could be a very important tool for the field), EMRE interaction with TM1, and interaction between MICU1 and EMRE.</i>
</p><p>We agree that the topology of MCU, IMS localization of MICU1/2, and the gatekeeping function of MICU1 have been established, and we have indeed cited appropriate references in the manuscript. However, we disagree with the reviewer that the dual function of EMRE is established. Prior to our work, the only known function of EMRE is keeping MCU open, but the physiological significance of such function is unclear. To be sure, it has been reported that EMRE interacts with MICU1, but the purpose and mechanism of this interaction, and how this molecular contact regulates uniporter function, have remained unknown until this study. We have endeavored in the Abstract, the Introduction, and the Discussion to emphasize and highlight the major contribution of this work: to provide a unifying picture of how the MCU-activation and MICU1-rentention functions of EMRE together prevent Ca<sup>2+</sup> leakage through the uniporter during resting cell conditions.</p><p><i>2) In MCU knockout cell lines and mouse tissues, loss of MCU causes loss of EMRE expression. In many figures, the authors claim loss of interaction (by IP) between EMRE and MCU, but EMRE protein expression seems normal. This raises the possibility that the IP conditions used here do not reflect the in vivo binding status of these two proteins. For example, in <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3F</span>, the authors mutate EMRE and evaluate which variants are able to interact with MCU. The S85W mutant evidently expresses very well, yet does not interact with MCU. Can the authors provide additional evidence, e.g., BN-PAGE, demonstrating a lack of a shift of the MCU holocomplex?</i>
</p><p>There is so far no evidence that the steady-state expression level of EMRE directly reflects the integrity of the MCU-EMRE interaction. Protein expression is a complicated function of the rate of proteolysis, polypeptide synthesis, protein insertion into membranes, etc., a subject that we have currently under study, and one far beyond the scope of this manuscript. An EMRE mutation such as S85W can in principle influence any of these factors and can thus alter EMRE&rsquo;s observed expression level.</p><p>We have in fact done proper controls for the CoIP experiment, showing that MCU pulls down WT and control Trp mutants, but not G81W and S85W EMRE. BN-PAGE would not provide additional information, since as in CoIP, it requires the protein complex to be first extracted into detergents before being analyzed by Western blot.</p><p><i>3) The authors claim that the &delta;-N, &delta;-C and WALP EMRE localize to the mitochondria. However, it's not clear how pure their mitochondrial preps are without markers for other organelles. Alternatively, they could consider simple microscopy experiments to look for co-localization with mitochondrial markers. Again, it is surprising to see that WALP EMRE expresses so well, even though it does not bind to MCU. How do the activities of &delta;-N and &delta;-C compare to WT? Please add WT EMRE traces and expression to <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B and C</span>.</i>
</p><p>In <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B&ndash;C</span>, we show that &Delta;N and &Delta;C EMRE, but not WALP, support MCU function. There is certainly a low possibility that WALP EMRE fails to activate MCU because it is targeted to other organelles, despite its mitochondrial targeting sequence. This possibility could be ruled out by extensive control experiments, such as the co-localization experiment suggested by the reviewer. However, we cannot justify the time required for these experiments, as at the end of the day it is the extensive Trp mutagenesis screening (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3D&ndash;F</span>) and the side-chain swap experiments (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4A&ndash;B</span>) that powerfully establish that TMH interactions between MCU and EMRE are necessary for uniporter function.</p><p>The activity of &Delta;N, &Delta;C, and WT EMRE are similar, as is compared in <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6E</span> using a more sensitive <sup>45</sup>Ca<sup>2+</sup> flux assay. Again, there is no evidence in the literature that EMRE steady-state expression is controlled solely by MCU binding.</p><p><i>4) One of the important new claims relates to the nature of the interaction between MCU and EMRE. The authors have performed tryptophan scanning mutagenesis of both transmembranes of MCU and identify six in TM1 versus two in TM2 that disrupt function. Based on this difference (six versus two), the authors prioritize TM1 as being more important for EMRE binding. They then show that two of these mutations can be introduced into the DdMCU without impairing function &ndash; and since DdMCU operates independently of EMRE, prioritize these two as important for human MCU binding to EMRE. Finally, the authors perform an amino acid swap between MCU and EMRE to support the conclusion that A241 in MCU and F77 in EMRE directly interact.</i>
</p><p><i>I have some concerns about the logic and inference. First, it's not clear how the authors can exclude the possibility that the other four residues on TM1, or the other two residues on TM2, are important for EMRE binding. Second, if this model is correct, then the MCU chimera with TM1 from human, and TM2 from C. elegans, ought in principle be activated by human EMRE, yet it is not. Third, in these functional studies shown in <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1</span>, IPs are not performed, so we do not know if EMRE is actually bound or not bound to MCU. The authors ought to review these data, add the necessary controls, and then state their conclusions carefully. Also, the authors cannot claim direct interaction &ndash; this is nuanced in the text, but in the figure legends, direct interaction is claimed.</i>
</p><p>Our side-chain swap experiment allows us to identify A241 as a residue interacting with EMRE, but we certainly do not exclude the possibility that other residues sensitive to Trp substitution might also be important for EMRE binding. The reviewer is right that our model would predict that an MCU chimera made of human-MCU TM1 and <i>C. elegans</i> MCU TM2 should respond to human EMRE. Unfortunately, such chimera fails to respond to both human and <i>C. elegans</i> EMRE. We cannot interpret a negative result like this, as the failure to detect function could result from many potential issues, such as misfolding of the chimera protein.</p><p><i>5) The co-IP data presented in <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span> suggests that MCU can interact with MICU1 independent of EMRE, and that MICU1 can interact with EMRE independent of MCU. These experiments lack proper controls. The authors could express another mitochondrial IM protein as a negative control.</i>
</p><p>The reviewer, citing precedents in the literature, suggests a common way to rule out non-specific binding of MICU1 to the affinity column: to show that another mitochondrial IM protein fails to bind to the column. However, we consider this kind of negative control to be a bit indirect. Instead, we opted for a more laborious but more powerful negative control that directly demonstrates that MICU1 is not retained in the column in the absence of MCU or EMRE (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span>).</p><p><i>Minor points from Reviewers 2 and 3:</i>
</p><p><i>1) A new paper in BBA reports functional mutagenesis of EMRE, showing that deletion of amino acids at EMRE's N terminus abolishes function, in conflict with the current paper. The discrepancy ought to be reconciled or at least discussed.</i>
</p><p>Yes, we saw this new paper when it appeared. It fully confirms our orientation results. There is in fact no discrepancy, as their substitutions and short deletion at the N-terminus were different from our more extensive deletion. We comment on these new results in the Discussion.</p><p><i>2) <span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B</span>: Western blot: what was the molecular weight for EMRE?</i><br/><br/>
</p><p>It is ~10&thinsp;kDa. We have added this information to the legends.</p><p><i>3) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5C and D</span>: The predicted mol. wt. for MICU1 and MICU2 are approximately 54 and 50&thinsp;KDa respectively. The observed bands in Western blots are near 64&thinsp;KDa for both. How do you explain the difference? You also mentioned that MICU1 antibody was not good enough to quantify the MICU1 expression level in MICU1 knockdown cell lines. A comment on reliability of your MICU1 antibodies may be needed.</i>
</p><p>We do not know why the MICUs travel more slowly than expected. Deviation from the expected ICU1 immunodetection: this is reliable, since MICU1 tagged with FLAG, and detected by anti-FLAG antibody. We have now revised the methods section to make sure that this is clear.</p><p><i>4) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5B</span> indicates that MICU2 remains membrane-bound even after harsher Na2CO3 treatment where as MICU1 comes off. This result seems to contradict the reports indicating that MICU2 binds MCU-complex via MICU1 either by forming a disulfide bond or other (Patron et al, 2014 as well as Kamer et al, 2014). This result was not discussed at all considering its importance.</i><br/><br/>
</p><p>In our Na<sub>2</sub>CO<sub>3</sub> extraction experiments, either MICU1 or MICU2 is overexpressed so that one of them is in large molecular excess than the other. This allows us to test interaction of individual MICU with lipid, with minimized complication from MICU1-MICU2 interaction. Our results agree with the idea that MICU1 and MICU2 form a heterodimer, as in the Patron and Kamer paper.</p><p><i>5) <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5E</span> shows that MICU1 can precipitate MCU in the absence of EMRE. Doesn't this observation contradict the strict requirement of EMRE C-terminal for MICU1 engagement with the pore-forming subunit (MCU)?</i>
</p><p>This is a good point but not a fatal one for our argument. Our results show that EMRE C-terminus is necessary for MCU to be &ldquo;fully&rdquo; gated by MICU1, but we did not rule out that without this interaction, there could be some MCU-MICU1 interaction that allows some binding and perhaps a less perfectly coupled form of uniporter gating. We actually think, as in Discussion, that this MCU-MICU1 interaction might be crucial for protist and plant uniporters, which do not have EMRE as a subunit.</p></div></div></div></div></div> <div id="queryDivNode"><div class="panel-body"><span queried-on="05/18/16 9:53am" queried-by="Alistair (eLife Sciences Publications, Ltd)" query-type="Author" id="QRY_1463561630901" class="queryText">Could you please add the missing text at the end of this paragraph: "HEK-293 cells, the source for all mitochondria used here, were supplied by Dr. D.E. Clapham and were authenticated by *** using the ***method and were confirmed to be mycoplasma-free by *** test."</span><span class="col-md-9 col-xs-9"><span queried-on="05/18/2016 9:13am" queried-by="cmiller@brandeis.edu (eLife Sciences Publications, Ltd)" class="authorReplyText col-xs-10" placeholder="add your reply here..." style="margin-top: 13px;">OK, text now added</span></span></div><div class="panel-body"><span queried-on="04/24/16 2:41pm" queried-by="Sarumathi (eLife Sciences Publications, Ltd)" query-type="Author" id="QRY_1461489068502" class="queryText">Please provide the manufacturer location (city and state [if USA], or country [if not USA]) for the manufacturer &ldquo;ClonTech&rdquo;. In addition, please check that these details are provided on the first mention of any other manufacturers cited in the article.</span><span class="col-xs-12" style="padding-top: 12px;"><span queried-on="05/17/2016 9:27am" queried-by="cmiller@brandeis.edu (eLife Sciences Publications, Ltd)" class="authorReplyText col-xs-10" placeholder="add your reply here..." contenteditable="true">Mountain View, CA</span></span></div><div class="panel-body"><span queried-on="05/07/16 4:05pm" queried-by="Sarumathi (eLife Sciences Publications, Ltd)" query-type="Author" id="QRY_1462617320310" class="queryText">Please provide the title, legend and image file for Figure &ldquo;<span class="query_x_label">XXX</span>&rdquo;</span><span class="col-xs-12" style="padding-top: 12px;"><span queried-on="05/17/2016 10:07am" queried-by="cmiller@brandeis.edu (eLife Sciences Publications, Ltd)" class="authorReplyText col-xs-10" placeholder="add your reply here..." contenteditable="true">The title should be "supplementary experimental procedure."</span></span></div></div>