<div class="WordSection1" style="page: WordSection1;" id="contentDivNode"><div class="front" data-edited="true"><p class="jrnlDOI">10.7554/eLife.08384</p><p class="jrnlArtType">Research Article</p><h1 class="jrnlArtTitle">Signal transducer and activator of transcription 5 (STAT5) paralog dose governs T cell effector and regulatory functions</h1><div data-editable="true" construct="true" wrapelement="div" class="jrnlAbsGroup"><h1 class="jrnlAbsHead">Abstract</h1><p class="jrnlAbsPara">The transcription factor STAT5 is fundamental to the mammalian immune system. However, the relationship between its two paralogs, STAT5A and STAT5B, and the extent to which they are functionally distinct, remain uncertain. Using mouse models of paralog deficiency, we demonstrate that they are not equivalent for CD4<sup>+</sup> 'helper' T cells, the principal orchestrators of adaptive immunity. Instead, we find that STAT5B is dominant for both effector and regulatory (Treg) responses and, therefore, uniquely necessary for immunological tolerance. Comparative analysis of genomic distribution and transcriptomic output confirm that STAT5B has fargreater impact but, surprisingly, the data point towards asymmetric expression (i.e. paralog dose), rather than distinct functional properties, as the key distinguishing feature. Thus, we propose a quantitative model of STAT5 paralog activity whereby relative abundance imposes functional specificity (or dominance) in the face of widespread structural homology.</p></div><div class="jrnlBoxBlock" box-type="elife-digest" data-editable="true"><h1 class="jrnlBoxCaption">eLife digest</h1><p class="jrnlBoxText">The immune system in mammals is one of the most complex networks in the animal kingdom. One way that its many components communicate is via proteins called cytokines, which are released by cells and detected by receptors on the surface of other cells. This leads to the activation of signals inside the responding cells that alter the activity of genes and, ultimately, direct how they behave.</p><p class="jrnlBoxText">STAT5 is a signal protein that is activated when certain cytokines bind to receptors on the cell surface. Consequently, it is an attractive target for drug therapies that seek to alter immune responses and there is keen interest in understanding how it works. It is an unusual protein in that there are two versions &ndash; termed STAT5A and STAT5B &ndash; that are produced by two separate genes. Together, STAT5A and STAT5B are fundamental to the immune system but there is considerable debate about whether they perform the same job or have distinct roles.</p><p class="jrnlBoxText">Villarino et al. used a combination of genetic and genomic approaches to investigate how both versions of STAT5 work in mice. The experiments show that STAT5B plays a much bigger role in immune cells than STAT5A. Unexpectedly, the experiments indicate that the disparity is not due to differences in protein activity, but is caused by differences in the amount of these proteins in cells.</p><p class="jrnlBoxText">Villarino et al.&rsquo;s findings resolve longstanding questions about the relationship between STAT5A and STAT5B within the immune system. A logical next step is to find the molecular mechanisms responsible for causing different amounts of STAT5A and STAT5B to be produced in immune cells. Future work will also compare the roles of STAT5A and STAT5B in non-immune cells and explore whether it might be possible to develop therapies that specifically target one version and not the other.</p></div></div><div class="body" data-edited="true"><h1 class="jrnlHead1" bold="true" patternhead="true">Introduction</h1><p class="jrnlSecPara">Signal transducers and activators of transcription (STAT) family proteins are an evolutionarily conserved set of transcription factors, which operate downstream of cytokine and hormone receptors to convert extracellular stimuli into biochemical signals that instruct gene expression (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R66 ">Villarino et al., 2015</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R60 ">Stark and Darnell, 2012</span>). In mammals, STAT5 is unique because it is encoded by two genes, termed <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i>, derived from a relatively recent duplication event (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R67 ">Wang and Levy, 2012</span>). In fact, the ancestral STAT5 gene appears to have duplicated on two separate occasions during vertebrate evolution. Once in teleosts, resulting in two paralogs on different chromosomes, and again in eutherians, resulting in the two contiguous mammalian paralogs (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R40 ">Liongue et al., 2012</span>).</p><p class="jrnlSecPara">Because of their recent divergence, STAT5A and STAT5B are homologous at the DNA, RNA and protein levels, which has led to persisting questions about whether they are redundant or functionally distinct. Genetically engineered mice lacking <i fontsize="11.0pt">Stat5a</i> or <i fontsize="11.0pt">Stat5b</i> have provided compelling evidence for both arguments. On one hand, there are phenotypic differences; <i fontsize="11.0pt">Stat5a</i>-deficient mice exhibit poor mammary function (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R42 ">Liu et al., 1997</span>), reduced hematopoietic stem cell proliferation (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R74 ">Zhang et al., 2000</span>) and diminished antibody class switching (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R24 ">Kagami et al., 2000</span>), while <i fontsize="11.0pt">Stat5b</i>-deficient mice exhibit dwarfism (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R64 ">Udy et al., 1997</span>), more pronounced lymphopenia, and greater defects in cytokine-driven lymphocyte proliferation (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R48 ">Moriggl et al., 1999</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R21 ">Imada et al., 1998</span>). On the other hand, deletion of <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> has comparable effects on some physiological processes, such as eosinophil recruitment (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R24 ">Kagami et al., 2000</span>), and the most dramatic phenotypes, such as infertility, anemia and perinatal lethality, are evident only in mice lacking both paralogs, which implies redundancy and/or cooperativity (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R63 ">Teglund et al., 1998</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R58 ">Socolovsky et al., 1999</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R10 ">Cui et al., 2004</span>). Genome-wide DNA-binding profiles also support both viewpoints. The target repertoires for STAT5A and STAT5B mostly overlap, which implies redundancy, but there are also a subset of sites that may be differentially bound, which implies specificity (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R37 ">Liao et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">2011</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R69 ">Yamaji et al., 2013</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R27 ">Kanai et al., 2014</span>). Consistent with the latter point, humans with germline mutations in <i fontsize="11.0pt">STAT5B</i> exhibit a range of clinical abnormalities, indicating that <i fontsize="11.0pt">STAT5A</i> cannot compensate for some vital functions (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R26 ">Kanai et al., 2012</span>).</p><p class="jrnlSecPara">Compound STAT5 deficiency manifests striking immunological abnormalities in mice, most notably lymphopenia, splenomegaly and autoimmunity. These are typically attributed to its role downstream of the common gamma chain (&#611;c) receptor and its dedicated Janus kinase, Jak3 (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R49 ">Moriggl et al., 1999</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R57 ">Snow et al., 2003</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R71 ">Yao et al., 2006</span>). The &#611;c is shared by 6 different cytokines, IL-2 IL-4, IL-7, IL-9, IL-15 and IL-21, each of which employs a unique co-receptor subunit that determines which cell types can respond (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R55 ">Rochman et al., 2009</span>). &#611;c cytokines impact all lymphocytes but have been most extensively studied in CD4<sup fontsize="11.0pt">+</sup> 'helper' T cells, the key orchestrators of adaptive immunity. Among the many functions ascribed to the &#611;c-STAT5 axis in this lineage are the ability to promote Th1- and Th2-type effector responses, to support T cell memory, to promote activation-induced cell death, to suppress Th17-type and T follicular helper cell (Tfh) responses, and to promote T regulatory cell (Treg) responses (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R48 ">Moriggl, Topham, et al., 1999</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R37 ">Liao et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">2011</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R11 ">Dooms et al., 2007</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R78 ">Zhu et al., 2003</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R25 ">Kagami et al., 2001</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R36 ">Lenardo, 1991</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R35 ">Laurence et al., 2007</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R2 ">Ballesteros-Tato et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R23 ">Johnston et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R44 ">Mahmud et al., 2013</span>).</p><p class="jrnlSecPara">To assess redundancy between STAT5 paralogs, we developed a mouse model where STAT5A and/or STAT5B were reduced but not absent, allowing us to compare their respective functions while avoiding the confounding lymphopenia associated with complete STAT5 deficiency. These studies reveal STAT5B as the dominant paralog in helper T cells; exhibiting far greater impact on pathogenic effector and host-protective regulatory responses and, therefore, uniquely required for immunological tolerance. Surprisingly, genome-wide DNA binding and transcriptome surveys did not uncover widespread differences in target gene selection but, instead, point towards relative abundance as the key distinguishing factor. Thus, we propose that asymmetric expression (i.e. paralog dose), rather than differential function, determines the dominant STAT5 paralog in lymphoid cells.</p><h1 class="jrnlHead1" bold="true" patternhead="true">Results</h1><h2 class="jrnlHead2" italic="true" headweightage="19">A dominant role for STAT5B in immunological tolerance</h2><p class="jrnlSecPara">To investigate the relationship between STAT5A and STAT5B, we generated a series of mice with pre-determined combinations of <i fontsize="11.0pt">Stat5</i> alleles, ranging from two alleles each of A and B (4 total) to one allele of either A or B (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1A</span>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R69 ">Yamaji et al., 2013</span>). We refer to each genotype according to the total number of <i fontsize="11.0pt">Stat5</i> alleles that are retained. For example, two-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice lack both <i fontsize="11.0pt">Stat5a</i> alleles but retain two of <i fontsize="11.0pt">Stat5b (Stat5a<sup>-/-</sup> Stat5b<sup>+/+</sup></i>), while one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice lack both <i fontsize="11.0pt">Stat5a</i> alleles and retain just one of <i fontsize="11.0pt">Stat5b (Stat5a<sup>-/-</sup> Stat5b<sup>+/-</sup></i>). All 8 genotypes were born at the expected Mendelian ratios and survived beyond 6 months of age, thereby demonstrating that a single allele of either paralog is sufficient to prevent the perinatal lethality seen in STAT5-null mice (<b fontsize="11.0pt">Data not shown</b>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R20 ">Hoelbl et al., 2006</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R10 ">Cui et al., 2004</span>). Red blood cell counts and hematocrits were comparable across all genotypes, indicating that a single allele is also enough to support erythropoiesis, but white blood cell (WBC) counts were sharply reduced in one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice, as well as two-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice. By contrast, two-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice had relatively normal WBC counts (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1B</span>).</p><div class="jrnlFigBlock" id="BLK_F1" data-stream-name="a_F1" data-page-num="3" data-float-placement="TopDoubleColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F1"><span class="label">Figure 1.</span> Stat5b is required for immunological tolerance.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Cartoon depicts the mutant mice used in this study. Genotypes are grouped according to total <i fontsize="11.0pt">Stat5</i> alleles. (<span class="partLabel">B</span>) Bar graphs show averaged RBC, hematocrit and WBC counts. (<span class="partLabel">C</span>) Scatter plot shows kidney pathology scores. (<span class="partLabel">D</span>) Scatter plot shows urinary albumin/creatinine protein ratios. (<span class="partLabel">E</span>) Bar graph shows ELISA measurements (O.D.) for anti-double stranded DNA antibodies in serum. (<span class="partLabel">F</span>) Micrographs show representative H &amp; E kidney sections (40X magnification). (<span class="partLabel">B&ndash;E</span>) Number of <i fontsize="11.0pt">Stat5a, Stat5b</i> and total <i fontsize="11.0pt">Stat5</i> alleles (i.e. genotype) is explained in the key below each graph. Data are compiled from 3&ndash;5 mice per genotype. Error bars indicate standard deviation.</p><img src="/ecs/article/80712/resources/elife08384f001.jpg" data-mce-src="/ecs/article/80712/resources/elife08384f001.jpg" data-primary-extension="tif" data-width="1502" data-height="2220" data-id="1943632" alt="ecs/article/80712/resources/elife08384f001.jpg" style="height: 685px;" height="685" width="463"/></div><p class="jrnlSecPara">None of the STAT5 mutants exhibited histological abnormalities in the liver, spleen or intestine, tissues known to be affected in STAT5-null mice (<b fontsize="11.0pt">Data not shown</b>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R57 ">Snow et al., 2003</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R71 ">Yao et al., 2006</span>). However, <i fontsize="11.0pt">Stat5b</i>-deficient mice did exhibit kidney pathology with a penetrance of 75% or 25%, depending on whether they harbored one or two <i fontsize="11.0pt">Stat5a</i> alleles (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>). Afflicted individuals presented a loss of glomerular structure, proteinuria and systemic anti-DNA antibodies (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D&ndash;F</span>). Therefore, as in humans, <i fontsize="11.0pt">Stat5b</i> is required for immunological tolerance in mice but, given the clear difference between having one or two <i fontsize="11.0pt">Stat5a </i>alleles, redundancy and/or cooperativity is also evident.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">STAT5 paralog dose governs T follicular helper cell responses</h2><p class="jrnlSecPara">To probe for immunological phenotypes, we first assessed the cellularity and composition of primary lymphoid organs. Although not completely lymphopenic like STAT5-null mice (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R71 ">Yao et al., 2006</span>), one- and two-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice did have fewer splenocytes than WT controls (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2A</span>). Cell counts were also reduced in one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice, suggesting that, while STAT5B may be dominant, STAT5A does have substantial influence. Lymph node cellularity was similarly affected by the loss of either paralog and, in fact, all genotypes with less than three-alleles had reduced cell counts (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2A</span>).</p><div class="jrnlFigBlock" id="BLK_F2" data-stream-name="a_F2" data-page-num="4" data-float-placement="BottomMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F2"><span class="label">Figure 2.</span> Impact of STAT5 paralog deficiency on B and T cells.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Bar graphs show averaged cell counts for spleens (top row) and lymph nodes (bottom row). Error bars indicate standard deviation. (<span class="partLabel">B</span>) Percentages of CD4<sup fontsize="11.0pt">+</sup> T cells (CD3<sup fontsize="11.0pt">+</sup> CD4<sup fontsize="11.0pt">+</sup> CD8&alpha;<sup fontsize="11.0pt">-</sup>), CD8<sup fontsize="11.0pt">+</sup> T cells (CD3<sup fontsize="11.0pt">+</sup> CD4<sup fontsize="11.0pt">-</sup> CD8&alpha;<sup fontsize="11.0pt">+</sup>) and B cells (CD3<sup fontsize="11.0pt">-</sup> B220<sup fontsize="11.0pt">+</sup>) were measured in spleens (top row) and LNs (bottom row). Box plots show log2 fold changes relative to wild type controls (WT=0; not shown). Dotted red lines indicate two-fold changes. (<span class="partLabel">A&ndash;B</span>) Number of <i fontsize="11.0pt">Stat5a, Stat5b</i> and total <i fontsize="11.0pt">Stat5</i> alleles is explained in the key below each graph. Data are compiled from 5 experiments. (<span class="partLabel">C</span>) Contour plots show percentages of GL7<sup fontsize="11.0pt">+</sup> Fas<sup fontsize="11.0pt">+ </sup>germinal center B cells in lymph nodes. Scatter plot shows percentages of LN resident GC B cells compiled from 3 experiments (3&ndash;4 mice per group). Genotypes are ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1E</span> (WT mice = white, one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice = blue, one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice = orange).</p><img src="/ecs/article/80712/resources/elife08384f002.jpg" data-mce-src="/ecs/article/80712/resources/elife08384f002.jpg" data-primary-extension="tif" data-width="2008" data-height="1389" data-id="1939362" alt="ecs/article/80712/resources/elife08384f002.jpg" height="507" width="733"/></div><p class="jrnlSecPara">Frequencies of CD4<sup fontsize="11.0pt">+</sup> T cells were comparable across all genotypes, whereas CD8<sup fontsize="11.0pt">+</sup> T cells were reduced in one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice and, to a lesser extent, in one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B</span>). By contrast, B cells were increased in one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice and, consistent with the appearance of auto-antibodies, GL7<sup fontsize="11.0pt">+</sup> Fas<sup fontsize="11.0pt">high</sup> IgD<sup fontsize="11.0pt">low</sup> germinal center (GC) B cells were dramatically enriched (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B&ndash;C</span> and<b fontsize="11.0pt"> data not shown</b>). One-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice had a more modest accumulation of GC B cells, again, illustrating both the relevance and redundancy of STAT5A (<span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2B&ndash;C</span>).</p><p class="jrnlSecPara">The ability to promote B cell responses is a defining characteristic of CD4<sup fontsize="11.0pt">+</sup> &lsquo;helper&rsquo; T cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R9 ">Crotty, 2011</span>). Therefore, given the appearance of GC B cells, we next investigated the CD4<sup fontsize="11.0pt">+</sup> T cell compartment. Not surprisingly, there was a marked accumulation of CD44<sup fontsize="11.0pt">high</sup> IL-7R&alpha;<sup fontsize="11.0pt">low</sup> effector/memory T cells in <i fontsize="11.0pt">Stat5b</i>-deficient mice which, as with the incidence of kidney disease, was more pronounced in those bearing one-allele of <i fontsize="11.0pt">Stat5a</i> than in those bearing two (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3A-B </span>&nbsp;&amp; <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1A</span>). We also measured production of IFN-&#611; and IL-17, two effector cytokines that are dysregulated in STAT5-null mice (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R35 ">Laurence et al., 2007</span>). IFN-&#611;<sup>+</sup> cells were highly enriched in the autoimmune-prone <i fontsize="11.0pt">Stat5b</i>-deficient mice but not age-matched <i fontsize="11.0pt">Stat5a</i>-deficient counterparts, suggesting that STAT5B may be particularly important for limiting Th1-type responses. IL-17A<sup fontsize="11.0pt">+</sup> Th17-type cells were also increased but this trend did not reach statistical significance (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3C&ndash;D</span>).</p><div class="jrnlFigBlock" id="BLK_F3" data-stream-name="a_F3" data-page-num="6" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F3"><span class="label">Figure 3.</span> Aberrant effector T cell responses in the absence of Stat5b.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Contour plots show percentages of CD44<sup fontsize="11.0pt">low</sup> IL-7R<sup fontsize="11.0pt">+</sup>naive and CD44<sup fontsize="11.0pt">high </sup>effector/memory CD4<sup fontsize="11.0pt">+</sup> T cells in the spleens of 8 week-old mice. (<span class="partLabel">B</span>) Scatter plots show percentages of naive CD4<sup fontsize="11.0pt">+</sup> T cells in spleens (left) and lymph nodes (right). Genotypes are ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1C</span>. (<span class="partLabel">C</span>) Contour plots show percentages of IFN-&#611;<sup>+</sup> or IL-17<sup fontsize="11.0pt">+</sup> CD4<sup fontsize="11.0pt">+</sup> T cells in the spleens of 8 week-old mice. (<span class="partLabel">D</span>) Box plots show log2 fold changes for IFN-&#611;<sup>+</sup> and IL-17<sup fontsize="11.0pt">+</sup> cells relative to wild type controls (WT=0; not shown). (<span class="partLabel">E</span>) Contour plots show percentages of PD1<sup fontsize="11.0pt">+</sup> CXCR5<sup fontsize="11.0pt">high </sup>Tfh cells in the spleens of 8 week-old mice. (<span class="partLabel">F</span>) Box plots show log2 fold changes for Tfh cells in spleens (left) and LNs (right) relative to wild type controls (WT=0; not shown). (<span class="partLabel">D</span> and <span class="partLabel">F</span>) Genotypes are ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D</span>. Data are compiled from 4 experiments (3&ndash;6 mice/group). Dotted red lines indicate two-fold changes.</p><img src="/ecs/article/80712/resources/elife08384f003.jpg" data-mce-src="../../ecs/article/80712/resources/elife08384f003.jpg" height="436" width="733"/></div><div class="jrnlFigBlock" id="BLK_F3-S1" data-stream-name="a_F3-S1"><p class="jrnlFigCaption" id="F3-S1"><span class="label">Figure 3&mdash;figure supplement 1.</span> Impact of Stat5 allele depletion on effector T cell responses.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Contour plots show percentages of CD44<sup fontsize="11.0pt">low</sup> IL-7R<sup fontsize="11.0pt">+ </sup>naive and CD44<sup fontsize="11.0pt">high </sup>effector/memory CD4<sup fontsize="11.0pt">+</sup> T cells. (<span class="partLabel">B</span>) Contour plots show percentages of PD1<sup fontsize="11.0pt">+</sup> CXCR5<sup fontsize="11.0pt">high </sup>T follicular helper cells. (<span class="partLabel">A</span> and <span class="partLabel">B</span>) Data are from spleens of 8 week-old mice and are representative of 4&ndash;5 experiments.</p><img src="/ecs/article/80712/resources/elife08384fs001.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs001.jpg" data-primary-extension="tif" data-width="1772" data-height="2062" data-id="1939682" alt="ecs/article/80712/resources/elife08384fs001.jpg" height="853" width="733"/></div><p class="jrnlSecPara">CD4<sup fontsize="11.0pt">+</sup> Tfh cells specialize in promoting B cell responses (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R9 ">Crotty, 2011</span>). Mirroring the abundance of GC B cells, there was dramatic accumulation of PD1<sup fontsize="11.0pt">high</sup> CXCR5<sup fontsize="11.0pt">high</sup> ICOS<sup fontsize="11.0pt">high </sup>Tfh cells in one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice, and a more modest enrichment in one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3E&ndash;F</span>, <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1B</span> &amp;<b fontsize="11.0pt"> data not shown</b>). At least two interpretations can be made for this disparity; either the two proteins are not functionally equivalent or the two genes have different outputs. Consistent with the latter view, mice lacking one-allele each of <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> (i.e. double-heterozygotes) had more Tfh cells than those lacking two-alleles of <i fontsize="11.0pt">Stat5a</i>, despite having the same total number of alleles. Moreover, the percentage of Tfh cells was comparable between two-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice and one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice, suggesting that two alleles of <i fontsize="11.0pt">Stat5a</i> are roughly equal to one allele of <i fontsize="11.0pt">Stat5b</i> (<span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3F</span> &amp; <span class="jrnlFigRef" data-citation-string=" F3-S1 ">Figure 3&mdash;figure supplement 1B</span>).</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">STAT5 paralog dose impacts multiple aspects of regulatory T cell function</h2><p class="jrnlSecPara">CD4<sup fontsize="11.0pt">+</sup> T regulatory (Treg) cells expressing the forkhead transcription factor, FOXP3, are essential for immunological tolerance (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R45 ">Malek and Castro, 2010</span>). Given the importance of STAT5 in Treg cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R44 ">Mahmud et al., 2013</span>), we next inspected this subset. Unlike STAT5-null mice, which exhibit a profound lack of Treg cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R72 ">Yao et al., 2007</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R6 ">Burchill et al., 2006</span>), frequencies of splenic FOXP3<sup fontsize="11.0pt">+</sup> cells were relatively normal across our STAT5 mutants. However, due to differences in overall cellularity, absolute counts were significantly lower in one- and two-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice, as well one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice. A similar trend was observed for LN resident Treg cells; frequencies were comparable to WT controls but total numbers were reduced in all genotypes bearing less than 3 total alleles (<span class="jrnlFigRef" data-citation-string=" F4-S1 ">Figure 4&mdash;figure supplement 1</span>). To further characterize the Treg compartment, we measured IL-2R&alpha;, a component of the IL-2 receptor that is critical for Treg cell homeostasis and function. It is also a both upstream and downstream of STAT5 signaling and, thus, can be viewed as an indicator of STAT5 activity (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R45 ">Malek and Castro, 2010</span>). We found that the percentage of IL-2R&alpha;<sup fontsize="11.0pt">+ </sup>Treg cells mirrored the total number of <i fontsize="11.0pt">Stat5</i> alleles; it was slightly reduced in mice with three alleles, lower in those with 2, and lower still in those with 1 (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4A&ndash;B</span>). We also noted that residual IL-2R&alpha;<sup fontsize="11.0pt">+ </sup>Treg cells from one-allele mice had reduced suppressive capacity and were unable to maintain expression of IL-2R&alpha; during in vitro culture (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4C&ndash;E</span>). Each of these phenotypes was more pronounced in the absence of <i fontsize="11.0pt">Stat5b</i> than <i fontsize="11.0pt">Stat5a</i>, again, illustrating both the dominance of the former and the relevance (and/or redundancy) of the latter.</p><div class="jrnlFigBlock" id="BLK_F4" data-stream-name="a_F4" data-page-num="7" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="25pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F4"><span class="label">Figure 4.</span> T regulatory cell function is impaired in Stat5b-deficient mice.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Contour plots show percentages of IL-2R&alpha;<sup fontsize="11.0pt">+</sup> cells within the FOXP3<sup fontsize="11.0pt">+</sup> Treg compartment in spleens of 8 week-old mice. (<span class="partLabel">B</span>) Box plots show log2 fold changes in the ratio of IL-2R&alpha;<sup fontsize="11.0pt">+</sup>/IL-2R&alpha;<sup fontsize="11.0pt">-</sup> Treg (WT=0; not shown). LN (top) and spleen (bottom) data are compiled from 5 experiments (4&ndash;6 mice/group) and genotypes ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D</span>. (<span class="partLabel">C</span>) IL-2R&alpha;<sup fontsize="11.0pt">+</sup> Treg cells from WT and <i fontsize="11.0pt">Stat5a-</i> or <i fontsize="11.0pt">Stat5b</i>-deficient mice were used for in vitro suppression assays. Histograms show CFSE dilution of responder T cells. (<span class="partLabel">D</span>) Line graph shows percent suppression across a range of responder:Treg ratios. Baseline is set according to WT controls at a 1:1 ratio. Data are compiled from 3 experiments. (<span class="partLabel">E</span>) Line graph shows the percent Treg cells that remained IL-2R&alpha;<sup fontsize="11.0pt">+</sup> during in vitro suppression. (<span class="partLabel">F</span>) IL-2R&alpha;<sup fontsize="11.0pt">+</sup> Treg cells were cultured with IL-2 for 72&thinsp;hr. Contour plots show the percentage of FOXP3<sup fontsize="11.0pt">+</sup> Treg cells expressing TBX21 (top) or IL-2<sup fontsize="11.0pt"/>(bottom). (<span class="partLabel">G</span>) Box plots show log2 fold changes for TBX21<sup fontsize="11.0pt">+</sup>, FOXP3<sup fontsize="11.0pt">+</sup> and IL-2<sup fontsize="11.0pt">+</sup> cells relative to wild type controls (WT=0; not shown). Data are compiled from 3 experiments. Dotted red lines indicate two-fold changes.</p><img src="/ecs/article/80712/resources/elife08384f004.jpg" data-mce-src="../../ecs/article/80712/resources/elife08384f004.jpg" height="790" width="733"/></div><div class="jrnlFigBlock" id="BLK_F4-S1" data-stream-name="a_F4-S1"><p class="jrnlFigCaption" id="F4-S1"><span class="label">Figure 4&mdash;figure supplement 1.</span> Impact of Stat5 allele depletion on Treg cells.</p><p class="jrnlFigCaption">Bar graphs show total numbers of Treg cells in spleens and LNs of 8 week old mice (4&ndash;6 mice per genotype). Number of <i fontsize="11.0pt">Stat5a, Stat5b</i> and total <i fontsize="11.0pt">Stat5</i> alleles (i.e. genotype) is explained in the key below each graph. Error bars indicate standard deviation.</p><img src="/ecs/article/80712/resources/elife08384fs002.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs002.jpg" data-primary-extension="tif" data-width="1417" data-height="678" data-id="1939562" alt="ecs/article/80712/resources/elife08384fs002.jpg" height="351" width="733"/></div><p class="jrnlSecPara">Given the appearance of IFN-&#611;<sup>+</sup> effector T cells in <i fontsize="11.0pt">Stat5b</i>-deficient mice, we next asked whether <i fontsize="11.0pt">Stat5b</i>-deficient Treg cells express TBX21, a transcription factor required for Treg cells to limit Th1-type responses (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R30 ">Koch et al., 2009</span>). Similar to conventional T cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">Liao et al., 2011</span>), we found that IL-2 was sufficient to induce TBX21 in WT Treg cells (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4F</span>). This effect was slightly reduced in the absence of <i fontsize="11.0pt">Stat5a</i> but almost completely abolished in the absence of <i fontsize="11.0pt">Stat5b</i>, consistent with the disparity of other STAT5-dependent parameters (e.g. T cell and Th1 cell frequencies). Both <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient Treg cells maintained FOXP3 expression similar to WT controls and, surprisingly, both gained the ability to produce IL-2, a cytokine that is typically restricted in this lineage (<span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4F&ndash;G</span>).</p><p class="jrnlSecPara">STAT5 is required for in vitro differentiation of induced regulatory (iTreg) T cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R72 ">Yao et al., 2007</span>). To dissect the contributions of STAT5A and STAT5B, we purified naive CD4<sup fontsize="11.0pt">+ </sup>T cells from our STAT5 mutants, cultured them under iTreg polarizing conditions and compared expression of FOXP3. We found that, although both paralogs appear to play a role, there&nbsp;were&nbsp;far&nbsp;fewer FOXP3<sup fontsize="11.0pt">+</sup> cells in one-allele <i fontsize="11.0pt">Stat5b</i>-deficient cultures &nbsp;than&nbsp;in&nbsp;(<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5A</span>). We also found that deletion of either paralog endowed FOXP3<sup fontsize="11.0pt">+</sup> iTreg cells with the ability to produce IL-2, which suggests that, beyond differentiation, STAT5 may limit the inflammatory potential of this subset (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5A&ndash;B</span>).</p><div class="jrnlFigBlock" id="BLK_F5" data-stream-name="a_F5" data-page-num="8" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F5"><span class="label">Figure 5.</span> Defective iTreg differentiation in the absence of Stat5b.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Naive CD4<sup fontsize="11.0pt">+</sup> T cells were cultured under iTreg-inducing conditions. Contour plots show percentages of FOXP3<sup fontsize="11.0pt">+</sup> and IL-2<sup fontsize="11.0pt">+</sup> cells. (<span class="partLabel">B</span>) Box plots show log2 fold changes for FOXP3<sup fontsize="11.0pt">+</sup> and IL-2<sup fontsize="11.0pt">+</sup> cells relative to wild type controls (WT=0; not shown). Data are compiled from 3 experiments and genotypes ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D</span>.&nbsp;(<span class="partLabel">C</span>) Naive T cells from one-allele <i fontsize="11.0pt">Stat5a-</i> or <i fontsize="11.0pt">Stat5b</i>-deficient mice were cultured as in (<span class="partLabel">A</span>) and processed for RNA-seq. GSEA plots show enrichment of Treg signature genes within the <i fontsize="11.0pt">Stat5a</i>-deficient (top) or <i fontsize="11.0pt">Stat5b</i>-deficient (bottom) datasets relative to WT controls. (<span class="partLabel">D</span>) Heat map shows a selection of STAT5-regulated, Treg signature transcripts. Data are presented as log2 fold changes relative to WT controls (not shown). RNA-seq analyses are compiled from 2 biological replicates per genotype.</p><img src="/ecs/article/80712/resources/elife08384f005.jpg" data-mce-src="../../ecs/article/80712/resources/elife08384f005.jpg" height="365" width="733"/></div><p class="jrnlSecPara">Next, we compared the transcriptomes of <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient T cells cultured under iTreg polarizing conditions. Gene set enrichment analysis revealed that the overall Treg gene signature - defined by a combination of FOXP3- and IL-2-dependent transcriptional programs (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R19 ">Hill et al., 2007</span>) - was similarly affected in both genotypes, meaning that there were no broad qualitative differences (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5C</span>). However, there were obvious quantitative differences; several key genes, including <i fontsize="11.0pt">Foxp3</i> and <i fontsize="11.0pt">Il2ra</i>, were more affected by the loss of STAT5B than STAT5A (<span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5D</span>). Thus, while both paralogs can impact Treg cell biology, deletion of <i fontsize="11.0pt">Stat5b</i> is clearly more disruptive, befitting its dominant station within immunological tolerance.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Specificity and redundancy of STAT5 paralogs for gene transcription</h2><p class="jrnlSecPara">To define the molecular basis for phenotypic differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient T cells, we employed a bioinformatic approach. First, we compared their transcriptomes either directly ex vivo or after acute exposure to STAT5-activating stimuli. The ex vivo set included naive T cells and<sup fontsize="11.0pt"/>Treg cells, while the in vitro set included naive T cells cultured with IL-7 and effector T cells cultured with IL-2 (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6A</span> &amp; <span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1A</span>). These pairings were chosen to match the expression patterns of requisite &#611;c co-receptors; IL-7R, which is highly expressed on naive T cells, and IL-2R&alpha;, which is highly expressed on effector T cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R55 ">Rochman et al., 2009</span>). One-allele mice were used because they exhibited the most dramatic T cell phenotypes.</p><div class="jrnlFigBlock" id="BLK_F6" data-stream-name="a_F6" data-page-num="9" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F6"><span class="label">Figure 6.</span> <i fontsize="11.0pt">Redundancy and specificity of STAT5 paralogs for gene transcription.</i>
</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Cartoons depict the cell types and experimental conditions used for RNA-seq. (<span class="partLabel">B</span>) Histograms show STAT5 paralog preference for all STAT5-regulated transcripts. Those which were more influenced by the loss of <i fontsize="11.0pt">Stat5a</i> are positioned to the left (X&lt;0) while those that were more influenced by the loss of <i fontsize="11.0pt">Stat5b</i> are positioned to the right (X&gt;0). Dotted red lines denote equivalence (X=0) and numbers indicate median paralog preference. (<span class="partLabel">C</span>) Pie charts depict paralog-specific transcripts. Those impacted only in <i fontsize="11.0pt">Stat5a</i>-deficient cells are indicated in blue, those impacted only in <i fontsize="11.0pt">Stat5b</i>-deficient cells are indicated in orange and those impacted in both genotypes are indicated in black. (<span class="partLabel">D</span>) Heat maps show a selection of STAT5-regulated transcripts. Data are presented as the log2 fold change relative to WT controls (not shown). (<span class="partLabel">E</span>) IL-2R&alpha; protein was measured in T cells treated with IL-2 (left) or IL-6 (right). Box plots show log2 fold changes for mean fluorescence intensity relative to wild type controls (WT=0; not shown). Genotypes are ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D</span>. Dotted red lines indicate a two-fold change. (<span class="partLabel">A&ndash;D</span>) RNA-seq analyses are compiled from 2&ndash;3 biological replicates per genotype.</p><img src="/ecs/article/80712/resources/elife08384f006.jpg" data-mce-src="/ecs/article/80712/resources/elife08384f006.jpg" data-primary-extension="tif" data-width="2126" data-height="1316" data-id="1939522" alt="ecs/article/80712/resources/elife08384f006.jpg" height="454" width="733"/></div><div class="jrnlFigBlock" id="BLK_F6-S1" data-stream-name="a_F6-S1"><p class="jrnlFigCaption" id="F6-S1"><span class="label">Figure 6&mdash;figure supplement 1.</span> Transcriptomic analysis of Stat5a- and Stat5b-deficient T cells.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Cartoons depict the experimental conditions used for RNA-seq. CD4<sup fontsize="11.0pt">+</sup> T cells were purified from WT and one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice, then transcriptomes measured either directly ex vivo or after in vitro treatment with acute STAT5 stimuli. The ex vivo set included naive cells (first row) and IL-2R&alpha;<sup fontsize="11.0pt">+</sup> Treg cells (second row), while the in vitro set included naive cells exposed to IL-7 (third row) and effector cells exposed to IL-2 (fourth row). (<span class="partLabel">B</span>) Multidimensional scaling (MDS) plots show the overall relatedness between experimental groups. Each biological replicate is represented by black (WT), blue (<i fontsize="11.0pt">Stat5a</i>-deficient) or orange (<i fontsize="11.0pt">Stat5b</i>-deficient) elements. (<span class="partLabel">C</span>) Volcano plots show fold changes and variances for all transcripts relative to WT controls. Those exhibiting &gt;1.5 fold change and p&lt;0.05 are highlighted. Numbers indicate the sum of transcripts that were down-regulated (upper left) or up-regulated (upper right) in <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient cells. Dotted red lines are drawn at 2 fold changes and 0.05 p values. (<span class="partLabel">D</span>) Venn diagrams indicate the number of transcripts exhibiting &gt;1.5 fold change and &lt;0.05 p values only in <i fontsize="11.0pt">Stat5a</i>-deficient cells (blue), only in <i fontsize="11.0pt">Stat5b</i>-deficient cells (orange) or in both genotypes (black). (<span class="partLabel">E</span>) XY plots show log&nbsp;2 fold change for STAT5-regulated genes in <i fontsize="11.0pt">Stat5a</i>-deficient (x axis) versus <i fontsize="11.0pt">Stat5b</i>-deficient (y axis) cells. Blue and orange elements represent transcripts designated as STAT5A- or STAT5B-specific, respectively. Dotted red lines are drawn at 1.5 fold changes. (<span class="partLabel">A&ndash;-E</span>) Analyses are compiled from 2&nbsp;to&nbsp;3 biological replicates per genotype.</p><img src="/ecs/article/80712/resources/elife08384fs003.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs003.jpg" data-primary-extension="tif" data-width="2126" data-height="1342" data-id="1939702" alt="ecs/article/80712/resources/elife08384fs003.jpg" height="463" width="733"/></div><div class="jrnlFigBlock" id="BLK_F6-S2" data-stream-name="a_F6-S2"><p class="jrnlFigCaption" id="F6-S2"><span class="label">Figure 6&mdash;figure supplement 2.</span> Transcriptomic analysis of Stat5a- and Stat5b-deficient T cells.</p><p class="jrnlFigCaption">Genome browser tracks show transcript abundance in WT (grey), <i fontsize="11.0pt">Stat5a</i>-deficient (blue) or <i fontsize="11.0pt">Stat5b</i>-deficient (orange) cells. Vertical RPKM scale varies from gene to gene (but not across experimental conditions) and is denoted by the numbers at the upper left of each column. Shown is one of 2&ndash;3 biological replicates per genotype.</p><img src="/ecs/article/80712/resources/elife08384fs004.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs004.jpg" data-primary-extension="tif" data-width="2126" data-height="866" data-id="1939582" alt="ecs/article/80712/resources/elife08384fs004.jpg" height="299" width="733"/></div><p class="jrnlSecPara">We began our transcriptomic survey by performing multidimensional scaling of the datasets, thereby gaining a broad overview of the experimental groups. <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient cells typically clustered together and equidistant from WT controls, suggesting that the loss of either paralog has comparable genome-wide effects (<span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1B</span>). Next, we used statistical variance to identify differentially expressed genes. Surprisingly, we found widespread discord between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient cells; many transcripts appeared dysregulated in the absence of one paralog or the other (<span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1C&ndash;D</span>). However, upon close inspection, we concluded that this disparity was largely due to the arbitrary&nbsp;fold-change cutoff that was used. Most genes that were designated as STAT5B-specific were also affected by the loss of STAT5A (and vice versa), albeit to a lesser degree that did not reach our 2-fold threshold (<span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1E</span>). To avoid this statistical artifact, we devised a &lsquo;paralog preference&rsquo; scale whereby all STAT5-sensitive genes were compiled and ranked according to how much they were impacted by the loss of <i fontsize="11.0pt">Stat5a</i> or <i fontsize="11.0pt">Stat5b</i>. This analysis revealed a binomial distribution for all experimental conditions. The majority of genes were in central bins, affected by both STAT5A and STAT5B, while membership in peripheral bins decreased steadily as paralog preference increased. Importantly, all curves were shifted towards STAT5B, suggesting that STAT5-sensitive genes are generally more impacted by STAT5B than STAT5A (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6B</span>). This latter trend was also evident at the protein level; IL-2-driven (but not IL-6-driven) induction of IL-2R&alpha; was clearly more diminished in <i fontsize="11.0pt">Stat5b</i>-deficient cells than in <i fontsize="11.0pt">Stat5a</i>-deficient counterparts (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6E</span>). Collectively, these data affirm that STAT5B is dominant over STAT5A while, at the same time, demonstrating pervasive redundancy at the level of gene transcription.</p><p class="jrnlSecPara">Beyond quantitative differences, our transcriptomic survey also revealed qualitative differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient cells. Using strict analysis criteria, we discovered that between 12% and 22% of all STAT5-sensitive genes can be classified as paralog-specific, meaning that they are solely dependent on either STAT5A or STAT5B. The absolute number of paralog-specific genes varied across cell states and stimuli, with the largest allotment found in IL-7-treated naive cells, and was typically skewed towards STAT5B (<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6C</span> &amp; <span class="jrnlFigRef" data-citation-string=" F6-S1 ">Figure 6&mdash;figure supplement 1E</span>). Thus, we can create 2 general categories: &lsquo;pan-STAT5' genes that are regulated by both STAT5A and STAT5B (e.g. <i fontsize="11.0pt">Pdk1, Cish, Lta</i>) and 'paralog-specific' genes that are regulated by either STAT5A (e.g. <i fontsize="11.0pt">Smc6</i>) or STAT5B (e.g. <i fontsize="11.0pt">Cd74</i>)(<span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6D</span> &amp; <span class="jrnlFigRef" data-citation-string=" F6-S2 ">Figure 6&mdash;figure supplement 2</span>). Given that pan-STAT5 genes are much more numerous, we propose that phenotypic differences between <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> deficient T cells are due largely to paralog preference, owing to the fact that deletion of <i fontsize="11.0pt">Stat5b</i> has greater quantitative impact, with limited contribution from qualitative,&nbsp;paralog-specific effects.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Paralog dose dictates genome-wide distribution of STAT5</h2><p class="jrnlSecPara">Functional divergence between STAT5A and STAT5B could be due to differences in target gene selection. Previous studies have addressed this issue by comparing genomic distributions by ChIP-seq (chromatin immunoprecipitation followed by massively parallel sequencing) using separate, paralog-specific antibodies in WT cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R37 ">Liao et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">2011</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R27 ">Kanai et al., 2014</span>). We took an alternative approach involving a single antibody that recognizes both paralogs and T cells from <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice, as well as &lsquo;double heterozygotes&rsquo; (hereafter referred to as <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> mice). In&nbsp;line&nbsp;with&nbsp;previous&nbsp;studies,&nbsp;1275 unique regions of STAT5 occupancy were&nbsp;called&nbsp;for&nbsp;WT cells(<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7A</span>). By contrast, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> cells had fewer peaks (658 total; <span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7A</span>) that tended to be less robust (i.e. lower signal intensity) than those found in WT controls (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7D</span>), indicating that changes in STAT5 availability can impact genomic distribution even when both paralogs are present. Total peaks were also reduced in <i fontsize="11.0pt">Stat5a</i>-deficient cells (609 total) and almost expunged in <i fontsize="11.0pt">Stat5b</i>-deficient cells (97 total), again, illustrating both the relevance of the former and the dominance of the latter (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7A</span>). STAT5 peaks were similarly localized across all genotypes - they typically congregated near transcriptional start sites but could also be found at distal regions, sometimes &gt;100 kb from annotated genes - and were highly enriched for STAT-binding motifs (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7B</span> &amp; <span class="jrnlFigRef" data-citation-string=" F7-S1 ">Figure 7&mdash;figure supplement 1</span>).</p><div class="jrnlFigBlock" id="BLK_F7" data-stream-name="a_F7" data-page-num="11" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F7"><span class="label">Figure 7.</span> Influence of paralog dose on genomic distribution of STAT5.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) CD4<sup fontsize="11.0pt">+</sup> T cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and two-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice were cultured in the presence of IL-2, then processed for pan-STAT5 ChIP-seq. Bar graph shows the total number of STAT5-bound peaks in each genotype. (<span class="partLabel">B</span>) Histogram shows distribution of STAT5-bound peaks relative to transcriptional start sites (TSS). (<span class="partLabel">C</span>) Circos plot shows overlap of STAT5-bound beaks across genotypes. Connection width represents the number of overlapping peaks. Only peaks shared with WT cells are shown. Those found only in WT cells are presented as a white semi-circle at the top. (<span class="partLabel">D</span>) Bar graph shows the percentage of WT peaks detected in each genotype (WT=100%). Violin plot depicts the total number of sequenced tags (i.e. peak intensity) for peak shared with WT controls. (<span class="partLabel">E</span>) Genome browser tracks show STAT5 peaks near selected genes. Numbers indicate the maximum peak height within the interval. (<span class="partLabel">A&ndash;E</span>) Data are representative of two biological replicates.</p><img src="/ecs/article/80712/resources/elife08384f007.jpg" data-mce-src="../../ecs/article/80712/resources/elife08384f007.jpg" height="255" width="733"/></div><div class="jrnlFigBlock" id="BLK_F7-S1" data-stream-name="a_F7-S1"><p class="jrnlFigCaption" id="F7-S1"><span class="label">Figure 7&mdash;figure supplement 1.</span> Transcription factor motifs associated with STAT5 binding peaks.</p><p class="jrnlFigCaption">CD4<sup fontsize="11.0pt">+</sup> T cells were isolated from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and two-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice, then cultured in the presence of IL-2 and processed for ChIP-seq. Chart shows the top ten transcription factor-binding motifs associated with STAT5 peaks in each genotype. p Values and enrichment (% STAT5 peaks with indicated motif &divide;% random peaks with indicated motif) are shown for one of two biological replicates.</p><img src="/ecs/article/80712/resources/elife08384fs005.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs005.jpg" data-primary-extension="tif" data-width="1772" data-height="796" data-id="1939602" alt="ecs/article/80712/resources/elife08384fs005.jpg" height="329" width="732"/></div><div class="jrnlFigBlock" id="BLK_F7-S2" data-stream-name="a_F7-S2"><p class="jrnlFigCaption" id="F7-S2"><span class="label">Figure 7&mdash;figure supplement 2.</span> Correlation between STAT5 binding and transcription of Il2ra, Bcl2 and Bcl6.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Effector CD4<sup fontsize="11.0pt">+</sup> T cells from one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice were cultured in the presence of IL-2 and processed for RNA-seq. (<span class="partLabel">B</span>) Effector CD4<sup fontsize="11.0pt">+</sup> T cells from one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice were transduced with control or STAT5A-expressing retrovirus and processed for RNA-seq. (<span class="partLabel">C</span>) Effector CD4<sup fontsize="11.0pt">+</sup> T cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and two-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice were cultured in the presence of IL-2 and processed for ChIP-seq. (<span class="partLabel">A&ndash;C</span>) Genome browser tracks show transcript abundance or STAT5 binding for selected genes. Numbers indicate either RPKM values for the most detected exon (RNA-seq) or maximum STAT5-binding peak intensity within the interval (ChIP-seq). Shown is one of two biological replicates for each genotype and/or experimental condition.</p><img src="/ecs/article/80712/resources/elife08384fs006.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs006.jpg" data-primary-extension="tif" data-width="1890" data-height="1559" data-id="1939652" alt="ecs/article/80712/resources/elife08384fs006.jpg" height="605" width="733"/></div><p class="jrnlSecPara">Most peaks found in <i fontsize="11.0pt">Stat5b</i>-deficient cells could be matched to peaks in <i fontsize="11.0pt">Stat5a</i>-deficient, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> or WT cells (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7C</span>). This implies a hierarchy whereby certain sites are preserved even when STAT5B is absent. STAT5B-independent peaks tended to occur near genes that were highly occupied in WT controls and whose expression was highly dysregulated in <i fontsize="11.0pt">Stat5b</i>-deficient cells (e.g. <i fontsize="11.0pt">Cish, Lta</i>), suggesting that only the most robust (i.e. high-affinity) STAT5-binding sites were preserved (<span class="jrnlFigRef" data-citation-string=" F8 ">Figure 8</span>). Peaks detected within <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and <i fontsize="11.0pt">Stat5a</i>-deficient cells also tended to be highly occupied in WT controls and dysregulated in STAT5-deficient cells, but the trend was not as dramatic, indicating that, while a full complement of STAT5 alleles may be necessary to achieve optimal responses, STAT5B has the greater influence on genomic distribution (<span class="jrnlFigRef" data-citation-string=" F8 ">Figure 8</span>).</p><div class="jrnlFigBlock" id="BLK_F8" data-stream-name="a_F8" data-page-num="12" data-float-placement="TopDoubleColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F8"><span class="label">Figure 8.</span> Preservation of high affinity targets in the absence of STAT5B.</p><p class="jrnlFigCaption">Circle plot relates STAT5 occupancy and STAT5-dependent transcription for genes bound in: (1) only WT cells (white circle), (2) WT and <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i>, cells (grey circle), (3) WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i>, and <i fontsize="11.0pt">Stat5a<sup>-/-</sup></i> cells (blue circle), (4) WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup>, Stat5a<sup>-/</sup></i><sup fontsize="11.0pt">-</sup> and <i fontsize="11.0pt">Stat5b<sup>-/-</sup></i> cells (orange circle). STAT5 ChIP-seq peaks were assigned to genes based on proximity to transcriptional start sites (+/- 10 kb). X axis denotes the average height of gene-associated peaks in WT cells. Y axis denotes the average mRNA expression variance (-log10 <i fontsize="11.0pt">p</i> value) for the corresponding peak-associated genes. Variance is derived from the comparison WT and one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient cells cultured in the presence of IL-2 (from <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6</span>). Size of each circle represents the total number of gene-associated peaks in each group (number is shown).</p><img src="/ecs/article/80712/resources/elife08384f008.jpg" data-mce-src="/ecs/article/80712/resources/elife08384f008.jpg" data-primary-extension="tif" data-width="1535" data-height="803" data-id="1939482" alt="ecs/article/80712/resources/elife08384f008.jpg" height="384" width="733"/></div><p class="jrnlSecPara">Among the genes that were engaged by STAT5 in WT cells and dysregulated in <i fontsize="11.0pt">Stat5</i>-deficient cells was <i fontsize="11.0pt">Il2ra</i>, which, as discussed, is a known STAT5 target gene that is critical for Treg function and homeostasis (<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7E</span> &amp; <span class="jrnlFigRef" data-citation-string=" F7-S2 ">Figure 7&mdash;figure supplement 2</span>). Another was <i fontsize="11.0pt">Bcl6</i>, considered the master transcription factor for Tfh differentiation (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R9 ">Crotty, 2011</span>)(<span class="jrnlFigRef" data-citation-string=" F7 ">Figure 7E</span> &amp; <span class="jrnlFigRef" data-citation-string=" F7-S2 ">Figure 7&mdash;figure supplement 2</span>). In this case, STAT5-binding appears to be a negative regulatory event; multiple studies (including the present work) have shown that STAT5 can suppress <i fontsize="11.0pt">Bcl6 </i>expression in T cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R51 ">Oestreich et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R39 ">Liao et al., 2014</span>). Thus, taken together, our RNA-seq and ChIP-seq data provide a molecular rationale for the Treg and Tfh phenotypes seen in <i fontsize="11.0pt">Stat5</i>-deficient mice.</p><h2 class="jrnlHead2" italic="true" wordcount="true" headweightage="19">Asymmetric expression of STAT5 paralogs in helper T cells</h2><p class="jrnlSecPara">Based on our RNA-seq and&nbsp;ChIP-seq&nbsp;studies, we reasoned that asymmetric expression, rather than widespread paralog-specific activity, likely explains the phenotypic differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient T cells. To explore this possibility, we mined various transcriptome datasets (including our own) and confimed that, indeed, <i fontsize="11.0pt">Stat5b</i> is more abundant than <i fontsize="11.0pt">Stat5a</i> at the mRNA level (<span class="jrnlFigRef" data-citation-string=" F9-S1 ">Figure 9&mdash;figure supplement 1</span>). Next, we used flow cytometry to measure total STAT5 protein in naive, regulatory (Treg), follicular (Tfh) and effector/memory T cells. Regardless of cellular subset, the results were clear: removing one-allele of <i fontsize="11.0pt">Stat5b </i>(<i fontsize="11.0pt">Stat5a<sup>+/+</sup> Stat5b<sup>+/-</sup></i>) had greater impact than removing one-allele of <i fontsize="11.0pt">Stat5a </i>(<i fontsize="11.0pt">Stat5a<sup>+/+</sup> Stat5b<sup>+/-</sup></i>) while, at&nbsp;the&nbsp;other&nbsp;end&nbsp;of&nbsp;the&nbsp;spectrum, retaining one-allele of <i fontsize="11.0pt">Stat5b </i>(<i fontsize="11.0pt">Stat5a<sup>-/-</sup> Stat5b<sup>+/-</sup></i>) &nbsp;than retaining one-allele of <i fontsize="11.0pt">Stat5a </i>(<i fontsize="11.0pt">Stat5a<sup>+/-</sup> Stat5b<sup>-/-</sup></i>)(<span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span>). A similar trend was observed for tyrosine-phosphorylated STAT5 dowsntream&nbsp;of IL-2 or IL-7 (<span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9B</span>). Thus, we conclude that STATB makes a greater contribution to the total STAT5 protein pool.</p><p class="jrnlSecPara">To determine how STAT5 availability (i.e. paralog dose) influences gene expression, we transduced <i fontsize="11.0pt">Stat5b</i>-deficient T cells with a STAT5A-expressing retrovirus, thereby increasing the total amount of STAT5 without re-introducing STAT5B. We first validated the system by measuring <i fontsize="11.0pt">Il2ra</i>, a well-documented STAT5 target, and found it to be highly induced at both the mRNA and protein levels (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10A</span>). Transcriptomic&nbsp;analysis&nbsp;revealed&nbsp;that,&nbsp;overall, ectopic STAT5A mobilized 320 genes, most of which fall within the pan-STAT5 category (e.g. <i fontsize="11.0pt">Cish, Lta</i>)(<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10A&ndash;B</span>). Applied to our paralog preference scale, these genes did not favor STAT5A, meaning that they were similarly affected in <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient cells (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10A</span>),&nbsp;and GSEA revealed a high degree of enrichment for both STAT5A- or STAT5B-dependent gene sets (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10C</span>). Thus, our data support the idea that differences&nbsp;in STAT5 protein concentrations underlie many (if not most) of the transcriptomic divergence between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient cells.</p><p class="jrnlSecPara"><br/>
</p><p class="jrnlSecPara">Having established that ectopic STAT5A can rescue gene expression in <i fontsize="11.0pt">Stat5b</i>-deficient T cells, we next asked whether it can rescue cellular differentiation. For these studies, naive T cells from wild type, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> or one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice were cultured under iTreg polarizing conditions, transduced with either control or STAT5A retrovirus and FOXP3 measured to assess lineage commitment. As&nbsp;expected,&nbsp;FOXP3 was reduced in control-transduced <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> cells and almost completely abolished in control-transduced <i fontsize="11.0pt">Stat5b</i>-deficient cells (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10D</span>). However, when ectopic STAT5A was introduced, the percentage of FOXP3<sup fontsize="11.0pt">+</sup> cells became comparable across all genotypes and, whether endogenous (top row) or ectopic (bottom row), there was a clear linear correlation between STAT5 and FOXP3 protein levels (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10D</span>). IL-2R&alpha;was also diminished in both&nbsp;<i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and <i fontsize="11.0pt">Stat5b</i>-deficients cells, and was&nbsp;restored by ectopic STAT5A (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10E</span>). These data argue that a threshold concentration of STAT5 must be reached to institute the Treg program and, given the conspicuous effect of ectopic STAT5A on WT cells (<span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10D</span> &amp; <span class="jrnlFigRef" data-citation-string=" F10-S1 ">Figure 10&mdash;figure supplement 1A</span>), they imply that STAT5 is a limiting resource for this process.</p><p class="jrnlSecPara">Although they share a common instructive cytokine (TGF-&beta;), Th17 cells and Treg cells have opposing pro- and anti-inflammatory functions. STAT5 is key to this divergence - it promotes Treg responses at the expense of Th17 responses &ndash; so we next investigated the effect of paralog dose on Th17&nbsp;differentiation. We found that the percentage of IL-17+ cells was 4-fold higher in <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> Th17 cultures and &gt;25-fold higher in <i fontsize="11.0pt">Stat5b</i>-deficient Th17 cultures than in WT controls, consistent with a high paralog dose threshold,&nbsp;and&nbsp;most&nbsp;importantly, that ectopic STAT5 not only extinguished IL-17 but also induced FOXP3 in all genotypes, thereby demonstrating that changes in STAT5 concentration can tip the balance between effector and regulatory T cells programs (<span class="jrnlFigRef" data-citation-string=" F10-S1 ">Figure 10&mdash;figure supplement 1B</span>).</p><h1 class="jrnlHead1" bold="true" patternhead="true">Discussion</h1><p class="jrnlSecPara">Although the importance of STAT5 is widely recognized, there is no consensus on whether its closely related paralogs, STAT5A and STAT5B, are redundant or functionally distinct. Assuming the latter, it is also unclear how specificity can be achieved given their extensive structural homology. Both positions are grounded in sound experimental evidence but, until the present studies, there has been no comprehensive inquiry on their relationship in immune cells. We addressed this longstanding question in primary CD4<sup fontsize="11.0pt">+</sup> helper T cells, the principal orchestrators of adaptive immunity. Using a combination of genetic and genomic approaches, we demonstrate that STAT5B is dominant over STAT5A and, thus, plays a non-redundant role in controlling effector and regulatory T cell responses. This conclusion is based on phenotypic differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient mice, as well as bioinformatic analyses showing that STAT5B has greater impact on both selection and transcription of STAT5 target genes. The disparity does not appear to be due to differences in genome wide distribution or transcriptional capacity but, instead, relates to differences in relative abundance. Consistent with the latter point, our loss- and gain-of-function studies demonstrate that a threshold concentration of STAT5 must be reached to execute STAT5-dependent gene expression and differentiation programs. Based on these findings, we submit that STAT5A and STAT5B are largely redundant at the molecular level, but not at the cellular or organismal levels, where STAT5B is dominant.</p><p class="jrnlSecPara">It has been proposed that the target repertoires of STAT5A and STAT5B vary due to subtle differences in their DNA-binding domains (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R4 ">Boucheron et al., 1998</span>). However, this notion has been disputed because the nature and location of the&nbsp;amino&nbsp;acid&nbsp;substitutions may not alter protein structure enough to impact specificity. In addition, multiple studies have shown that the consensus DNA-binding motifs for STAT5A and STAT5B are identical, although it should be noted that these measured optimal binding to synthetic oligonucleotides in cell free systems, leaving open the possibility that divergent binding properties become apparent only at lower affinity sites or in the context of native chromatin (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R59 ">Soldaini et al., 1999</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R12 ">Ehret et al., 2001</span>). Indeed, differential binding of STAT5A or STAT5B has been detected at several loci in primary immune cells but it remains unclear whether this reflects <i fontsize="11.0pt">bonafide</i> differences in specificity or other factors that may influence target gene selection (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R37 ">Liao et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">2011</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R69 ">Yamaji et al., 2013</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R27 ">Kanai et al., 2014</span>). For instance, it is known that STAT5A and STAT5B can exhibit distinct phosphorylation patterns, so preferential binding may reflect cell type- or stimulus-specific differences in activation rather than distinct targeting capabilities (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R7 ">Caldenhoven et al., 1998</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R18 ">Hennighausen and Robinson, 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R56 ">Rosen et al., 1996</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R46 ">Meinke et al., 1996</span>). Our work supports this latter view by establishing that, even before activation, relative abundance of STAT5A versus STAT5B determines which paralog will dominate a given transcriptional response.</p><div class="jrnlFigBlock" id="BLK_F9" data-stream-name="a_F9" data-page-num="13" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F9"><span class="label">Figure 9.</span> Relative abundance of STAT5A and STAT5B in helper T cells.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Total STAT5 protein was measured in naive, Treg, Tfh and effector/memory T cells. Donut charts indicate the percentage of total STAT5 protein accounted for by each paralog. Histograms show representative flow cytometry data from one of three experiments. (<span class="partLabel">B</span>) Naive, Treg and effector/memory T cells were treated with IL-2 or IL-7 and phospho-STAT5 measured by flow cytometry. Box plots show log2 fold changes for the percentage of p-STAT5<sup fontsize="11.0pt">high</sup> cells relative to wild type controls (WT=0; not shown). Genotypes are ordered as in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1D</span>. Dotted red lines indicate a two-fold change (3&ndash;4 replicates/group).</p><img src="/ecs/article/80712/resources/elife08384f009.jpg" data-mce-src="../../ecs/article/80712/resources/elife08384f009.jpg" height="318" width="733"/></div><div class="jrnlFigBlock" id="BLK_F9-S1" data-stream-name="a_F9-S1"><p class="jrnlFigCaption" id="F9-S1"><span class="label">Figure 9&mdash;figure supplement 1.</span> Relative abundance of STAT5A and STAT5B in helper T cells.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) Donut charts indicate the percentage total STAT5 mRNA (top) or protein (bottom) accounted for by each paralog in ex vivo naive or Treg cells. mRNA data are compiled from 2&ndash;3 RNA-seq replicates and protein data are compiled from 3 flow cytometry replicates (<span class="partLabel">B</span>) Donut charts indicate the percentage total STAT5 mRNA accounted for by each paralog in mouse naive or Treg cells. Data were sourced from the Immunological Genome Project (top) or the EMBL-EBI Expression Atlas (bottom). (<span class="partLabel">C</span>) Donut charts indicate the percentage total STAT5 mRNA accounted for by each paralog in human naive or Treg cells. Data were sourced from the BioGPS Primary Cell Atlas.</p><img src="/ecs/article/80712/resources/elife08384fs007.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs007.jpg" data-primary-extension="tif" data-width="2008" data-height="567" data-id="1939322" alt="ecs/article/80712/resources/elife08384fs007.jpg" height="207" width="733"/></div><div class="jrnlFigBlock" id="BLK_F9-S2" data-stream-name="a_F9-S2"><p class="jrnlFigCaption" id="F9-S2"><span class="label">Figure 9&mdash;figure supplement 2.</span> Stat5a and Stat5b are transcribed from opposite DNA strands.</p><p class="jrnlFigCaption">Genome browser tracks show relative abundance and DNA strand origin of <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> mRNAs within mouse spleen, thymus and mammary gland. Transcripts originating from the minus (-) strand are presented in the ascending orientation while those from the plus (+) strand are in the descending orientation. Numbers indicate RNA-seq FPKM values for the most detected exon within the interval. Data are sourced from the ENCODE consortium and one of 2 biological replicates visualized using the UCSC genome browser.</p><img src="/ecs/article/80712/resources/elife08384fs008.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs008.jpg" data-primary-extension="tif" data-width="1772" data-height="1372" data-id="1939302" alt="ecs/article/80712/resources/elife08384fs008.jpg" height="568" width="733"/></div><div class="jrnlFigBlock" id="BLK_F9-S3" data-stream-name="a_F9-S3"><p class="jrnlFigCaption" id="F9-S3"><span class="label">Figure 9&mdash;figure supplement 3.</span> Putative lymphocyte-restricted enhancers within the Stat5a/b locus.</p><p class="jrnlFigCaption">Genome browser tracks display DNAse I hypersensitivity sites (DHS) within the mouse <i fontsize="11.0pt">Stat5a/b</i> locus. Shown are data for primary CD4<sup fontsize="11.0pt">+</sup> T cells (naive, activated, Treg), as well as spleen, thymus and multiple non-lymphoid tissues. 3 putative lymphoid-specific DHS sites are highlighted. Numbers indicate maximum tag count for all DHS sites within the interval. Data are sourced from the ENCODE consortium and one of 2 biological replicates visualized using the UCSC genome browser.</p><img src="/ecs/article/80712/resources/elife08384fs009.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs009.jpg" data-primary-extension="tif" data-width="1417" data-height="2243" data-id="1939342" alt="ecs/article/80712/resources/elife08384fs009.jpg" height="1161" width="733"/></div><p class="jrnlSecPara">Recent work has shown that small oscillations in transcription factor availability can have genome-wide consequences (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R5 ">Brewster et al., 2014</span>). The idea that STAT5 concentration can impact cellular function also has precedent. Of particular interest are studies reporting severe immunological phenotypes in transgenic mice which over-express STAT5A or STAT5B (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R28 ">Kelly et al., 2003a</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R29 ">Kelly et al., 2003b</span>), and studies showing that <i fontsize="11.0pt">Stat5a/b</i> haplo-insufficiency ameliorates contact hypersensitivity (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R50 ">Nivarthi et al., 2014</span>). In addition, we have previously demonstrated that ectopic STAT5A can expand the target repertoire of STAT5 in mouse embryonic fibroblasts (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R79 ">Zhu et al., 2012</span>), and have explored the concept of STAT5 gene dosage in the context of mammary development, finding that a high STAT5 threshold must be reached for mammary epithelial cell differentiation (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R69 ">Yamaji et al., 2013</span>). Given that STAT5A is the dominant paralog in mammary epithelium, we can infer that asymmetric expression of STAT5 paralogs is not just a feature of immune cells, and that it must be controlled in a tissue-specific manner (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R47 ">Metser et al., 2016</span>). Several mechanisms may explain this phenomenon. First, it is known that <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> are transcribed from opposite DNA strands and, thus, may be subject to strand-specific modes of regulation (<span class="jrnlFigRef" data-citation-string=" F9-S2 ">Figure 9&mdash;figure supplement 2</span>). Second, differential transcription could be achieved through paralog-specific enhancer elements whose accessibility is tissue- and/or cell type-restricted. We have recently characterized an intergenic enhancer that drives expression of <i fontsize="11.0pt">Stat5a</i> in mammary epithelium and have identified multiple DNase hypersensitivity sites within the <i fontsize="11.0pt">Stat5b</i> locus that are present in T cells but not in non-lymphoid tissues, perhaps indicating an analogous mechanism for immune cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R47 ">Metser et al., 2016</span>)(<span class="jrnlFigRef" data-citation-string=" F9-S3 ">Figure 9&mdash;figure supplement 3</span>). Third, paralog-specific epigenetic modifications, such as histone or DNA methylation, may impose distinct transcriptional outputs, as shown for tumor cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R75 ">Zhang et al., 2007</span>). Fourth, the 3&rsquo; UTRs of <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> are highly divergent so it is possible that their mRNAs are subject to post-transcriptional regulation via distinct sets of microRNAs and/or RNA-binding proteins (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R41 ">Liu et al., 1995</span>).</p><p class="jrnlSecPara">Beyond asserting the dominance of STAT5B, our work also affirms the importance of STAT5A. Several observations support this latter point: 1) deletion of one <i fontsize="11.0pt">Stat5a</i> allele exaggerates the gross and cellular phenotypes of <i fontsize="11.0pt">Stat5b</i>-deficient mice, 2) transcription of STAT5 target genes is typically influenced by both STAT5A or STAT5B, and 3) ectopic STAT5A can rescue gene expression in <i fontsize="11.0pt">Stat5b</i>-deficient cells. Furthermore, just one-allele of either <i fontsize="11.0pt">Stat5a</i> or <i fontsize="11.0pt">Stat5b</i> is sufficient to prevent the perinatal lethality and anemia seen in STAT5-null mice, suggesting that molecular redundancy protects the most critical &lsquo;life-and-death&rsquo; functions. We also identified a small subset of genes that appear to be regulated by either STAT5A or STAT5B, some of which have known immunological functions. Given that STAT5B is more abundant, it can be argued that all STAT5B-dependency may be due to a high paralog dose threshold, but this cannot explain the appearance of STAT5A-dependent genes. Thus, we propose that phenotypic differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient T cells result from widespread &lsquo;paralog preference&rsquo; and circumscribed 'paralog specificity'.</p><p class="jrnlSecPara">Our ChIP-seq studies indicate that the overall availability of STAT5, whether STAT5A or STAT5B, has profound influence on target gene selection. Previous studies have compared genomic distribution of STAT5A and STAT5B in primary CD4<sup fontsize="11.0pt">+</sup> T cells and found that they mostly overlap, thereby supporting the idea of redundancy (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R37 ">Liao et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R38 ">2011</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R27 ">Kanai et al., 2014</span>). However, they also identified a subset of sites that are occupied by one paralog or the other and, thus, have been taken as evidence for paralog specificity. All such comparisons (including ours) should be interpreted with care. Shared sites can be appointed with confidence but, due to technical confounders (e.g. differences in antibody affinity), incongruent sites cannot be definitively classified as STAT5A- or STAT5B-specific. The mechanisms underlying differential binding must also be considered. It is possible that <i fontsize="11.0pt">bona fide</i> paralog-specific binding sites do exist, but these are probably only a minor fraction. In most cases, differential binding likely reflects competition; the more abundant paralog is more likely to be detected. Given this nuance, claims that certain genes are uniquely regulated by STAT5A or STAT5B should be tempered. For instance, it has been suggested that <i fontsize="11.0pt">Bcl2l1</i> is regulated only by STAT5A and that <i fontsize="11.0pt">Bcl2, Il2ra</i> and <i fontsize="11.0pt">Foxp3</i> are regulated only by STAT5B (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R27 ">Kanai et al., 2014</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R22 ">Jenks et al., 2013</span>). Our data indicate that these are more accurately described as 'pan-STAT5' genes that are more impacted by deletion of one paralog or the other.</p><div class="jrnlFigBlock" id="BLK_F10" data-stream-name="a_F10" data-page-num="14" data-float-placement="TopMarginalColumn" data-top-gap="12pt" data-bot-gap="12pt" data-left-gap="12pt" data-right-gap="12pt"><p class="jrnlFigCaption" id="F10"><span class="label">Figure 10.</span> Paralog dose governs STAT5-driven gene transcription.</p><p class="jrnlFigCaption">(<span class="partLabel">A</span>) CD4<sup fontsize="11.0pt">+</sup> T cells from one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice were transduced with STAT5A retrovirus, then processed for RNA-seq. Contour plots (left) show correlation between the transduction marker (hNGFR) and IL-2R&alpha; protein. Volcano plot (middle) shows log2 fold changes and variances for all transcripts relative to control retrovirus. Those exhibiting &gt;1.5 fold change and p&lt;0.05 are depicted in blue. Dotted red lines indicate 2 fold change and 0.05 <i fontsize="11.0pt">p</i> value. Histogram (right) shows STAT5 paralog preference for transcripts mobilized by ectopic STAT5A. Dotted red line denotes equivalence and number indicates median paralog preference. (<span class="partLabel">B</span>) Heat map shows selected transcripts in STAT5A-transduced helper T cells (top row) or IL-2 treated <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient cells (bottom rows; from <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6</span>). Data are presented as the log2 fold change relative to controls (not shown). (<span class="partLabel">C</span>) GSEA plots show enrichment of STAT5A-dependent (left) or STAT5B-dependent (right) genes within the STAT5A-RV dataset. (<span class="partLabel">A&ndash;C</span>). RNA-seq analysis is compiled from 2 biological replicates. (<span class="partLabel">D</span>) CD4<sup fontsize="11.0pt">+</sup> T cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice were transduced with control (top row) or STAT5A (bottom row) retrovirus under iTreg polarizing conditions. Contour plots show total STAT5 and FOXP3 protein levels in transduced cells. (<span class="partLabel">E</span>) Histograms denote IL-2R&alpha; protein levels on transduced cells. (<span class="partLabel">D&ndash;E</span>) Shown is one of two independent experiments.</p><img src="/ecs/article/80712/resources/elife08384f010.jpg" data-mce-src="/ecs/article/80712/resources/elife08384f010.jpg" data-primary-extension="tif" data-width="1890" data-height="2142" data-id="1939622" alt="ecs/article/80712/resources/elife08384f010.jpg" height="831" width="733"/></div><div class="jrnlFigBlock" id="BLK_F10-S1" data-stream-name="a_F10-S1"><p class="jrnlFigCaption" id="F10-S1"><span class="label">Figure 10&mdash;figure supplement 1.</span> STAT5 paralog dose tips the balance between effector and regulatory T cell programs.</p><p class="jrnlFigCaption">CD4<sup fontsize="11.0pt">+</sup> T cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice were were transduced with control or STAT5A retrovirus under (<span class="partLabel">A</span>) non-polarizing or (<span class="partLabel">B</span>) Th17-polarizing conditions. Contour plots denote FOXP3 and IL-17A<sup fontsize="11.0pt"/>protein levels in transduced cells. Shown is one of two independent experiments.</p><img src="/ecs/article/80712/resources/elife08384fs010.jpg" data-mce-src="/ecs/article/80712/resources/elife08384fs010.jpg" data-primary-extension="tif" data-width="1890" data-height="1946" data-id="1939382" alt="ecs/article/80712/resources/elife08384fs010.jpg" height="755" width="733"/></div><p class="jrnlSecPara">STAT5 is essential for immunological tolerance (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R44 ">Mahmud et al., 2013</span>). This principle is well illustrated in humans with congenital <i fontsize="11.0pt">STAT5B</i> defects, who typically manifest a range of autoimmune symptoms (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R26 ">Kanai et al., 2012</span>), and is further supported by the present work, which demonstrates that <i fontsize="11.0pt">Stat5b</i> deficiency leads to spontaneous kidney disease in mice. The link between STAT5 and autoimmunity is often attributed to its role downstream of IL-2/IL-2R&alpha; in Treg cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R45 ">Malek and Castro, 2010</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R44 ">Mahmud et al., 2013</span>). Our work clearly endorses this viewpoint and brings to mind the autoimmune phenotype of Treg-deficient <i fontsize="11.0pt">Scurfy</i> mice which, like <i fontsize="11.0pt">Stat5b</i>-deficient mice<i fontsize="11.0pt">, </i>exhibit both autoantibodies and kidney disease {Aschermann:2013gn}. We demonstrate that, similar to <i fontsize="11.0pt">STAT5B</i>-deficient humans (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R8 ">Cohen et al., 2006</span>), Treg cells are functionally compromised in <i fontsize="11.0pt">Stat5b</i>-deficient mice, but, surprisingly, the baseline frequency of FOXP3<sup fontsize="11.0pt">+</sup> cells was not reduced, likely reflecting immunological and/or environmental differences between the two species. We also present new ideas about why STAT5-deficient Treg cells are impaired. First, they acquire the ability to produce IL-2, a cytokine that is typically restricted in Treg cells (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R45 ">Malek and Castro, 2010</span>). This finding is consistent with previous studies demonstrating that STAT5 can suppress IL-2 production by conventional T cells and, given that Treg cells are thought to operate, in part, by consuming IL-2, it provides one explanation for their ineffectiveness (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R65 ">Villarino et al., 2007</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R52 ">Pandiyan et al., 2007</span>). Second, they fail to express TBX21, a transcription factor that is required to limit Th1-type T cell responses (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R30 ">Koch et al., 2009</span>). Previous studies have shown that STAT1-activating cytokines (e.g. interferons, IL-27) can induce TBX21 in Treg cells but we are the first to show that IL-2, a STAT5-activating cytokine, can do it (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R16 ">Hall et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R31 ">Koch et al., 2012</span>).</p><p class="jrnlSecPara">Aside from its role in Treg cells, STAT5 promotes immunological tolerance via effector cell-intrinsic mechanisms. Given the dramatic accumulation of Tfh cells in our STAT5 mutants, the capacity of Tfh cells to promote autoimmunity, and recent work showing that STAT5 can suppress Tfh differentiation (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R2 ">Ballesteros-Tato et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R23 ">Johnston et al., 2012</span>), we conclude that exaggerated Tfh responses factor heavily in the autoantibody responses and attendant kidney pathology seen in <i fontsize="11.0pt">Stat5b</i>-deficient mice. Our data also suggest an intimate relationship between STAT5 and BCL6, the &lsquo;master&rsquo; transcription factor for Tfh cells. We report that STAT5 directly engages the <i fontsize="11.0pt">Bcl6</i> locus, where it likely acts as a transcriptional repressor, and that STAT5 binding sites are often enriched for BCL6 motifs, consistent with published accounts of co-localization between these two transcription factors (Y. <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R77 ">Zhang et al., 2012</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R39 ">Liao et al., 2014</span>). These findings strongly implicate Tfh cells in the pathogenesis of <i fontsize="11.0pt">Stat5b</i>-deficient mice but, since these are germline &lsquo;knockouts&rsquo;, we must consider the (likely) possibility that intrinsic defects in other cell types contribute to the autoimmune phenotype. For instance, multiple dendritic cell subsets are known to be dysregulated in <i fontsize="11.0pt">Stat5-</i> or <i fontsize="11.0pt">Jak3</i>-deficient mice (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R13 ">Esashi et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R70 ">Yamaoka, 2005</span>), and its influence on non-immune cells, particularly downstream of hormone receptors, cannot be ignored (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R33 ">Kuhrt and Wojchowski, 2015</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R18 ">Hennighausen and Robinson, 2008</span>).</p><p class="jrnlSecPara">Because of its prominent role within the immune system, STAT5 has long been viewed as an attractive target for therapeutic intervention. Clinical use of STAT5-activating cytokines and growth factors (e.g. IL-2, erythropoetin) is now commonplace and the recent approval of Jak3 inhibitors for the treatment of autoimmune disease and malignancy points to sustained interest in this pathway (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R66 ">Villarino et al., 2015</span>). Consequently, a detailed understanding of how STAT5 signaling works is imperative not only to inform new drugs, but also to improve existing regimens. The present study yields multiple clinically relevant insights and, in particular, raises two key issues that should be considered. First, partial inhibition of STAT5 expression and or activity may be sufficient to have desired effects on immune cell function. Second, targeting of STAT5A may be safer (though perhaps less robust) than targeting of STAT5B. Therefore, taking a broad view, our findings provide a molecular rationale for exploiting STAT5 paralog redundancy in clinical settings.</p><h1 class="jrnlHead1" bold="true" patternhead="true">Materials &amp; methods</h1><h2 class="jrnlHead2">Experimental Animals</h2><p class="jrnlSecPara">STAT5 mutants were generated as described (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R69 ">Yamaji et al., 2013</span>). Briefly, mice lacking the entire <i fontsize="11.0pt">Stat5</i> locus (<i fontsize="11.0pt">Stat5a/b<sup>+/-</sup></i>) were crossed with mice lacking one-allele of <i fontsize="11.0pt">Stat5a (Stat5a<sup>+/- </sup>Stat5b<sup>+/+</sup></i>) or <i fontsize="11.0pt">Stat5b (Stat5a<sup>+/+</sup> Stat5b<sup>+/-</sup></i>) to produce 8 combinations of <i fontsize="11.0pt">Stat5</i> alleles (<span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1A</span>). We refer to each genotype according to the total number of <i fontsize="11.0pt">Stat5</i> alleles that are retained. For example, two-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice lack both <i fontsize="11.0pt">Stat5a</i> alleles but retain two <i fontsize="11.0pt">Stat5b</i> alleles (<i fontsize="11.0pt">Stat5a<sup>-/-</sup> Stat5b<sup>+/+</sup></i>), while one-allele <i fontsize="11.0pt">Stat5a</i>-deficient mice lack both <i fontsize="11.0pt">Stat5a</i> alleles but retain one <i fontsize="11.0pt">Stat5b</i> allele (<i fontsize="11.0pt">Stat5a<sup>-/-</sup> Stat5b<sup>+/-</sup></i>). CD45.1<sup fontsize="11.0pt">+ </sup>C57BL/6 mice were purchased from Jackson Labs (Bar Harbor, ME). Animals were handled in accordance with NIH guidelines and all experiments approved by the NIAMS Animal Care and Use Committee.</p><h2 class="jrnlHead2">Blood, urine and lymphoid tissue analysis</h2><p class="jrnlSecPara">Complete blood counts were taken from 8&nbsp;to&nbsp;12-week old mice (NIH Clinical Center, Division of Veterinary Services, Bethesda, MD). Anti-double stranded DNA antibodies were measured in serum collected from 4&nbsp;to&nbsp;6 month old mice (Calbiotech, Spring Valley, CA). Albumin/creatinine ratio was measured in urine collected from 4&nbsp;to&nbsp;6 month old mice (Exocell, Philadelphia, PA). Spleen and lymph node (cervical, axillary, brachial and inguinal) cellularity was measured in 8&ndash;12 week old mice using a Nexcelom X1 Cellometer (Lawrence, MA).</p><h2 class="jrnlHead2">Histology</h2><p class="jrnlSecPara">Kidneys were dissected from 4&nbsp;to&nbsp;6-month old mice, fixed, embedded in paraffin, sectioned and stained with haematoxylin and eosin (American Histolabs, Gaithersburg, MD). Blinded scoring was performed by a veterinary pathologist (Diagnostic &amp; Research Services Branch, National Institutes of Health, Bethesda, MD). Specimens from at least 3 mice per genotype were inspected. Micrograph images were collected using a BioRevo BZ-9000 digital microscope (Keyence, Itasca, IL).</p><h2 class="jrnlHead2">Flow cytometry</h2><p class="jrnlSecPara">For surface proteins, cells were stained directly ex vivo with fluorochrome labelled anti-mouse CD3&epsilon;, CD4, CD8&alpha;, CD25 (IL-2R&alpha;), CD44, CD45R (B220), CD95 (FAS), CD127 (IL-7R), CD185 (CXCR5), CD279 (PD1), GL-7, and IgD. For intracellular proteins, cells were fixed and permeabilized using transcription factor staining buffer set (eBioscience, San Diego, CA), then stained with fluorochrome labelled anti-mouse FOXP3 and/or TBX21. For cytokine production, cells were stimulated with Phorbol 12-myristate 13-acetate and ionomycin for 4&thinsp;hr (50 ng/ml and 500 ng/ml, respectively; Sigma-Aldrich, St. Louis, MO), treated with Brefeldin A for 2&thinsp;hr (10&thinsp;&mu;g/ml; Sigma-Aldrich), fixed (2% formaldehyde; Sigma-Aldrich), permeabilized (0.25% Saponin; Sigma/Aldrich), and stained with fluorochrome-labelled anti-mouse IFN-&#611;, IL-2 and/or IL-17A. For IL-2R&alpha; induction, naive CD4<sup fontsize="11.0pt">+</sup> CD44<sup fontsize="11.0pt">low</sup> CD25<sup fontsize="11.0pt">-</sup> cells were purified from pooled lymph nodes and spleens using a FACS Aria Cell Sorter (&gt;98% purity; BD Biosciences, San Diego, CA). These were stimulated with plate-bound anti-CD3 (10&thinsp;&mu;g/ml; Clone 17A2) and soluble anti-CD28 (1&thinsp;&mu;g/ml; Clone 37.51) in the presence of soluble anti-mouse IL-2, IL-4 and IFN-&#611; (10&thinsp;&mu;g/ml each; Clones S4B6, BVD6-24G2 and XMG1.2; BioXcell, West Lebanon, NH) for 18&thinsp;hr, then treated with human IL-2 (100 units/ml; NIH/NCI BRB Preclinical Repository) or mouse IL-6 (20 ng/ml; eBioscience) for 18&thinsp;hr and stained with fluorochrome labelled anti-mouse CD25. For tyrosine-phosphorylated STAT5, splenocytes were treated directly ex vivo with human IL-2 (100 units/ml) or mouse IL-7 (20 ng/ml; eBioscience) for 1&thinsp;hr, or stimulated with anti-CD3 and anti-CD28 in the presence of anti-mouse IL-2 for 18&thinsp;hr, then pulsed with human IL-2 for 1&thinsp;hr (100 units/ml). These were then fixed with 2% formaldehyde, permeabilized with 100% methanol and stained with Alexa Fluor 647-labelled anti-human/mouse pY694 STAT5 (Clone 47; BD Biosciences) in conjunction with fluorochrome labelled anti-mouse CD3&epsilon;, CD4, CD25, CD44, CD127 and/or FOXP3. Total STAT5 protein was measured in splenocytes directly ex vivo&nbsp;or following retroviral transduction of purified CD4<sup fontsize="11.0pt">+</sup> T cells (described below). In both cases, cells were fixed with 2% formaldehyde, permeabilized with 100% methanol, then stained with a rabbit polyclonal IgG that recognizes both STAT5A and STAT5B (sc-835; Santa Cruz Biotechnology, Santa Cruz, CA) in conjunction with fluorochrome labelled anti-mouse CD3&epsilon;, CD4, CD25, CD44, CD127, (IL-7R), CD185 (CXCR5), CD279 (PD1), IL-17A and/or FOXP3. Phycoerythrin-labelled goat anti-rabbit IgG was used for detection (ac-3739; Santa Cruz Biotechnology). Normal rabbit IgG was used as a negative control (ac-2027; Santa Cruz Biotechnology).</p><p class="jrnlSecPara">All fluorochrome-labelled antibodies were purchased from eBioscience, BD Biosciences or Biolegend (San Diego, CA), unless noted otherwise. Data were collected on a FACSverse cytometer (BD Biosciences) and analyzed using FlowJo software (FlowJo LLC, Ashland, OR). Compiled cytometry data are presented as scatter plots where each element represents a single replicate (horizontal line indicates the mean), or box plots where the the fold change for each replicate was calculated relative to WT controls and log 2 transformed (horizontal line indicates the mean and whiskers indicate minimum and maximum values). Cells were maintained in supplemented tissue culture medium (RPMI-1640 with 10% fetal calf serum, 1% sodium pyruvate, 1% nonessential amino acids, 0.1% &beta;-Mercaptoethanol, 100&thinsp;U/ml penicillin, 100 mg/ml streptomycin; Life Technologies, Grand Island, NY) and cultured at a density of 0.25&ndash;0.5 x 10<sup fontsize="11.0pt">6</sup> cells/ml in flat bottomed 96 well plates (200&thinsp;ml/ well; Sigma/Costar, St. Louis, MO).</p><h2 class="jrnlHead2">T regulatory cell assays</h2><p class="jrnlSecPara">For in vitro suppression assays, CD4<sup fontsize="11.0pt">+</sup> CD25<sup fontsize="11.0pt">high</sup> Neuropilin<sup fontsize="11.0pt">+</sup> Treg cells were sorted from WT and one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice. Naive, CD4<sup fontsize="11.0pt">+</sup> CD44<sup fontsize="11.0pt">low</sup> CD25<sup fontsize="11.0pt">-</sup> responder cells were sorted from congenic CD45.1 mice and labelled with Carboxyfluorescein succinimidyl ester (CFSE; Sigma-Aldrich). CD11c<sup fontsize="11.0pt">+</sup> antigen presenting cells (APCs) were purified from WT mice using positive selection beads (Miltenyi Biotec). 5 x 10<sup fontsize="11.0pt">4</sup> CD4<sup fontsize="11.0pt">+</sup> responder cells were stimulated with soluble anti-mouse CD3&epsilon; (1&thinsp;&mu;g/ml) in round bottom 96-well plates containing 1 x 10<sup fontsize="11.0pt">4</sup> APCs and varying numbers of Treg cells, ranging from 5 x 10<sup fontsize="11.0pt">4</sup> (1:1 ratio) to 1.56 x 10<sup fontsize="11.0pt">3</sup> (1:32 ratio). After 96&thinsp;hr, cells were stained with fluorochrome-labelled anti-mouse CD4, CD45.1, and CD25. Percent suppression was calculated relative to WT controls and reflects the percentage of responder cells exhibiting at least one cell division. For 'Treg only' cultures, cells were stimulated with anti-CD3 and anti-CD28 in the presence human IL-2 (100 units/ml) for 72&thinsp;hr. For iTreg differentiation, naive CD4<sup fontsize="11.0pt">+</sup> CD44<sup fontsize="11.0pt">low</sup> CD25<sup fontsize="11.0pt">-</sup> cells were sorted and cultured for 72&thinsp;hr in the presence anti-CD3, anti-CD28, human TGF-&beta; (10 ng/ml; R&amp;D Systems, Minneapolis, MN), human IL-2 (100 units/ml) and anti-mouse IL-2, IL-4 and IFN-&#611;.</p><h2 class="jrnlHead2">RNA sequencing and transcriptome analysis</h2><p class="jrnlSecPara">Cell sorting was used to purify cells from pooled lymph nodes and spleens of WT and one-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice (&gt;99% purity). Ex vivo groups included naive T cells (CD4<sup fontsize="11.0pt">+</sup> CD44<sup fontsize="11.0pt">low</sup> CD25<sup fontsize="11.0pt">-</sup>) and Treg cells (CD4<sup fontsize="11.0pt">+</sup> CD25<sup fontsize="11.0pt">high</sup> Neuropilin<sup fontsize="11.0pt">+</sup>). In vitro groups included naive T cells that were treated with mouse IL-7 for 18&thinsp;hr, effector T cells that were stimulated with anti-CD3 and anti-CD28 in the presence of human IL-2 for 72&thinsp;hr, and induced Treg cells. All cultures included anti-mouse IL-2, IL-4 and IFN-&#611; (10&thinsp;&mu;g/ml each). Equal numbers of cells (0.5&ndash;2.5 x 10<sup fontsize="11.0pt">5</sup>) were collected for each replicate. These were lysed in Trizol reagent and total RNA isolated by phenol-chloroform extraction with GlycoBlue as co-precipitant (7-15&thinsp;&mu;g per sample; Life Technologies). Single-end libraries were prepared with 0.1&ndash;0.5&thinsp;&mu;g of total RNA using the TruSeq RNA Sample Preparation Kit V2 and sequenced for 50 cycles with a HiSeq 2500 instrument (4&ndash;6 samples multiplexed per lane; Illumina, San Diego, CA). 50 bp reads were then mapped onto mouse genome build mm9 using TopHat and further processed using Cufflinks (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R15 ">Garber et al., 2011</span>). 2&ndash;3 biological replicates were sequenced per genotype for every cell type and culture condition. QC-passing read counts are presented in <span class="jrnlSupplRef" data-citation-string=" SP1 ">Supplementary file 1</span>.</p><p class="jrnlSecPara">Datasets are normalized based on RPKM (reads per kilobase exon model per million mapped reads) and purged of micro-RNAs, sno-RNAs and sca-RNAs. To minimize fold-change artifacts caused by low abundance transcripts, a small offset (0.2&ndash;0.3; equal to the second quartile of each dataset) was added to all RPKM values (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R68 ">Warden, Yuan, and Wu, 2013</span>). When multiple fragments were detected for a single gene, only the most abundant (i.e. highest average RPKM across all 3 genotypes) was considered for downstream analyses. Transcripts with RPKM values of less than 1 for all genotypes within a given cell type or condition were excluded. Fold change and variance across genotypes and biological replicates were calculated using EdgeR (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R54 ">Robinson, McCarthy, and Smyth, 2009</span>). Transcripts were classified as differentially expressed if they exhibited a &gt;1.5 fold change and significant pairwise variance (p&lt;0.05) relative to WT controls. The 500 transcripts with greatest variance within each cell type or condition were used for multidimensional scaling (MDS) using the RobiNA software package (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R43 ">Lohse et al., 2012</span>).</p><p class="jrnlSecPara">A 'paralog preference' scale was devised to illustrate the relative impact of <i fontsize="11.0pt">Stat5a</i> - or <i fontsize="11.0pt">Stat5b-</i> deficiency. First, all transcripts that were differentially expressed in <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>- deficient cells (relative to WT controls) were pooled to generate a single list of STAT5-regulated genes for each cell type or condition. Next, the absolute fold change was calculated and multiplied by the higher of the two RPKMs (WT or KO), thereby generating a 'paralog score'. Note that the use of absolute fold change negates the distinction between up- and down-regulated genes, while the multiplication step improves the score for high-abundance transcripts. The paralog score for STAT5B was then divided by the paralog score for STAT5A and the resulting 'preference score' was log 2 transformed so that transcripts which are more impacted by the loss of STAT5B are assigned positive values while those which are more impacted by the loss of STAT5A are assigned negative values. All transcripts were then segregated into 12 bins according to preference scores (Bin 1 includes values of less than -5, Bin 2 ranges from -5 to -4, and so on). Data are displayed as histograms and the median preference score is indicated.</p><p class="jrnlSecPara">To identify &lsquo;paralog-specific&rsquo; transcripts, we first identified those exhibiting &gt;1.5 fold change and significant variance (p&lt;0.05) when comparing <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient cells directly to one another. Next, we refined this list by stipulating that transcripts must be differentially expressed in one KO relative to WT controls (&gt;1.5 fold change) but not in the other (&lt;1.2 fold change). Rare transcripts with opposite expression patterns (i.e. up-regulated in one genotype but down-regulated in the other) were excluded. Data are presented as pie charts.</p><p class="jrnlSecPara">All volcano plots, XY plots, histograms and pie charts were generated with the DataGraph software suite (Visual Data Tools, Inc.). Heat maps were generated with Multi Experiment Viewer (MeV; J. Craig Venter Institute, La Jolla, CA). Genome browser files (BigWig format) were processed to remove intronic reads using TopHat and are displayed with the Integrative Genomics Viewer (IGV; Broad Institute, Cambridge, MA).</p><p class="jrnlSecPara">GSEA analysis was performed as described (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R62 ">Subramanian et al., 2005</span>). Unabridged RNA-seq datasets were used in conjunction with the following user-generated Gene Sets: 1) Treg signature genes (132 members)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R19 ">Hill et al., 2007</span>), 2) IL-2-regulated, STAT5A-dependent genes (258 members)(from the comparison of WT and 'one copy' <i fontsize="11.0pt">Stat5a</i>-deficient T cells; <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6</span>), 3) IL-2-regulated, STAT5B-dependent genes (329 members)(from the comparison of WT and 'one copy' <i fontsize="11.0pt">Stat5b</i>-deficient T cells; <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6</span>). Enrichment score curves and member ranks were generated by the GSEA software (Broad Institute). Normalized enrichment score (NES), false discovery rate (FDR) and nominal p Value is shown on each plot.</p><p class="jrnlSecPara">See <span class="jrnlSupplRef" data-citation-string=" SP2 ">Supplementary file 2</span> for RPKM, fold change and p values for all experimental groups and conditions, <span class="jrnlSupplRef" data-citation-string=" SP3 ">Supplementary file 3</span> for paralog preference calculations and <span class="jrnlSupplRef" data-citation-string=" SP4 ">Supplementary file 4</span> for paralog-specific genes.</p><h2 class="jrnlHead2">Chromatin immuno-precipitation and DNA sequencing</h2><p>Cell sorting was used to purify naive CD4<sup fontsize="11.0pt">+</sup> CD44<sup fontsize="11.0pt">low</sup> CD25<sup fontsize="11.0pt">-</sup> cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> and two-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice (&gt;99% purity). These were stimulated with anti-CD3 and anti-CD28 in the presence of human IL-2 for 48&thinsp;hr (10&thinsp;U/ml with anti-mouse IL-2, IL-4 and IFN-&#611;), then pulsed with IL-2 (100&thinsp;U/ml) for one hour before fixing with 1% formaldehyde. They were then lysed (1 x 10<sup fontsize="11.0pt">7</sup> cells/sample), sonicated and immuno-precipated using a polyclonal rabbit anti-mouse IgG that recognizes both STAT5A and STAT5B (ab7969; Abcam, Cambridge, MA). Recovered STAT5-bound DNA fragments, along with un-precipitated &lsquo;input controls&rsquo;, were blunt-end ligated to adaptors and single-end libraries constructed using the NEBNext ChIP-Seq Library Prep for Illumina kit (New England Biolabs, Ipswich, MA). Sequencing was performed on a HiSeq 2500 instrument (50 cycles; Illumina) and short reads (50 bps) aligned using Bowtie (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R34 ">Langmead et al., 2009</span>). Non-redundant reads were mapped to the mouse genome (mm9) and aggregated into peaks and using MACS 1.4.2 (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R14 ">Feng et al., 2012</span>). Only peaks with &gt;3 fold enrichment over background and p<i fontsize="11.0pt"> v</i>alues &lt;0.00005 were called. Positive false discovery rates, or <i fontsize="11.0pt">q</i>-values, were calculated empirically for each peak and all were below 0.2% (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R61 ">Storey, 2003</span>). 2 biological replicates were sequenced per genotype. 'bamCorrelate' from deepTools 1.5 was used to calculate Spearman's rank correlation coefficients as a measure of inter-replicate variability (WT=0.82, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i>=0.83, <i fontsize="11.0pt">Stat5a</i>-deficient=0.82, <i fontsize="11.0pt">Stat5b</i>-deficient=0.81; all pairwise p-values &lt;2.2. x 10<sup fontsize="11.0pt">-16</sup>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R53 ">Ramirez et al., 2014</span>). Read depth for all replicates is presented in <span class="jrnlSupplRef" data-citation-string=" SP1 ">Supplementary file 1</span>.</p><p>Peaks were annotated to the nearest known gene using HOMER (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R76 ">Zhang et al., 2008</span>; <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R17 ">Heinz et al., 2010</span>). Localization was calculated as the percentage of peaks found within 10 kb intervals of the nearest transcriptional start sites and plotted as histograms. Direct comparison between experimental groups (i.e. peak overlap) was done with PAPST (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R3 ">Bible et al., 2015</span>). Circos plot was generated by inputing the number of shared peaks between experimental groups to the Circos Table Viewer (<a href="http://mkweb.bcgsc.ca/tableviewer" data-mce-href="http://mkweb.bcgsc.ca/tableviewer">http://mkweb.bcgsc.ca/tableviewer</a>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R32 ">Krzywinski et al., 2009</span>). Violin plot was generated by inputing tag numbers to the online BoxPlotR applet (<a href="http://boxplot.tyerslab.com" data-mce-href="http://boxplot.tyerslab.com">http://boxplot.tyerslab.com</a>)(<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R1 ">Editorial, 2014</span>). Transcription factor motif analysis was done with HOMER using an 'in house' database generated by applying de novo motif discovery to published ChIP-seq datasets. Genome browser files are displayed with IGV.</p><h2 class="jrnlHead2">ENCODE analysis</h2><p>Strand-specific RNA sequencing data were generated by the ENCODE Transcriptome Group from Cold Spring Harbor Laboratories (U.S.A.) and the Center for Genomic Regulation (Spain)(<a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424400999_9OI4vJsT1sakRAPyi9mNSC7V81zc&amp;g=wgEncodeCshlLongRnaSeq" data-mce-href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424400999_9OI4vJsT1sakRAPyi9mNSC7V81zc&amp;g=wgEncodeCshlLongRnaSeq">https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424400999_9OI4vJsT1sakRAPyi9mNSC7V81zc&amp;g=wgEncodeCshlLongRnaSeq</a>). DNAseI hyper-sensitivity data was generated by the University of Washington ENCODE group (<a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424401115_qgaAWZ6Xs38F1laFE3UuHnvAG7AS&amp;g=wgEncodeUwDnase" data-mce-href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424401115_qgaAWZ6Xs38F1laFE3UuHnvAG7AS&amp;g=wgEncodeUwDnase">https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=424401115_qgaAWZ6Xs38F1laFE3UuHnvAG7AS&amp;g=wgEncodeUwDnase</a>). Data are used in accordance with the ENCODE data release policy (<span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R73 ">Yue et al., 2014</span>) and visualized with the UCSC genome browser, focusing on the mouse <i fontsize="11.0pt">Stat5a/b</i> locus (chr11:100642045-100746483).</p><h2 class="jrnlHead2">Retroviral gene transduction</h2><p>Retroviral vector expressing phosphatase-insensitive STAT5A was generated as described ( <span class="jrnlBibRef" data-type="ruleset" data-citation-string=" R78 ">Zhu et al., 2003</span>). Plasmids were transfected into Phoenix packaging cells using Lipofectamine (Invitrogen) and the resulting viral supernatants used to transduce CD4<sup fontsize="11.0pt">+</sup> cells from WT, <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> or one-allele <i fontsize="11.0pt">Stat5b</i>-deficient mice. These were stimulated (anti-CD3/CD28) in the presence of anti-mouse IL-2 for 48&thinsp;hr, exposed to viral supernatant for 1&thinsp;hr (at 2200&thinsp;rpm, 18&deg;C), and cultured for an additional 48&thinsp;hr in the presence of human IL-2 (100&thinsp;U/ml). For some experiments, cells were cultured under iTreg (10 ng/ml human TGF-&beta;) or Th17 (2.5 ng/ml human TGF-&beta; + 20 ng/ml mouse IL-6) polarizing conditions before and after transduction (both in the presence of anti-mouse IL-4 and IFN-&#611;). For RNA-seq, 1-2 x 10<sup fontsize="11.0pt">5</sup> cells expressing the bicistronic transduction marker (human NGFR) were purified by cell sorting. Transcripts that were significantly impacted (&gt;1.5 fold change, p&lt;0.05) by ectopic STAT5A relative to empty vector were enumerated using EdgeR.</p><h2 class="jrnlHead2">Relative paralog measurements</h2><p>Transcriptome data for CD4<sup fontsize="11.0pt">+</sup> naive and Treg cells was sourced from: 1) Immunological Genome Project (mouse microarrays: <a href="http://www.immgen.org" data-mce-href="http://www.immgen.org">http://www.immgen.org</a>), 2) EMBL-EBI Expression Atlas (mouse RNA-seq: <a href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2582" data-mce-href="https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2582">https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2582</a>), 3) BioGPS Primary Cell Atlas (human microarrays: <a href="http://biogps.org/dataset/BDS_00013/" data-mce-href="http://biogps.org/dataset/BDS_00013/">http://biogps.org/dataset/BDS_00013/</a>), and 4) our RNA-seq catalogue (described above; <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6</span>). Normalized expression values (microarray signal intensity or FPKM) for <i fontsize="11.0pt">Stat5a</i> and <i fontsize="11.0pt">Stat5b</i> were first divided by one another to generate a paralog ratio which was then converted to a percentage (% total STAT5 mRNA accounted for by each paralog) and presented as pie charts.</p><p>Total STAT5 protein was measured by flow cytometry in naive (CD3&epsilon;<sup fontsize="11.0pt">+ </sup>CD4<sup fontsize="11.0pt">+ </sup>CD44<sup fontsize="11.0pt">low</sup> IL-7R<sup fontsize="11.0pt">+</sup>), Treg (CD3&epsilon;<sup fontsize="11.0pt"> + </sup>CD4<sup fontsize="11.0pt">+ </sup>FOXP3<sup fontsize="11.0pt">+</sup>), Tfh (CD3&epsilon;<sup fontsize="11.0pt"> + </sup>CD4<sup fontsize="11.0pt">+ </sup>PD1<sup fontsize="11.0pt">+</sup> CXCR5<sup fontsize="11.0pt">high) </sup>and effector/memory (CD3&epsilon;<sup fontsize="11.0pt"> +</sup>CD4<sup fontsize="11.0pt">+</sup>CD44<sup fontsize="11.0pt">high</sup>) T cells from one- or three-allele <i fontsize="11.0pt">Stat5a</i>- or <i fontsize="11.0pt">Stat5b</i>-deficient mice, as well as <i fontsize="11.0pt">Stat5a/b<sup>het</sup></i> mice and WT controls. Mean fluorescence intensity (MFI) was first divided by the baseline (i.e. WT controls) to generate 'fold change' values which, in turn, were divided across <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient genotypes bearing the same total number of alleles. The resulting paralog ratios (one for one-allele cells and one for three-allele cells) were then averaged, converted to a percentage (% total STAT5 protein accounted for by each paralog) and presented as pie charts.</p><h2 class="jrnlHead2">Statistics</h2><p>Unpaired ANOVA was used to quantify statistical deviation between experimental groups. In all figures, black asterisks denote significant differences (p&lt;0.05) between the indicated group and WT controls. Orange asterisks denote significant differences between <i fontsize="11.0pt">Stat5a</i>- and <i fontsize="11.0pt">Stat5b</i>-deficient mice bearing the same total number of STAT5 alleles.</p><h2 class="jrnlHead2">Data deposition</h2><p>All sequencing data have been deposited to the Gene Expression Omnibus under the accession number GSE77656.</p></div><div class="back" data-edited="true"><h1 class="jrnlAckHead">Acknowledgements</h1><p class="jrnlAckPara">We thank members of the O'Shea lab for helpful discussions, Victoria Hoffman for pathology scoring, Gustavo Gutierrez-Cruz for sequencing and the NIAMS Flow Cytometry Group for cell sorting. This study utilized the high-performance computational capabilities of the Helix System at the National Institutes of Health, Bethesda, MD (<a href="http://helix.nih.gov" data-mce-href="http://helix.nih.gov">http://helix.nih.gov</a>) and was supported by NIDDK and NIAMS intramural research programs. JJO'S and the NIH hold patents related to therapeutic targeting of Jak kinases and have a Collaborative Research Agreement and Development Award with Pfizer Inc.</p><h1 class="jrnlRefHead" bold="true">References</h1><p class="jrnlRefText noPMID" seqid="1" id="R2" data-seq-id="R1" sortby="Ballesteros-Tato 2012" data-old-id="R1" data-cited="true" movetopanel="yes" data-doi="10.1016/j.immuni.2012.02.012"><span class="RefAuthor"><span class="RefSurName">Ballesteros-Tato</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Le&oacute;n</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Graf</span> <span class="RefGivenName">BA</span></span>, <span class="RefAuthor"><span class="RefSurName">Moquin</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Adams</span> <span class="RefGivenName">PS</span></span>, <span class="RefAuthor"><span class="RefSurName">Lund</span> <span class="RefGivenName">FE</span></span>, <span class="RefAuthor"><span class="RefSurName">Randall</span> <span class="RefGivenName">TD</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Interleukin-2 inhibits germinal center formation by limiting T follicular helper cell differentiation</span>. <span class="RefJournalTitle"><em>Elsevier Inc</em></span> <span class="RefVolume"><strong>36</strong></span>:<span class="RefFPage">847</span>&ndash;<span class="RefLPage">856</span>.</p><p class="jrnlRefText" seqid="2" id="R3" data-seq-id="R2" sortby="Bible 2015" data-old-id="R2" data-cited="true" movetopanel="yes" data-doi="10.1371/journal.pone.0127285" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25970601"><span class="RefAuthor"><span class="RefSurName">Bible</span> <span class="RefGivenName">PW</span></span>, <span class="RefAuthor"><span class="RefSurName">Kanno</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Wei</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Brooks</span> <span class="RefGivenName">SR</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Morasso</span> <span class="RefGivenName">MI</span></span>, <span class="RefAuthor"><span class="RefSurName">Loganantharaj</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Sun</span> <span class="RefGivenName">HW</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">PAPST, a user friendly and powerful Java platform for chIP-seq peak co-localization analysis and beyond</span>. <span class="RefJournalTitle"><em>PloS One</em></span> <span class="RefVolume"><strong>10</strong></span>:<span class="RefFPage">e0127285</span>.</p><p class="jrnlRefText noPMID" seqid="3" id="R4" data-seq-id="R3" sortby="Boucheron 1998" data-old-id="R3" data-cited="true" movetopanel="yes" data-doi="10.1074/jbc.273.51.33936" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9852045"><span class="RefAuthor"><span class="RefSurName">Boucheron</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Dumon</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Santos</span> <span class="RefGivenName">SC</span></span>, <span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Gisselbrecht</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Gouilleux</span> <span class="RefGivenName">F</span></span>. <span class="RefYear">1998</span>. <span class="RefArticleTitle">A single amino acid in the DNA binding regions of STAT5A and STAT5B confers distinct DNA binding specificities</span>. <span class="RefJournalTitle"><em><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:42 pm" data-cid="1462266721002" data-userid="cross-ref" data-username="cross-ref" data-time="1462266721000">The </span>Journal of Biological Chemistry</em></span> <span class="RefVolume"><strong>273</strong></span>:<span class="RefFPage">33936</span>&ndash;<span class="RefLPage">33941</span>.</p><p class="jrnlRefText" seqid="4" id="R5" data-seq-id="R4" sortby="Brewster 2014" data-old-id="R4" data-cited="true" data-doi="10.1016/j.cell.2014.02.022" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24612990" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Brewster</span> <span class="RefGivenName">RC</span></span>, <span class="RefAuthor"><span class="RefSurName">Weinert</span> <span class="RefGivenName">FM</span></span>, <span class="RefAuthor"><span class="RefSurName">Garcia</span> <span class="RefGivenName">HG</span></span>, <span class="RefAuthor"><span class="RefSurName">Song</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Rydenfelt</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Phillips</span> <span class="RefGivenName">R</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">The transcription factor titration effect dictates level of gene expression</span>. <span class="RefJournalTitle"><em>Cell</em></span> <span class="RefVolume"><strong>156</strong></span>:<span class="RefFPage">1312</span>&ndash;<span class="RefLPage">1323</span>.</p><p class="jrnlRefText noPMID" seqid="5" id="R6" data-seq-id="R5" sortby="Burchill 2006" data-old-id="R5" data-cited="true" movetopanel="yes" data-doi="10.4049/jimmunol.178.1.280"><span class="RefAuthor"><span class="RefSurName">Burchill</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Yang</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Vogtenhuber</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Blazar</span> <span class="RefGivenName">BR</span></span>, <span class="RefAuthor"><span class="RefSurName">Farrar</span> <span class="RefGivenName">MA</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">IL-2 receptor <span class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:43 pm" data-cid="1462266786002" data-userid="cross-ref" data-username="cross-ref" data-time="1462266786000">beta</span>-dependent STAT5 activation is required for the development of <span data-diff-rid="R121" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:43 pm" data-cid="1462266786003" data-userid="cross-ref" data-username="cross-ref" data-time="1462266786000">Foxp3</span><span data-diff-rid="R121" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:43 pm" data-cid="1462266786004" data-userid="cross-ref" data-username="cross-ref" data-time="1462266786000">foxp3</span>+ regulatory T cells</span>. <span class="RefJournalTitle"><em><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:43 pm" data-cid="1462266786005" data-userid="cross-ref" data-username="cross-ref" data-time="1462266786000">The </span>Journal of Immunology</em></span> <span class="RefVolume"><strong>178</strong></span>:<span class="RefFPage">280</span>&ndash;<span class="RefLPage">290</span>.</p><p class="jrnlRefText" seqid="6" id="R7" data-seq-id="R6" sortby="Caldenhoven 1998" data-old-id="R6" data-cited="true" data-doi="10.1002/stem.160397" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9831865" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Caldenhoven</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">van Dijk</span> <span class="RefGivenName">TB</span></span>, <span class="RefAuthor"><span class="RefSurName">Tijmensen</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Raaijmakers</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Lammers</span> <span class="RefGivenName">JW</span></span>, <span class="RefAuthor"><span class="RefSurName">Koenderman</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">de Groot</span> <span class="RefGivenName">RP</span></span>. <span class="RefYear">1998</span>. <span class="RefArticleTitle">Differential activation of functionally distinct STAT5 proteins by IL-5 and GM-CSF during eosinophil and neutrophil differentiation from human CD34+ hematopoietic stem cells</span>. <span class="RefJournalTitle"><em>Stem Cells</em></span> <span class="RefVolume"><strong>16</strong></span>:<span class="RefFPage">397</span>&ndash;<span class="RefLPage">403</span>.</p><p class="jrnlRefText noPMID" seqid="7" id="R8" data-seq-id="R7" sortby="Cohen 2006" data-old-id="R7" data-cited="true" movetopanel="yes" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/16920911" data-doi="10.4049/jimmunol.177.5.2770"><span class="RefAuthor"><span class="RefSurName">Cohen</span> <span class="RefGivenName">AC</span></span>, <span class="RefAuthor"><span class="RefSurName">Nadeau</span> <span class="RefGivenName">KC</span></span>, <span class="RefAuthor"><span class="RefSurName">Tu</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Hwa</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Dionis</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Bezrodnik</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Teper</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Gaillard</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Heinrich</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Krensky</span> <span class="RefGivenName">AM</span></span>, <span class="RefAuthor"><span class="RefSurName">Rosenfeld</span> <span class="RefGivenName">RG</span></span>, <span class="RefAuthor"><span class="RefSurName">Lewis</span> <span class="RefGivenName">DB</span></span>. <span class="RefYear">2006</span>. <span class="RefArticleTitle">Cutting edge: Decreased accumulation and regulatory function of CD4+ CD25(high) T cells in human STAT5b deficiency</span>. <span class="RefJournalTitle"><em>Journal of Immunology</em></span> <span class="RefVolume"><strong>177</strong></span>:<span class="RefFPage">2770</span>&ndash;<span class="RefLPage">2774</span> .</p><p class="jrnlRefText" seqid="8" id="R9" data-seq-id="R8" sortby="Crotty 2011" data-old-id="R8" data-cited="true" data-doi="10.1146/annurev-immunol-031210-101400" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/21314428" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Crotty</span> <span class="RefGivenName">S</span></span>. <span class="RefYear">2011</span>. <span class="RefArticleTitle">Follicular helper CD4 T cells (TFH)</span>. <span class="RefJournalTitle"><em>Annual Review of Immunology</em></span> <span class="RefVolume"><strong>29</strong></span>:<span class="RefFPage">621</span>&ndash;<span class="RefLPage">663</span>.</p><p class="jrnlRefText" seqid="9" id="R10" data-seq-id="R9" sortby="Cui 2004" data-old-id="R9" data-cited="true" data-doi="10.1128/MCB.24.18.8037-8047.2004" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/15340066" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Riedlinger</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Miyoshi</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Tang</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Deng</span> <span class="RefGivenName">CX</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">2004</span>. <span class="RefArticleTitle">Inactivation of Stat5 in mouse mammary epithelium during pregnancy reveals distinct functions in cell proliferation, survival, and differentiation</span>. <span class="RefJournalTitle"><em>Molecular and Cellular Biology</em></span> <span class="RefVolume"><strong>24</strong></span>:<span class="RefFPage">8037</span>&ndash;<span class="RefLPage">8047</span>.</p><p class="jrnlRefText" seqid="10" id="R11" data-seq-id="R10" sortby="Dooms 2007" data-old-id="R10" data-cited="true" data-doi="10.1084/jem.20062381" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17312008" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Dooms</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Wolslegel</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Lin</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Abbas</span> <span class="RefGivenName">AK</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Interleukin-2 enhances CD4+ T cell memory by promoting the generation of IL-7R alpha-expressing cells</span>. <span class="RefJournalTitle"><em>Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>204</strong></span>:<span class="RefFPage">547</span>&ndash;<span class="RefLPage">557</span>.</p><p class="jrnlRefText noPMID" seqid="11" id="R1" data-seq-id="R11" data-doi="10.1038/nmeth.2837" sortby=" 2014" data-old-id="R11" movetopanel="yes"><span class="RefCollaboration">Editorial</span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Kick the bar chart habit</span>. <span class="RefJournalTitle"><em>Nature Methods</em></span> <span class="RefVolume"><strong>11</strong></span>:<span class="RefFPage">113</span>.</p><p class="jrnlRefText" seqid="12" id="R12" data-seq-id="R12" sortby="Ehret 2001" data-old-id="R12" data-cited="true" data-doi="10.1074/jbc.M001748200" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/11053426" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Ehret</span> <span class="RefGivenName">GB</span></span>, <span class="RefAuthor"><span class="RefSurName">Reichenbach</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Schindler</span> <span class="RefGivenName">U</span></span>, <span class="RefAuthor"><span class="RefSurName">Horvath</span> <span class="RefGivenName">CM</span></span>, <span class="RefAuthor"><span class="RefSurName">Fritz</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Nabholz</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Bucher</span> <span class="RefGivenName">P</span></span>. <span class="RefYear">2001</span>. <span class="RefArticleTitle">DNA binding specificity of different STAT proteins. Comparison of in vitro specificity with natural target sites</span>. <span class="RefJournalTitle"><em>Journal of Biological Chemistry</em></span> <span class="RefVolume"><strong>276</strong></span>:<span class="RefFPage">6675</span>&ndash;<span class="RefLPage">6688</span>.</p><p class="jrnlRefText" seqid="13" id="R13" data-seq-id="R13" sortby="Esashi 2008" data-old-id="R13" data-cited="true" data-doi="10.1016/j.immuni.2008.02.013" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18342552" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Esashi</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">YH</span></span>, <span class="RefAuthor"><span class="RefSurName">Perng</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Qin</span> <span class="RefGivenName">XF</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">YJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Watowich</span> <span class="RefGivenName">SS</span></span>. <span class="RefYear">2008</span>. <span class="RefArticleTitle">The signal transducer STAT5 inhibits plasmacytoid dendritic cell development by suppressing transcription factor IRF8</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>28</strong></span>:<span class="RefFPage">509</span>&ndash;<span class="RefLPage">520</span>.</p><p class="jrnlRefText" seqid="14" id="R14" data-seq-id="R14" sortby="Feng 2012" data-old-id="R14" data-cited="true" movetopanel="yes" data-doi="10.1038/nprot.2012.101" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22936215"><span class="RefAuthor"><span class="RefSurName">Feng</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Qin</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">XS</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Identifying ChIP-seq enrichment using MACS</span>. <span class="RefJournalTitle"><em>Nature Protocols</em></span> <span class="RefVolume"><strong>7</strong></span>:<span class="RefFPage">1728</span>&ndash;<span class="RefLPage">1740</span>.</p><p class="jrnlRefText" seqid="15" id="R15" data-seq-id="R15" sortby="Garber 2011" data-old-id="R15" data-cited="true" data-doi="10.1038/nmeth.1613" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/21623353" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Garber</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Grabherr</span> <span class="RefGivenName">MG</span></span>, <span class="RefAuthor"><span class="RefSurName">Guttman</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Trapnell</span> <span class="RefGivenName">C</span></span>. <span class="RefYear">2011</span>. <span class="RefArticleTitle">Computational methods for transcriptome annotation and quantification using RNA-seq</span>. <span class="RefJournalTitle"><em>Nature Methods</em></span> <span class="RefVolume"><strong>8</strong></span>:<span class="RefFPage">469</span>&ndash;<span class="RefLPage">477</span>.</p><p class="jrnlRefText" seqid="16" id="R16" data-seq-id="R16" sortby="Hall 2012" data-old-id="R16" data-cited="true" movetopanel="yes" data-doi="10.1016/j.immuni.2012.06.014" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22981537"><span class="RefAuthor"><span class="RefSurName">Hall</span> <span class="RefGivenName">AO</span></span>, <span class="RefAuthor"><span class="RefSurName">Beiting</span> <span class="RefGivenName">DP</span></span>, <span class="RefAuthor"><span class="RefSurName">Tato</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">John</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Oldenhove</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Lombana</span> <span class="RefGivenName">CG</span></span>, <span class="RefAuthor"><span class="RefSurName">Pritchard</span> <span class="RefGivenName">GH</span></span>, <span class="RefAuthor"><span class="RefSurName">Silver</span> <span class="RefGivenName">JS</span></span>, <span class="RefAuthor"><span class="RefSurName">Bouladoux</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Stumhofer</span> <span class="RefGivenName">JS</span></span>, <span class="RefAuthor"><span class="RefSurName">Harris</span> <span class="RefGivenName">TH</span></span>, <span class="RefAuthor"><span class="RefSurName">Grainger</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Wojno</span> <span class="RefGivenName">ED</span></span>, <span class="RefAuthor"><span class="RefSurName">Wagage</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Roos</span> <span class="RefGivenName">DS</span></span>, <span class="RefAuthor"><span class="RefSurName">Scott</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Turka</span> <span class="RefGivenName">LA</span></span>, <span class="RefAuthor"><span class="RefSurName">Cherry</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Reiner</span> <span class="RefGivenName">SL</span></span>, <span class="RefAuthor"><span class="RefSurName">Cua</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Belkaid</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Elloso</span> <span class="RefGivenName">MM</span></span>, <span class="RefAuthor"><span class="RefSurName">Hunter</span> <span class="RefGivenName">CA</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">The cytokines interleukin 27 and interferon-&gamma; promote distinct Treg cell populations required to limit infection-induced pathology</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>37</strong></span>:<span class="RefFPage">511</span>&ndash;<span class="RefLPage">523</span>.</p><p class="jrnlRefText" seqid="17" id="R17" data-seq-id="R17" sortby="Heinz 2010" data-old-id="R17" data-cited="true" data-doi="10.1016/j.molcel.2010.05.004" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/20513432" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Heinz</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Benner</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Spann</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Bertolino</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Lin</span> <span class="RefGivenName">YC</span></span>, <span class="RefAuthor"><span class="RefSurName">Laslo</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Cheng</span> <span class="RefGivenName">JX</span></span>, <span class="RefAuthor"><span class="RefSurName">Murre</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Singh</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Glass</span> <span class="RefGivenName">CK</span></span>. <span class="RefYear">2010</span>. <span class="RefArticleTitle">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</span>. <span class="RefJournalTitle"><em>Molecular Cell</em></span> <span class="RefVolume"><strong>38</strong></span>:<span class="RefFPage">576</span>&ndash;<span class="RefLPage">589</span>.</p><p class="jrnlRefText" seqid="18" id="R18" data-seq-id="R18" sortby="Hennighausen 2008" data-old-id="R18" data-cited="true" data-doi="10.1101/gad.1643908" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18347089" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>. <span class="RefYear">2008</span>. <span class="RefArticleTitle">Interpretation of cytokine signaling through the transcription factors STAT5A and STAT5B</span>. <span class="RefJournalTitle"><em>Genes &amp; Development</em></span> <span class="RefVolume"><strong>22</strong></span>:<span class="RefFPage">711</span>&ndash;<span class="RefLPage">721</span>.</p><p class="jrnlRefText" seqid="19" id="R19" data-seq-id="R19" sortby="Hill 2007" data-old-id="R19" data-cited="true" data-doi="10.1016/j.immuni.2007.09.010" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18024188" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Hill</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Feuerer</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Tash</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Haxhinasto</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Perez</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Melamed</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Mathis</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Benoist</span> <span class="RefGivenName">C</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Foxp3 transcription-factor-dependent and -independent regulation of the regulatory T cell transcriptional signature</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>27</strong></span>:<span class="RefFPage">786</span>&ndash;<span class="RefLPage">800</span>.</p><p class="jrnlRefText" seqid="20" id="R20" data-seq-id="R20" sortby="Hoelbl 2006" data-old-id="R20" data-cited="true" data-doi="10.1182/blood-2005-09-3596" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/16493008" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Hoelbl</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Kovacic</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Kerenyi</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Simma</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Warsch</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Beug</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Sexl</span> <span class="RefGivenName">V</span></span>. <span class="RefYear">2006</span>. <span class="RefArticleTitle">Clarifying the role of Stat5 in lymphoid development and Abelson-induced transformation</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>107</strong></span>:<span class="RefFPage">4898</span>&ndash;<span class="RefLPage">4906</span>.</p><p class="jrnlRefText noPMID" seqid="21" id="R21" data-seq-id="R21" sortby="Imada 1998" data-old-id="R21" data-cited="true" movetopanel="yes" data-doi="10.1084/jem.188.11.2067" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9841920"><span class="RefAuthor"><span class="RefSurName">Imada</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Bloom</span> <span class="RefGivenName">ET</span></span>, <span class="RefAuthor"><span class="RefSurName">Nakajima</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Horvath-Arcidiacono</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Udy</span> <span class="RefGivenName">GB</span></span>, <span class="RefAuthor"><span class="RefSurName">Davey</span> <span class="RefGivenName">HW</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">1998</span>. <span class="RefArticleTitle">Stat5b is essential for natural killer cell-mediated proliferation and cytolytic activity</span>. <span class="RefJournalTitle"><em><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:45 pm" data-cid="1462266911002" data-userid="cross-ref" data-username="cross-ref" data-time="1462266911000">The </span>Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>188</strong></span>:<span class="RefFPage">2067</span>&ndash;<span class="RefLPage">2074</span>.</p><p class="jrnlRefText" seqid="22" id="R22" data-seq-id="R22" sortby="Jenks 2013" data-old-id="R22" data-cited="true" movetopanel="yes" data-doi="10.1016/j.clim.2013.04.014" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/23773921"><span class="RefAuthor"><span class="RefSurName">Jenks</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Seki</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Kanai</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Huang</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Morgan</span> <span class="RefGivenName">AA</span></span>, <span class="RefAuthor"><span class="RefSurName">Scalco</span> <span class="RefGivenName">RC</span></span>, <span class="RefAuthor"><span class="RefSurName">Nath</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Bucayu</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Wit</span> <span class="RefGivenName">JM</span></span>, <span class="RefAuthor"><span class="RefSurName">Al-Herz</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Ramadan</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Jorge</span> <span class="RefGivenName">AA</span></span>, <span class="RefAuthor"><span class="RefSurName">Bacchetta</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hwa</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Rosenfeld</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Nadeau</span> <span class="RefGivenName">KC</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">Differentiating the roles of STAT5B and STAT5A in human CD4+ T cells</span>. <span class="RefJournalTitle"><em>Clinical Immunology</em></span> <span class="RefVolume"><strong>148</strong></span>:<span class="RefFPage">227</span>&ndash;<span class="RefLPage">236</span>.</p><p class="jrnlRefText" seqid="23" id="R23" data-seq-id="R23" sortby="Johnston 2012" data-old-id="R23" data-cited="true" data-doi="10.1084/jem.20111174" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22271576" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Johnston</span> <span class="RefGivenName">RJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Choi</span> <span class="RefGivenName">YS</span></span>, <span class="RefAuthor"><span class="RefSurName">Diamond</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Yang</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Crotty</span> <span class="RefGivenName">S</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">STAT5 is a potent negative regulator of TFH cell differentiation</span>. <span class="RefJournalTitle"><em>Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>209</strong></span>:<span class="RefFPage">243</span>&ndash;<span class="RefLPage">250</span>.</p><p class="jrnlRefText noPMID" seqid="24" id="R24" data-seq-id="R24" sortby="Kagami 2000" data-old-id="R24" data-cited="true" movetopanel="yes" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10666213"><span class="RefAuthor"><span class="RefSurName">Kagami</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Nakajima</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Kumano</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Suzuki</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Suto</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Imada</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Davey</span> <span class="RefGivenName">HW</span></span>, <span class="RefAuthor"><span class="RefSurName">Saito</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Takatsu</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Iwamoto</span> <span class="RefGivenName">I</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">Both stat5a and stat5b are required for antigen-induced eosinophil and T-cell recruitment into the tissue</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>95</strong></span>:<span class="RefFPage">1370</span>&ndash;<span class="RefLPage">1377</span>.</p><p class="jrnlRefText noPMID" seqid="25" id="R25" data-seq-id="R25" sortby="Kagami 2001" data-old-id="R25" data-cited="true" movetopanel="yes" data-doi="10.1182/blood.V97.8.2358" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/11290598"><span class="RefAuthor"><span class="RefSurName">Kagami</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Nakajima</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Suto</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Hirose</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Suzuki</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Morita</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Kato</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">Saito</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Kitamura</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Iwamoto</span> <span class="RefGivenName">I</span></span>. <span class="RefYear">2001</span>. <span class="RefArticleTitle">Stat5a regulates T helper cell differentiation by several distinct mechanisms</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>97</strong></span>:<span class="RefFPage">2358</span>&ndash;<span class="RefLPage">2365</span>.</p><p class="jrnlRefText" seqid="26" id="R26" data-seq-id="R26" sortby="Kanai 2012" data-old-id="R26" data-cited="true" data-doi="10.3389/fimmu.2012.00234" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22912632" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kanai</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Jenks</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Nadeau</span> <span class="RefGivenName">KC</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">The STAT5b pathway defect and autoimmunity</span>. <span class="RefJournalTitle"><em>Frontiers in Immunology</em></span> <span class="RefVolume"><strong>3</strong></span>:<span class="RefELocation">234</span>.</p><p class="jrnlRefText" seqid="27" id="R27" data-seq-id="R27" sortby="Kanai 2014" data-old-id="R27" data-cited="true" data-doi="10.1371/journal.pone.0086790" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24497979" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kanai</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Seki</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Jenks</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Kohli</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Kawli</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Martin</span> <span class="RefGivenName">DP</span></span>, <span class="RefAuthor"><span class="RefSurName">Snyder</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Bacchetta</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Nadeau</span> <span class="RefGivenName">KC</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Identification of STAT5A and STAT5B target genes in human T cells</span>. <span class="RefJournalTitle"><em>PloS One</em></span> <span class="RefVolume"><strong>9</strong></span>:<span class="RefFPage">e86790</span>.</p><p class="jrnlRefText noPMID" seqid="28" id="R28" data-seq-id="R28" sortby="Kelly 2003" data-old-id="R28" data-cited="true" data-doi="10.4049/jimmunol.170.1.210" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kelly</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Spolski</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Imada</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Bollenbacher</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2003a</span>. <span class="RefArticleTitle">A role for stat5 in CD8+ T cell homeostasis</span>. <span class="RefJournalTitle"><em>Journal of Immunology</em></span> <span class="RefVolume"><strong>170</strong></span>:<span class="RefFPage">210</span>&ndash;<span class="RefLPage">217</span>.</p><p class="jrnlRefText" seqid="29" id="R29" data-seq-id="R29" sortby="Kelly 2003" data-old-id="R29" data-doi="10.1084/jem.20021548" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/12835478" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kelly</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Spolski</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Kovanen</span> <span class="RefGivenName">PE</span></span>, <span class="RefAuthor"><span class="RefSurName">Suzuki</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Bollenbacher</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Pise-Masison</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Radonovich</span> <span class="RefGivenName">MF</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Jenkins</span> <span class="RefGivenName">NA</span></span>, <span class="RefAuthor"><span class="RefSurName">Copeland</span> <span class="RefGivenName">NG</span></span>, <span class="RefAuthor"><span class="RefSurName">Morse</span> <span class="RefGivenName">HC</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2003b</span>. <span class="RefArticleTitle">Stat5 synergizes with T cell receptor/antigen stimulation in the development of lymphoblastic lymphoma</span>. <span class="RefJournalTitle"><em>Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>198</strong></span>:<span class="RefFPage">79</span>&ndash;<span class="RefLPage">89</span>.</p><p class="jrnlRefText" seqid="30" id="R30" data-seq-id="R30" sortby="Koch 2009" data-old-id="R30" data-cited="true" data-doi="10.1038/ni.1731" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19412181" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Koch</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Tucker-Heard</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Perdue</span> <span class="RefGivenName">NR</span></span>, <span class="RefAuthor"><span class="RefSurName">Killebrew</span> <span class="RefGivenName">JR</span></span>, <span class="RefAuthor"><span class="RefSurName">Urdahl</span> <span class="RefGivenName">KB</span></span>, <span class="RefAuthor"><span class="RefSurName">Campbell</span> <span class="RefGivenName">DJ</span></span>. <span class="RefYear">2009</span>. <span class="RefArticleTitle">The transcription factor T-bet controls regulatory T cell homeostasis and function during type 1 inflammation</span>. <span class="RefJournalTitle"><em>Nature Immunology</em></span> <span class="RefVolume"><strong>10</strong></span>:<span class="RefFPage">595</span>&ndash;<span class="RefLPage">602</span>.</p><p class="jrnlRefText noPMID" seqid="31" id="R31" data-seq-id="R31" sortby="Koch 2012" data-old-id="R31" data-cited="true" movetopanel="yes" data-doi="10.1016/j.immuni.2012.05.031" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22960221"><span class="RefAuthor"><span class="RefSurName">Koch</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Thomas</span> <span class="RefGivenName">KR</span></span>, <span class="RefAuthor"><span class="RefSurName">Perdue</span> <span class="RefGivenName">NR</span></span>, <span class="RefAuthor"><span class="RefSurName">Smigiel</span> <span class="RefGivenName">KS</span></span>, <span class="RefAuthor"><span class="RefSurName">Srivastava</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Campbell</span> <span class="RefGivenName">DJ</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">T-bet(+) Treg cells undergo abortive Th1 cell differentiation due to impaired expression of IL-12 receptor &beta;2</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>37</strong></span>:<span class="RefFPage">501</span>&ndash;<span class="RefLPage">510</span>.</p><p class="jrnlRefText" seqid="32" id="R32" data-seq-id="R32" sortby="Krzywinski 2009" data-old-id="R32" data-cited="true" data-doi="10.1101/gr.092759.109" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19541911" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Krzywinski</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Schein</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Birol</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">Connors</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Gascoyne</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Horsman</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Jones</span> <span class="RefGivenName">SJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Marra</span> <span class="RefGivenName">MA</span></span>. <span class="RefYear">2009</span>. <span class="RefArticleTitle">Circos: an information aesthetic for comparative genomics</span>. <span class="RefJournalTitle"><em>Genome Research</em></span> <span class="RefVolume"><strong>19</strong></span>:<span class="RefFPage">1639</span>&ndash;<span class="RefLPage">1645</span>.</p><p class="jrnlRefText" seqid="33" id="R33" data-seq-id="R33" sortby="Kuhrt 2015" data-old-id="R33" data-cited="true" data-doi="10.1182/blood-2014-11-575357" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25887776" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Kuhrt</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Wojchowski</span> <span class="RefGivenName">DM</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">Emerging EPO and EPO receptor regulators and signal transducers</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>125</strong></span>.</p><p class="jrnlRefText" seqid="34" id="R34" data-seq-id="R34" sortby="Langmead 2009" data-old-id="R34" data-cited="true" movetopanel="yes" data-doi="10.1186/gb-2009-10-3-r25" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19261174"><span class="RefAuthor"><span class="RefSurName">Langmead</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Trapnell</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Pop</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Salzberg</span> <span class="RefGivenName">SL</span></span>. <span class="RefYear">2009</span>. <span class="RefArticleTitle">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <span class="RefJournalTitle"><em>Genome Biology</em></span> <span class="RefVolume"><strong>10</strong></span>:<span class="RefFPage">R25</span>.</p><p class="jrnlRefText" seqid="35" id="R35" data-seq-id="R35" sortby="Laurence 2007" data-old-id="R35" data-cited="true" data-doi="10.1016/j.immuni.2007.02.009" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17363300" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Laurence</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Tato</span> <span class="RefGivenName">CM</span></span>, <span class="RefAuthor"><span class="RefSurName">Davidson</span> <span class="RefGivenName">TS</span></span>, <span class="RefAuthor"><span class="RefSurName">Kanno</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Chen</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Yao</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Blank</span> <span class="RefGivenName">RB</span></span>, <span class="RefAuthor"><span class="RefSurName">Meylan</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Siegel</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Shevach</span> <span class="RefGivenName">EM</span></span>, <span class="RefAuthor"><span class="RefSurName">O'shea</span> <span class="RefGivenName">JJ</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Interleukin-2 signaling via STAT5 constrains T helper 17 cell generation</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>26</strong></span>:<span class="RefFPage">371</span>&ndash;<span class="RefLPage">381</span>.</p><p class="jrnlRefText" seqid="36" id="R36" data-seq-id="R36" sortby="Lenardo 1991" data-old-id="R36" data-cited="true" data-doi="10.1038/353858a0" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/1944559" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Lenardo</span> <span class="RefGivenName">MJ</span></span>. <span class="RefYear">1991</span>. <span class="RefArticleTitle">Interleukin-2 programs mouse alpha beta T lymphocytes for apoptosis</span>. <span class="RefJournalTitle"><em>Nature</em></span> <span class="RefVolume"><strong>353</strong></span>:<span class="RefFPage">858</span>&ndash;<span class="RefLPage">861</span>.</p><p class="jrnlRefText" seqid="37" id="R37" data-seq-id="R37" sortby="Liao 2008" data-old-id="R37" data-cited="true" data-doi="10.1038/ni.1656" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18820682" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Liao</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Schones</span> <span class="RefGivenName">DE</span></span>, <span class="RefAuthor"><span class="RefSurName">Oh</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Roh</span> <span class="RefGivenName">TY</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhao</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2008</span>. <span class="RefArticleTitle">Priming for T helper type 2 differentiation by interleukin 2-mediated induction of interleukin 4 receptor alpha-chain expression</span>. <span class="RefJournalTitle"><em>Nature Immunology</em></span> <span class="RefVolume"><strong>9</strong></span>:<span class="RefFPage">1288</span>&ndash;<span class="RefLPage">1296</span>.</p><p class="jrnlRefText" seqid="38" id="R38" data-seq-id="R38" sortby="Liao 2011" data-old-id="R38" data-cited="true" data-doi="10.1038/ni.2030" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/21516110" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Liao</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Lin</span> <span class="RefGivenName">JX</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2011</span>. <span class="RefArticleTitle">Modulation of cytokine receptors by IL-2 broadly regulates differentiation into helper T cell lineages</span>. <span class="RefJournalTitle"><em>Nature Immunology</em></span> <span class="RefVolume"><strong>12</strong></span>:<span class="RefFPage">551</span>&ndash;<span class="RefLPage">559</span>.</p><p class="jrnlRefText noPMID" seqid="39" id="R39" data-seq-id="R39" sortby="Liao 2014" data-old-id="R39" data-cited="true" movetopanel="yes" data-doi="10.1073/pnas.1301138111"><span class="RefAuthor"><span class="RefSurName">Liao</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Spolski</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Du</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">West</span> <span class="RefGivenName">EE</span></span>, <span class="RefAuthor"><span class="RefSurName">Ren</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Mitra</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">Opposing actions of IL-2 and IL-21 on Th9 differentiation correlate with their differential regulation of BCL6 expression</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>111</strong></span>:<span class="RefFPage">3508</span>&ndash;<span class="RefLPage">3513</span>.</p><p class="jrnlRefText" seqid="40" id="R40" data-seq-id="R40" sortby="Liongue 2012" data-old-id="R40" data-cited="true" data-doi="10.1371/journal.pone.0032777" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22412924" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Liongue</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Sullivan</span> <span class="RefGivenName">LA</span></span>, <span class="RefAuthor"><span class="RefSurName">Trengove</span> <span class="RefGivenName">MC</span></span>, <span class="RefAuthor"><span class="RefSurName">Ward</span> <span class="RefGivenName">AC</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Evolution of JAK-STAT pathway components: mechanisms and role in immune system development</span>. <span class="RefJournalTitle"><em>PloS One</em></span> <span class="RefVolume"><strong>7</strong></span>:<span class="RefELocation">e32777</span>.</p><p class="jrnlRefText noPMID" seqid="41" id="R41" data-seq-id="R41" sortby="Liu 1995" data-old-id="R41" data-cited="true" movetopanel="yes" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/7568026" data-doi="10.1073/pnas.92.19.8831"><span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>, <span class="RefAuthor"><span class="RefSurName">Gouilleux</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Groner</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">1995</span>. <span class="RefArticleTitle">Cloning and expression of Stat5 and an additional homologue (Stat5b) involved in prolactin signal transduction in mouse mammary tissue</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>92</strong></span>:<span class="RefFPage">8831</span>&ndash;<span class="RefLPage">8835</span> .</p><p class="jrnlRefText noPMID" seqid="42" id="R42" data-seq-id="R42" sortby="Liu 1997" data-old-id="R42" data-cited="true" movetopanel="yes" data-doi="10.1101/gad.11.2.179" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9009201"><span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>, <span class="RefAuthor"><span class="RefSurName">Wagner</span> <span class="RefGivenName">KU</span></span>, <span class="RefAuthor"><span class="RefSurName">Garrett</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Wynshaw-Boris</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">1997</span>. <span class="RefArticleTitle">Stat5a is mandatory for adult mammary gland development and lactogenesis</span>. <span class="RefJournalTitle"><em>Genes &amp; Development</em></span> <span class="RefVolume"><strong>11</strong></span>:<span class="RefFPage">179</span>&ndash;<span class="RefLPage">186</span>.</p><p class="jrnlRefText noPMID" seqid="43" id="R43" data-seq-id="R43" sortby="Lohse 2012" data-old-id="R43" data-cited="true" movetopanel="yes" data-doi="10.1093/nar/gks540" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22684630"><span class="RefAuthor"><span class="RefSurName">Lohse</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Bolger</span> <span class="RefGivenName">AM</span></span>, <span class="RefAuthor"><span class="RefSurName">Nagel</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Fernie</span> <span class="RefGivenName">AR</span></span>, <span class="RefAuthor"><span class="RefSurName">Lunn</span> <span class="RefGivenName">JE</span></span>, <span class="RefAuthor"><span class="RefSurName">Stitt</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Usadel</span> <span class="RefGivenName">B</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics</span>. <span class="RefJournalTitle"><em>Nucleic Acids Research</em></span> <span class="RefVolume"><strong>40</strong></span>:<span class="RefFPage">W622</span>&ndash;<span class="RefLPage">627</span>.</p><p class="jrnlRefText noPMID" seqid="44" id="R44" data-seq-id="R44" sortby="Mahmud 2013" data-old-id="R44" data-cited="true" movetopanel="yes" data-doi="10.4161/jkst.23154" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24058794"><span class="RefAuthor"><span class="RefSurName">Mahmud</span> <span class="RefGivenName">SA</span></span>, <span class="RefAuthor"><span class="RefSurName">Manlove</span> <span class="RefGivenName">LS</span></span>, <span class="RefAuthor"><span class="RefSurName">Farrar</span> <span class="RefGivenName">MA</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">Interleukin-2 and STAT5 in regulatory T cell development and function</span>. <span class="RefJournalTitle"><em>JAK-STAT</em></span> <span class="RefVolume"><strong>2</strong></span>:<span class="RefFPage">e23154</span>.</p><p class="jrnlRefText noPMID" seqid="45" id="R45" data-seq-id="R45" sortby="Malek 2010" data-old-id="R45" data-cited="true" movetopanel="yes" data-doi="10.1016/j.immuni.2010.08.004" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/20732639"><span class="RefAuthor"><span class="RefSurName">Malek</span><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:48 pm" data-cid="1462267128002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267128000"> </span><span><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:48 pm" data-cid="1462267128003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267128000">TR</span></span></span>, <span class="RefAuthor"><span class="RefSurName">Castro</span> <span class="RefGivenName">I</span></span> <span class="RefAuthor" compared="true"><span class="RefSurName">Thomas</span> <span class="RefGivenName">R</span></span>. <span class="RefYear">2010</span>. <span class="RefArticleTitle">Interleukin-2 receptor signaling: <span data-diff-rid="R101" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:48 pm" data-cid="1462267128004" data-userid="cross-ref" data-username="cross-ref" data-time="1462267128000">at</span><span data-diff-rid="R101" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:48 pm" data-cid="1462267128005" data-userid="cross-ref" data-username="cross-ref" data-time="1462267128000">At</span> the interface between tolerance and immunity</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>33</strong></span>:<span class="RefFPage">153</span>&ndash;<span class="RefLPage">165</span>.</p><p class="jrnlRefText noPMID" seqid="46" id="R46" data-seq-id="R46" sortby="Meinke 1996" data-old-id="R46" data-cited="true" movetopanel="yes" data-doi="10.1128/MCB.16.12.6937" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/8943349"><span class="RefAuthor"><span class="RefSurName">Meinke</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Barahmand-Pour</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">W&ouml;hrl</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Stoiber</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Decker</span> <span class="RefGivenName">T</span></span>. <span class="RefYear">1996</span>. <span class="RefArticleTitle">Activation of different <span data-diff-rid="R121" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:49 pm" data-cid="1462267180002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267180000">Stat5</span><span data-diff-rid="R121" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:49 pm" data-cid="1462267180003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267180000">stat5</span> isoforms contributes to cell-type-restricted signaling in response to interferons</span>. <span class="RefJournalTitle"><em>Molecular and Cellular Biology</em></span> <span class="RefVolume"><strong>16</strong></span>:<span class="RefFPage">6937</span>&ndash;<span class="RefLPage">6944</span>.</p><p class="jrnlRefText" seqid="48" id="R47" data-seq-id="R47" sortby="Metser 2016" data-old-id="R47" data-cited="true" data-doi="10.1093/nar/gkv999" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/26446995" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Metser</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Shin</span> <span class="RefGivenName">HY</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Yoo</span> <span class="RefGivenName">KH</span></span>, <span class="RefAuthor"><span class="RefSurName">Oh</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Villarino</span> <span class="RefGivenName">AV</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Kang</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">2016</span>. <span class="RefArticleTitle">An autoregulatory enhancer controls mammary-specific STAT5 functions</span>. <span class="RefJournalTitle"><em>Nucleic Acids Research</em></span> <span class="RefVolume"><strong>44</strong></span>:<span class="RefFPage">1052</span>&ndash;<span class="RefLPage">1063</span>.</p><p class="jrnlRefText noPMID" seqid="49" id="R48" data-seq-id="R48" sortby="Moriggl 1999" data-old-id="R48" data-cited="true" movetopanel="yes" data-doi="10.1016/S1074-7613(00)80025-4" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10072077"><span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Topham</span> <span class="RefGivenName">DJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Teglund</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Sexl</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">McKay</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Hoffmeyer</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">van Deursen</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Sangster</span> <span class="RefGivenName">MY</span></span>, <span class="RefAuthor"><span class="RefSurName">Bunting</span> <span class="RefGivenName">KD</span></span>, <span class="RefAuthor"><span class="RefSurName">Grosveld</span> <span class="RefGivenName">GC</span></span>, <span class="RefAuthor"><span class="RefSurName">Ihle</span> <span class="RefGivenName">JN</span></span>. <span class="RefYear">1999</span>. <span class="RefArticleTitle">Stat5 is required for <span data-diff-rid="R191" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:49 pm" data-cid="1462267197002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267197000">IL</span><span data-diff-rid="R191" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:49 pm" data-cid="1462267197003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267197000">il</span>-2-induced cell cycle progression of peripheral T cells</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>10</strong></span>:<span class="RefFPage">249</span>&ndash;<span class="RefLPage">259</span>.</p><p class="jrnlRefText noPMID" seqid="50" id="R49" data-seq-id="R49" sortby="Moriggl 1999" data-old-id="R49" movetopanel="yes" data-doi="10.1016/S1074-7613(00)80097-7" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10485657"><span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Sexl</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Piekorz</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Topham</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Ihle</span> <span class="RefGivenName">JN</span></span>. <span class="RefYear">1999</span>. <span class="RefArticleTitle">Stat5 activation is uniquely associated with cytokine signaling in peripheral T cells</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>11</strong></span>:<span class="RefFPage">225</span>&ndash;<span class="RefLPage">230</span>.</p><p class="jrnlRefText" seqid="51" id="R50" data-seq-id="R50" sortby="Nivarthi 2014" data-old-id="R50" data-cited="true" data-doi="10.1111/all.12535" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25333229" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Nivarthi</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Prchal-Murphy</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Swoboda</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Hager</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Schlederer</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Kenner</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Tuckermann</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Sexl</span> <span class="RefGivenName">V</span></span>, <span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Ermakova</span> <span class="RefGivenName">O</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">Stat5 gene dosage in T cells modulates CD8+ T-cell homeostasis and attenuates contact hypersensitivity response in mice</span>. <span class="RefJournalTitle"><em>Allergy</em></span> <span class="RefVolume"><strong>70</strong></span>.</p><p class="jrnlRefText" seqid="52" id="R51" data-seq-id="R51" sortby="Oestreich 2012" data-old-id="R51" data-cited="true" data-doi="10.1038/ni.2242" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22406686" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Oestreich</span> <span class="RefGivenName">KJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Mohn</span> <span class="RefGivenName">SE</span></span>, <span class="RefAuthor"><span class="RefSurName">Weinmann</span> <span class="RefGivenName">AS</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Molecular mechanisms that control the expression and activity of Bcl-6 in TH1 cells to regulate flexibility with a TFH-like gene profile</span>. <span class="RefJournalTitle"><em>Nature Immunology</em></span> <span class="RefVolume"><strong>13</strong></span>:<span class="RefFPage">405</span>&ndash;<span class="RefLPage">411</span>.</p><p class="jrnlRefText" seqid="53" id="R52" data-seq-id="R52" sortby="Pandiyan 2007" data-old-id="R52" data-cited="true" data-doi="10.1038/ni1536" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17982458" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Pandiyan</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Zheng</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Ishihara</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Reed</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Lenardo</span> <span class="RefGivenName">MJ</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">CD4+CD25+Foxp3+ regulatory T cells induce cytokine deprivation-mediated apoptosis of effector CD4+ T cells</span>. <span class="RefJournalTitle"><em>Nature Immunology</em></span> <span class="RefVolume"><strong>8</strong></span>:<span class="RefFPage">1353</span>&ndash;<span class="RefLPage">1362</span>.</p><p class="jrnlRefText" seqid="54" id="R53" data-seq-id="R53" sortby="Ramirez 2014" data-old-id="R53" data-cited="true" movetopanel="yes" data-doi="10.1093/nar/gku365" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24799436"><span class="RefAuthor"><span class="RefSurName">Ram&iacute;rez</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">D&uuml;ndar</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Diehl</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Gr&uuml;ning</span> <span class="RefGivenName">BA</span></span>, <span class="RefAuthor"><span class="RefSurName">Manke</span> <span class="RefGivenName">T</span></span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">deepTools: a flexible platform for exploring deep-sequencing data</span>. <span class="RefJournalTitle"><em>Nucleic Acids Research</em></span> <span class="RefVolume"><strong>42</strong></span>:<span class="RefFPage">W187</span>&ndash;<span class="RefLPage">191</span>.</p><p class="jrnlRefText" seqid="55" id="R54" data-seq-id="R54" sortby="Robinson 2009" data-old-id="R54" data-cited="true" movetopanel="yes" data-doi="10.1093/bioinformatics/btp616" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19910308"><span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">MD</span></span>, <span class="RefAuthor"><span class="RefSurName">McCarthy</span> <span class="RefGivenName">DJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Smyth</span> <span class="RefGivenName">GK</span></span>. <span class="RefYear">2010</span>. <span class="RefArticleTitle">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</span>. <span class="RefJournalTitle"><em>Bioinformatics</em></span> <span class="RefVolume"><strong>26</strong></span>:<span class="RefFPage">139</span>&ndash;<span class="RefLPage">140</span>.</p><p class="jrnlRefText" seqid="56" id="R55" data-seq-id="R55" sortby="Rochman 2009" data-old-id="R55" data-cited="true" movetopanel="yes" data-doi="10.1038/nri2580" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/19543225"><span class="RefAuthor"><span class="RefSurName">Rochman</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Spolski</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2009</span>. <span class="RefArticleTitle">New insights into the regulation of T cells by gamma(c) family cytokines</span>. <span class="RefJournalTitle"><em>Nature Reviews. Immunology</em></span> <span class="RefVolume"><strong>9</strong></span>:<span class="RefFPage">480</span>&ndash;<span class="RefLPage">490</span>.</p><p class="jrnlRefText noPMID" seqid="57" id="R56" data-seq-id="R56" sortby="Rosen 1996" data-old-id="R56" data-cited="true" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/8695838" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Rosen</span> <span class="RefGivenName">RL</span></span>, <span class="RefAuthor"><span class="RefSurName">Winestock</span> <span class="RefGivenName">KD</span></span>, <span class="RefAuthor"><span class="RefSurName">Chen</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Finbloom</span> <span class="RefGivenName">DS</span></span>. <span class="RefYear">1996</span>. <span class="RefArticleTitle">Granulocyte-macrophage colony-stimulating factor preferentially activates the 94-kD STAT5A and an 80-kD STAT5A isoform in human peripheral blood monocytes</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>88</strong></span>:<span class="RefFPage">1206</span>&ndash;<span class="RefLPage">1214</span>.</p><p class="jrnlRefText noPMID" seqid="58" id="R57" data-seq-id="R57" sortby="Snow 2003" data-old-id="R57" data-cited="true" movetopanel="yes" data-doi="10.4049/jimmunol.171.10.5042"><span class="RefAuthor"><span class="RefSurName">Snow</span> <span class="RefGivenName">JW</span></span>, <span class="RefAuthor"><span class="RefSurName">Abraham</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Ma</span> <span class="RefGivenName">MC</span></span>, <span class="RefAuthor"><span class="RefSurName">Herndier</span> <span class="RefGivenName">BG</span></span>, <span class="RefAuthor"><span class="RefSurName">Pastuszak</span> <span class="RefGivenName">AW</span></span>, <span class="RefAuthor"><span class="RefSurName">Goldsmith</span> <span class="RefGivenName">MA</span></span>. <span class="RefYear">2003</span>. <span class="RefArticleTitle">Loss of tolerance and autoimmunity affecting multiple organs in <span data-diff-rid="R131" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:51 pm" data-cid="1462267278002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267278000">STAT5A</span><span data-diff-rid="R131" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:51 pm" data-cid="1462267278003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267278000">stat5a</span>/<span data-diff-rid="R132" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:51 pm" data-cid="1462267278004" data-userid="cross-ref" data-username="cross-ref" data-time="1462267278000">5B</span><span data-diff-rid="R132" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:51 pm" data-cid="1462267278005" data-userid="cross-ref" data-username="cross-ref" data-time="1462267278000">5b</span>-deficient mice</span>. <span class="RefJournalTitle"><em><span class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:51 pm" data-cid="1462267278006" data-userid="cross-ref" data-username="cross-ref" data-time="1462267278000">The </span>Journal of Immunology</em></span> <span class="RefVolume"><strong>171</strong></span>:<span class="RefFPage">5042</span>&ndash;<span class="RefLPage">5050</span>.</p><p class="jrnlRefText noPMID" seqid="59" id="R58" data-seq-id="R58" sortby="Socolovsky 1999" data-old-id="R58" data-cited="true" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10428030" movetopanel="yes" data-doi="10.1016/S0092-8674(00)81013-2"><span class="RefAuthor"><span class="RefSurName">Socolovsky</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Fallon</span> <span class="RefGivenName">AE</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Brugnara</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Lodish</span> <span class="RefGivenName">HF</span></span>. <span class="RefYear">1999</span>. <span class="RefArticleTitle">Fetal anemia and apoptosis of red cell progenitors in Stat5a-/-5b-/- mice: a direct role for Stat5 in Bcl-X(L) induction</span>. <span class="RefJournalTitle"><em>Cell</em></span> <span class="RefVolume"><strong>98</strong></span>:<span class="RefFPage">181</span>&ndash;<span class="RefLPage">191</span> .</p><p class="jrnlRefText noPMID" seqid="60" id="R59" data-seq-id="R59" sortby="Soldaini 1999" data-old-id="R59" data-cited="true" movetopanel="yes" data-doi="10.1128/MCB.20.1.389-401.2000" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10594041"><span class="RefAuthor"><span class="RefSurName">Soldaini</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">John</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Moro</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Bollenbacher</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Schindler</span> <span class="RefGivenName">U</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">DNA binding site selection of dimeric and tetrameric <span data-diff-rid="R131" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267395002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267395000">Stat5</span><span data-diff-rid="R131" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267395003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267395000">stat5</span> proteins reveals a large repertoire of divergent tetrameric <span data-diff-rid="R132" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267395004" data-userid="cross-ref" data-username="cross-ref" data-time="1462267395000">Stat5a</span><span data-diff-rid="R132" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267395005" data-userid="cross-ref" data-username="cross-ref" data-time="1462267395000">stat5a</span> binding sites</span>. <span class="RefJournalTitle"><em>Molecular and Cellular Biology</em></span> <span class="RefVolume"><strong>20</strong></span>:<span class="RefFPage">389</span>&ndash;<span class="RefLPage">401</span>.</p><p class="jrnlRefText" seqid="61" id="R60" data-seq-id="R60" sortby="Stark 2012" data-old-id="R60" data-cited="true" data-doi="10.1016/j.immuni.2012.03.013" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22520844" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Stark</span> <span class="RefGivenName">GR</span></span>, <span class="RefAuthor"><span class="RefSurName">Darnell</span> <span class="RefGivenName">JE</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">The JAK-STAT pathway at twenty</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>36</strong></span>:<span class="RefFPage">503</span>&ndash;<span class="RefLPage">514</span>.</p><p class="jrnlRefText" seqid="62" id="R61" data-seq-id="R61" sortby="Storey 2003" data-old-id="R61" data-cited="true" movetopanel="yes" data-doi="10.1901/jaba.2003.31-512" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18633452"><span class="RefAuthor"><span class="RefSurName">Van der Laan</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Jewell</span> <span class="RefGivenName">NP</span></span>. <span class="RefYear">2003</span>. <span class="RefArticleTitle">Current&nbsp;status and right-censored data structures when observing a marker at the censoring time</span>. <span class="RefJournalTitle"><em>Annals of Statistics</em></span> <span class="RefVolume"><strong>31</strong></span>.</p><p class="jrnlRefText" seqid="63" id="R62" data-seq-id="R62" sortby="Subramanian 2005" data-old-id="R62" data-cited="true" data-doi="10.1073/pnas.0506580102" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/16199517" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Subramanian</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Tamayo</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Mootha</span> <span class="RefGivenName">VK</span></span>, <span class="RefAuthor"><span class="RefSurName">Mukherjee</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Ebert</span> <span class="RefGivenName">BL</span></span>, <span class="RefAuthor"><span class="RefSurName">Gillette</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Paulovich</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Pomeroy</span> <span class="RefGivenName">SL</span></span>, <span class="RefAuthor"><span class="RefSurName">Golub</span> <span class="RefGivenName">TR</span></span>, <span class="RefAuthor"><span class="RefSurName">Lander</span> <span class="RefGivenName">ES</span></span>, <span class="RefAuthor"><span class="RefSurName">Mesirov</span> <span class="RefGivenName">JP</span></span>. <span class="RefYear">2005</span>. <span class="RefArticleTitle">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>102</strong></span>:<span class="RefFPage">15545</span>&ndash;<span class="RefLPage">15550</span>.</p><p class="jrnlRefText noPMID" seqid="64" id="R63" data-seq-id="R63" sortby="Teglund 1998" data-old-id="R63" data-cited="true" movetopanel="yes" data-doi="10.1016/S0092-8674(00)81444-0" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9630227"><span class="RefAuthor"><span class="RefSurName">Teglund</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">McKay</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Schuetz</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">van Deursen</span> <span class="RefGivenName">JM</span></span>, <span class="RefAuthor"><span class="RefSurName">Stravopodis</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Brown</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Bodner</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Grosveld</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Ihle</span> <span class="RefGivenName">JN</span></span>. <span class="RefYear">1998</span>. <span class="RefArticleTitle">Stat5a and <span data-diff-rid="R171" class="del cts-1" title="Deleted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267408002" data-userid="cross-ref" data-username="cross-ref" data-time="1462267408000">Stat5b</span><span data-diff-rid="R171" class="ins cts-1" title="Inserted by cross-ref - 2016/05/03 14:53 pm" data-cid="1462267408003" data-userid="cross-ref" data-username="cross-ref" data-time="1462267408000">stat5b</span> proteins have essential and nonessential, or redundant, roles in cytokine responses</span>. <span class="RefJournalTitle"><em>Cell</em></span> <span class="RefVolume"><strong>93</strong></span>:<span class="RefFPage">841</span>&ndash;<span class="RefLPage">850</span>.</p><p class="jrnlRefText noPMID" seqid="65" id="R64" data-seq-id="R64" sortby="Udy 1997" data-old-id="R64" data-cited="true" movetopanel="yes" data-doi="10.1073/pnas.94.14.7239" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/9207075"><span class="RefAuthor"><span class="RefSurName">Udy</span> <span class="RefGivenName">GB</span></span>, <span class="RefAuthor"><span class="RefSurName">Towers</span> <span class="RefGivenName">RP</span></span>, <span class="RefAuthor"><span class="RefSurName">Snell</span> <span class="RefGivenName">RG</span></span>, <span class="RefAuthor"><span class="RefSurName">Wilkins</span> <span class="RefGivenName">RJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Park</span> <span class="RefGivenName">SH</span></span>, <span class="RefAuthor"><span class="RefSurName">Ram</span> <span class="RefGivenName">PA</span></span>, <span class="RefAuthor"><span class="RefSurName">Waxman</span> <span class="RefGivenName">DJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Davey</span> <span class="RefGivenName">HW</span></span>. <span class="RefYear">1997</span>. <span class="RefArticleTitle">Requirement of STAT5b for sexual dimorphism of body growth rates and liver gene expression</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>94</strong></span>:<span class="RefFPage">7239</span>&ndash;<span class="RefLPage">7244</span>.</p><p class="jrnlRefText" seqid="66" id="R65" data-seq-id="R65" sortby="Villarino 2007" data-old-id="R65" data-cited="true" data-doi="10.1084/jem.20061198" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17227909" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Villarino</span> <span class="RefGivenName">AV</span></span>, <span class="RefAuthor"><span class="RefSurName">Tato</span> <span class="RefGivenName">CM</span></span>, <span class="RefAuthor"><span class="RefSurName">Stumhofer</span> <span class="RefGivenName">JS</span></span>, <span class="RefAuthor"><span class="RefSurName">Yao</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">YK</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Hunter</span> <span class="RefGivenName">CA</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Helper T cell IL-2 production is limited by negative feedback and STAT-dependent cytokine signals</span>. <span class="RefJournalTitle"><em>Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>204</strong></span>:<span class="RefFPage">65</span>&ndash;<span class="RefLPage">71</span>.</p><p class="jrnlRefText noPMID" seqid="67" id="R66" data-seq-id="R66" sortby="Villarino 2015" data-old-id="R66" data-cited="true" movetopanel="yes" data-doi="10.4049/jimmunol.1401867" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25527793"><span class="RefAuthor"><span class="RefSurName">Villarino</span> <span class="RefGivenName">AV</span></span>, <span class="RefAuthor"><span class="RefSurName">Kanno</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Ferdinand</span> <span class="RefGivenName">JR</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>. <span class="RefYear">2015</span>. <span class="RefArticleTitle">Mechanisms of Jak/STAT signaling in Immunity and disease</span>. <span class="RefJournalTitle"><em>Journal of Immunology</em></span> <span class="RefVolume"><strong>194</strong></span>:<span class="RefFPage">21</span>&ndash;<span class="RefLPage">27</span>.</p><p class="jrnlRefText" seqid="68" id="R67" data-seq-id="R67" sortby="Wang 2012" data-old-id="R67" data-cited="true" data-doi="10.4161/jkst.19418" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/24058748" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Levy</span> <span class="RefGivenName">DE</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Comparative evolutionary genomics of the STAT family of transcription factors</span>. <span class="RefJournalTitle"><em>JAK-STAT</em></span> <span class="RefVolume"><strong>1</strong></span>:<span class="RefFPage">23</span>&ndash;<span class="RefLPage">33</span>.</p><p class="jrnlRefText noPMID" seqid="69" id="R68" data-seq-id="R68" sortby="Warden 2013" data-old-id="R68" data-cited="true" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Warden</span> <span class="RefGivenName">CD</span></span>, <span class="RefAuthor"><span class="RefSurName">Yuan</span> <span class="RefGivenName">Y-C</span></span>, <span class="RefAuthor"><span class="RefSurName">Wu</span> <span class="RefGivenName">X</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">Optimal calculation of RNA-Seq fold-change values</span>. <span class="RefJournalTitle"><em>International Journal of Computational Bioinformatics and in Silico Modeling</em></span> <span class="RefVolume"><strong>10</strong></span>.</p><p class="jrnlRefText" seqid="70" id="R69" data-seq-id="R69" sortby="Yamaji 2013" data-old-id="R69" data-cited="true" data-doi="10.1093/nar/gks1310" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/23275557" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Yamaji</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Kang</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">2013</span>. <span class="RefArticleTitle">Sequential activation of genetic programs in mouse mammary epithelium during pregnancy depends on STAT5A/B concentration</span>. <span class="RefJournalTitle"><em>Nucleic Acids Research</em></span> <span class="RefVolume"><strong>41</strong></span>:<span class="RefFPage">1622</span>&ndash;<span class="RefLPage">1636</span>.</p><p class="jrnlRefText" seqid="71" id="R70" data-seq-id="R70" sortby="Yamaoka 2005" data-old-id="R70" data-cited="true" data-doi="10.1182/blood-2005-02-0769" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/16020505" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Yamaoka</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Min</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhou</span> <span class="RefGivenName">YJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Paul</span> <span class="RefGivenName">WE</span></span>, <span class="RefAuthor"><span class="RefSurName">O'shea</span> <span class="RefGivenName">JJ</span></span>. <span class="RefYear">2005</span>. <span class="RefArticleTitle">Jak3 negatively regulates dendritic-cell cytokine production and survival</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>106</strong></span>:<span class="RefFPage">3227</span>&ndash;<span class="RefLPage">3233</span>.</p><p class="jrnlRefText" seqid="73" id="R71" data-seq-id="R72" sortby="Yao 2006" data-old-id="R72" data-cited="true" data-doi="10.1073/pnas.0507350103" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/16418296" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Yao</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Cui</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Watford</span> <span class="RefGivenName">WT</span></span>, <span class="RefAuthor"><span class="RefSurName">Bream</span> <span class="RefGivenName">JH</span></span>, <span class="RefAuthor"><span class="RefSurName">Yamaoka</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Hissong</span> <span class="RefGivenName">BD</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Durum</span> <span class="RefGivenName">SK</span></span>, <span class="RefAuthor"><span class="RefSurName">Jiang</span> <span class="RefGivenName">Q</span></span>, <span class="RefAuthor"><span class="RefSurName">Bhandoola</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>. <span class="RefYear">2006</span>. <span class="RefArticleTitle">Stat5a/b are essential for normal lymphoid development and differentiation</span>. <span class="RefJournalTitle"><em>Proceedings of the National Academy of Sciences of the United States of America</em></span> <span class="RefVolume"><strong>103</strong></span>:<span class="RefFPage">1000</span>&ndash;<span class="RefLPage">1005</span>.</p><p class="jrnlRefText" seqid="72" id="R72" data-seq-id="R71" sortby="Yao 2007" data-old-id="R71" data-cited="true" data-doi="10.1182/blood-2006-11-055756" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17227828" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Yao</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Kanno</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Kerenyi</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Stephens</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Durant</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Watford</span> <span class="RefGivenName">WT</span></span>, <span class="RefAuthor"><span class="RefSurName">Laurence</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Robinson</span> <span class="RefGivenName">GW</span></span>, <span class="RefAuthor"><span class="RefSurName">Shevach</span> <span class="RefGivenName">EM</span></span>, <span class="RefAuthor"><span class="RefSurName">Moriggl</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Wu</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">O'Shea</span> <span class="RefGivenName">JJ</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">Nonredundant roles for Stat5a/b in directly regulating Foxp3</span>. <span class="RefJournalTitle"><em>Blood</em></span> <span class="RefVolume"><strong>109</strong></span>:<span class="RefFPage">4368</span>&ndash;<span class="RefLPage">4375</span>.</p><p class="jrnlRefText noPMID" seqid="74" id="R73" data-seq-id="R73" sortby="Yue 2014" data-old-id="R73" data-cited="true" movetopanel="yes" data-doi="10.1038/nature13992" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/25409824"><span class="RefAuthor"><span class="RefSurName">Yue</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Cheng</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Breschi</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Vierstra</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Wu</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Ryba</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Sandstrom</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Ma</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Davis</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Pope</span> <span class="RefGivenName">BD</span></span>, <span class="RefAuthor"><span class="RefSurName">Shen</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Pervouchine</span> <span class="RefGivenName">DD</span></span>, <span class="RefAuthor"><span class="RefSurName">Djebali</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Thurman</span> <span class="RefGivenName">RE</span></span>, <span class="RefAuthor"><span class="RefSurName">Kaul</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Rynes</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Kirilusha</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Marinov</span> <span class="RefGivenName">GK</span></span>, <span class="RefAuthor"><span class="RefSurName">Williams</span> <span class="RefGivenName">BA</span></span>, <span class="RefAuthor"><span class="RefSurName">Trout</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Amrhein</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor"><span class="RefSurName">Fisher-Aylor</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Antoshechkin</span> <span class="RefGivenName">I</span></span>, <span class="RefAuthor"><span class="RefSurName">DeSalvo</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">See</span> <span class="RefGivenName">LH</span></span>, <span class="RefAuthor"><span class="RefSurName">Fastuca</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Drenkow</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Zaleski</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Dobin</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Prieto</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Lagarde</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Bussotti</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Tanzer</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Denas</span> <span class="RefGivenName">O</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Bender</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Byron</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Groudine</span> <span class="RefGivenName">MT</span></span>, <span class="RefAuthor"><span class="RefSurName">McCleary</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Pham</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Ye</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor"><span class="RefSurName">Kuan</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Edsall</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Wu</span> <span class="RefGivenName">YC</span></span>, <span class="RefAuthor"><span class="RefSurName">Rasmussen</span> <span class="RefGivenName">MD</span></span>, <span class="RefAuthor"><span class="RefSurName">Bansal</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor"><span class="RefSurName">Kellis</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Keller</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Morrissey</span> <span class="RefGivenName">CS</span></span>, <span class="RefAuthor"><span class="RefSurName">Mishra</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Jain</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Dogan</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor"><span class="RefSurName">Harris</span> <span class="RefGivenName">RS</span></span>, <span class="RefAuthor"><span class="RefSurName">Cayting</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Kawli</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Boyle</span> <span class="RefGivenName">AP</span></span>, <span class="RefAuthor"><span class="RefSurName">Euskirchen</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Kundaje</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Lin</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Lin</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Jansen</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Malladi</span> <span class="RefGivenName">VS</span></span>, <span class="RefAuthor"><span class="RefSurName">Cline</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor"><span class="RefSurName">Erickson</span> <span class="RefGivenName">DT</span></span>, <span class="RefAuthor"><span class="RefSurName">Kirkup</span> <span class="RefGivenName">VM</span></span>, <span class="RefAuthor"><span class="RefSurName">Learned</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Sloan</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Rosenbloom</span> <span class="RefGivenName">KR</span></span>, <span class="RefAuthor"><span class="RefSurName">Lacerda de Sousa</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Beal</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Pignatelli</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Flicek</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Lian</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Kahveci</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Kent</span> <span class="RefGivenName">WJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Ramalho Santos</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Herrero</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Notredame</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Johnson</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Vong</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Bates</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Neri</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor"><span class="RefSurName">Diegel</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Canfield</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Sabo</span> <span class="RefGivenName">PJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Wilken</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor"><span class="RefSurName">Reh</span> <span class="RefGivenName">TA</span></span>, <span class="RefAuthor"><span class="RefSurName">Giste</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Shafer</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Kutyavin</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Haugen</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor"><span class="RefSurName">Dunn</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Reynolds</span> <span class="RefGivenName">AP</span></span>, <span class="RefAuthor"><span class="RefSurName">Neph</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Humbert</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Hansen</span> <span class="RefGivenName">RS</span></span>, <span class="RefAuthor"><span class="RefSurName">De Bruijn</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Selleri</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Rudensky</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Josefowicz</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Samstein</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Eichler</span> <span class="RefGivenName">EE</span></span>, <span class="RefAuthor"><span class="RefSurName">Orkin</span> <span class="RefGivenName">SH</span></span>, <span class="RefAuthor"><span class="RefSurName">Levasseur</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Papayannopoulou</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Chang</span> <span class="RefGivenName">KH</span></span>, <span class="RefAuthor"><span class="RefSurName">Skoultchi</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Gosh</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Disteche</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Treuting</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Weiss</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Blobel</span> <span class="RefGivenName">GA</span></span>, <span class="RefAuthor"><span class="RefSurName">Cao</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhong</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Good</span> <span class="RefGivenName">PJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Lowdon</span> <span class="RefGivenName">RF</span></span>, <span class="RefAuthor"><span class="RefSurName">Adams</span> <span class="RefGivenName">LB</span></span>, <span class="RefAuthor"><span class="RefSurName">Zhou</span> <span class="RefGivenName">XQ</span></span>, <span class="RefAuthor"><span class="RefSurName">Pazin</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Feingold</span> <span class="RefGivenName">EA</span></span>, <span class="RefAuthor"><span class="RefSurName">Wold</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor"><span class="RefSurName">Taylor</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Mortazavi</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor"><span class="RefSurName">Weissman</span> <span class="RefGivenName">SM</span></span>, <span class="RefAuthor"><span class="RefSurName">Stamatoyannopoulos</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor"><span class="RefSurName">Snyder</span> <span class="RefGivenName">MP</span></span>, <span class="RefAuthor"><span class="RefSurName">Guigo</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Gingeras</span> <span class="RefGivenName">TR</span></span>, <span class="RefAuthor"><span class="RefSurName">Gilbert</span> <span class="RefGivenName">DM</span></span>, <span class="RefAuthor"><span class="RefSurName">Hardison</span> <span class="RefGivenName">RC</span></span>, <span class="RefAuthor"><span class="RefSurName">Beer</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor"><span class="RefSurName">Ren</span> <span class="RefGivenName">B</span></span>.<span class="RefCollaboration">Mouse ENCODE Consortium</span>. <span class="RefYear">2014</span>. <span class="RefArticleTitle">A comparative encyclopedia of DNA elements in the mouse genome</span>. <span class="RefJournalTitle"><em>Nature</em></span> <span class="RefVolume"><strong>515</strong></span>:<span class="RefFPage">355</span>&ndash;<span class="RefLPage">364</span>.</p><p class="jrnlRefText noPMID" seqid="76" id="R74" data-seq-id="R75" sortby="Zhang 2000" data-old-id="R75" data-cited="true" movetopanel="yes" data-doi="10.1084/jem.192.5.719" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/10974037"><span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Fukuda</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Hangoc</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor"><span class="RefSurName">Cooper</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor"><span class="RefSurName">Spolski</span> <span class="RefGivenName">R</span></span>, <span class="RefAuthor"><span class="RefSurName">Leonard</span> <span class="RefGivenName">WJ</span></span>, <span class="RefAuthor"><span class="RefSurName">Broxmeyer</span> <span class="RefGivenName">HE</span></span>. <span class="RefYear">2000</span>. <span class="RefArticleTitle">Essential role of signal transducer and activator of transcription (Stat)5a but not Stat5b for Flt3-dependent signaling</span>. <span class="RefJournalTitle"><em>The Journal of Experimental Medicine</em></span> <span class="RefVolume"><strong>192</strong></span>:<span class="RefFPage">719</span>&ndash;<span class="RefLPage">728</span>.</p><p class="jrnlRefText" seqid="75" id="R75" data-seq-id="R74" sortby="Zhang 2007" data-old-id="R74" data-cited="true" data-doi="10.1038/nm1659" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/17922009" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">Q</span></span>, <span class="RefAuthor"><span class="RefSurName">Wang</span> <span class="RefGivenName">HY</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">X</span></span>, <span class="RefAuthor"><span class="RefSurName">Wasik</span> <span class="RefGivenName">MA</span></span>. <span class="RefYear">2007</span>. <span class="RefArticleTitle">STAT5A is epigenetically silenced by the tyrosine kinase NPM1-ALK and acts as a tumor suppressor by reciprocally inhibiting NPM1-ALK expression</span>. <span class="RefJournalTitle"><em>Nature Medicine</em></span> <span class="RefVolume"><strong>13</strong></span>:<span class="RefFPage">1341</span>&ndash;<span class="RefLPage">1348</span>.</p><p class="jrnlRefText" seqid="78" id="R76" data-seq-id="R77" sortby="Zhang 2008" data-old-id="R77" data-cited="true" data-doi="10.1186/gb-2008-9-9-r137" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/18798982" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor"><span class="RefSurName">Meyer</span> <span class="RefGivenName">CA</span></span>, <span class="RefAuthor"><span class="RefSurName">Eeckhoute</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Johnson</span> <span class="RefGivenName">DS</span></span>, <span class="RefAuthor"><span class="RefSurName">Bernstein</span> <span class="RefGivenName">BE</span></span>, <span class="RefAuthor"><span class="RefSurName">Nusbaum</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor"><span class="RefSurName">Myers</span> <span class="RefGivenName">RM</span></span>, <span class="RefAuthor"><span class="RefSurName">Brown</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor"><span class="RefSurName">Li</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Liu</span> <span class="RefGivenName">XS</span></span>. <span class="RefYear">2008</span>. <span class="RefArticleTitle">Model-based analysis of ChIP-Seq (MACS)</span>. <span class="RefJournalTitle"><em>Genome Biology</em></span> <span class="RefVolume"><strong>9</strong></span>:<span class="RefFPage">R137</span>.</p><p class="jrnlRefText" seqid="77" id="R77" data-seq-id="R76" sortby="Zhang 2012" data-old-id="R76" data-cited="true" data-doi="10.1128/MCB.06312-11" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22158971" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Zhang</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"><span class="RefSurName">Laz</span> <span class="RefGivenName">EV</span></span>, <span class="RefAuthor"><span class="RefSurName">Waxman</span> <span class="RefGivenName">DJ</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Dynamic, sex-differential STAT5 and BCL6 binding to sex-biased, growth hormone-regulated genes in adult mouse liver</span>. <span class="RefJournalTitle"><em>Molecular and Cellular Biology</em></span> <span class="RefVolume"><strong>32</strong></span>:<span class="RefFPage">880</span>&ndash;<span class="RefLPage">896</span>.</p><p class="jrnlRefText noPMID" seqid="80" id="R78" data-seq-id="R79" sortby="Zhu 2003" data-old-id="R79" data-cited="true" movetopanel="yes" data-doi="10.1016/S1074-7613(03)00292-9" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/14614860"><span class="RefAuthor"><span class="RefSurName">Zhu</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Cote-Sierra</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor"><span class="RefSurName">Guo</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Paul</span> <span class="RefGivenName">WE</span></span>. <span class="RefYear">2003</span>. <span class="RefArticleTitle">Stat5 activation plays a critical role in Th2 differentiation</span>. <span class="RefJournalTitle"><em>Immunity</em></span> <span class="RefVolume"><strong>19</strong></span>:<span class="RefFPage">739</span>&ndash;<span class="RefLPage">748</span>.</p><p class="jrnlRefText" seqid="79" id="R79" data-seq-id="R78" sortby="Zhu 2012" data-old-id="R78" data-cited="true" data-doi="10.1093/nar/gks056" data-pmid="http://www.ncbi.nlm.nih.gov/pubmed/22319210" movetopanel="yes"><span class="RefAuthor"><span class="RefSurName">Zhu</span> <span class="RefGivenName">BM</span></span>, <span class="RefAuthor"><span class="RefSurName">Kang</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor"><span class="RefSurName">Yu</span> <span class="RefGivenName">JH</span></span>, <span class="RefAuthor"><span class="RefSurName">Chen</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor"><span class="RefSurName">Smith</span> <span class="RefGivenName">HE</span></span>, <span class="RefAuthor"><span class="RefSurName">Lee</span> <span class="RefGivenName">D</span></span>, <span class="RefAuthor"><span class="RefSurName">Sun</span> <span class="RefGivenName">HW</span></span>, <span class="RefAuthor"><span class="RefSurName">Wei</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor"><span class="RefSurName">Hennighausen</span> <span class="RefGivenName">L</span></span>. <span class="RefYear">2012</span>. <span class="RefArticleTitle">Genome-wide analyses reveal the extent of opportunistic STAT5 binding that does not yield transcriptional activation of neighboring genes</span>. <span class="RefJournalTitle"><em>Nucleic Acids Research</em></span> <span class="RefVolume"><strong>40</strong></span>:<span class="RefFPage">4461</span>&ndash;<span class="RefLPage">4472</span>.</p><p bold="true" class="jrnlDeleted"><span class="del" data-userid="6662" data-username="Vinoth (eLife Sciences Publications, Ltd)">Supplementary File Legends</span>
</p><div class="jrnlSupplBlock" id="BLK_SP1" data-stream-name="a_SP1"><p class="jrnlSupplCaption" id="SP1"><span class="label">Supplementary file 1.</span> <i fontsize="11.0pt">Read depth for sequencing experiments.</i>
</p><p class="jrnlSupplCaption">Spreadsheet reports QC-passing reads for all RNA-seq and ChIP-seq experiments.</p><p><a download="Supplemental_file_1" href="/ecs/article/80712/resources/Supplemental_file_1.xlsx" data-href="/ecs/article/80712/resources/Supplemental_file_1.xlsx" data-mce-href="ecs/article/80712/resources/Supplemental_file_1.xlsx">Supplemental_file_1</a>
</p></div><div class="jrnlSupplBlock" id="BLK_SP2" data-stream-name="a_SP2"><p class="jrnlSupplCaption" id="SP2"><span class="label">Supplementary file 2.</span> <i fontsize="11.0pt">RPKM values for transcriptome analysis of STAT5-deficient CD4<sup>+</sup> T cells.</i>
</p><p class="jrnlSupplCaption">Spreadsheet includes RPKM values for all experimental groups and biological replicates.</p><p><a download="Supplemental_file_2" href="/ecs/article/80712/resources/Supplemental_file_2.xlsx" data-href="/ecs/article/80712/resources/Supplemental_file_2.xlsx" data-mce-href="ecs/article/80712/resources/Supplemental_file_2.xlsx">Supplemental_file_2</a>
</p></div><div class="jrnlSupplBlock" id="BLK_SP3" data-stream-name="a_SP3"><p class="jrnlSupplCaption" id="SP3"><span class="label">Supplementary file 3.</span> <i fontsize="11.0pt">Paralog preference calculations for STAT5-deficient CD4<sup>+</sup> T cells.</i>
</p><p class="jrnlSupplCaption">Spreadsheet includes paralog preference calculations for all relevant cell types and experimental conditions.</p><p><a download="Supplemental_file_3" href="/ecs/article/80712/resources/Supplemental_file_3.xlsx" data-href="/ecs/article/80712/resources/Supplemental_file_3.xlsx" data-mce-href="ecs/article/80712/resources/Supplemental_file_3.xlsx">Supplemental_file_3</a>
</p></div><div class="jrnlSupplBlock" id="BLK_SP4" data-stream-name="a_SP4"><p class="jrnlSupplCaption" id="SP4"><span class="label">Supplementary file 4.</span> <i fontsize="11.0pt">STAT5 paralog-specific genes.</i>
</p><p class="jrnlSupplCaption">Spreadsheet lists STAT5 paralog-specific genes for all relevant cell types and experimental conditions.</p><p><a download="Supplemental_file_4" href="/ecs/article/80712/resources/Supplemental_file_4.xlsx" data-href="/ecs/article/80712/resources/Supplemental_file_4.xlsx" data-mce-href="ecs/article/80712/resources/Supplemental_file_4.xlsx">Supplemental_file_4</a>
</p></div><div class="sub-article" data-editable="true" article-type="article-commentary" id="sa1"><h1 class="sub-article-title">Decision letter</h1><div class="sub-body"><div class="jrnlBoxBlock"><p class="jrnlBoxText">In the interests of transparency, eLife includes the editorial decision letter and accompanying author responses. A lightly edited version of the letter sent to the authors after peer review is shown, indicating the most substantive concerns; minor comments are not usually included.</p></div><p>Thank you for submitting your work entitled "STAT5 paralog dose governs T cell effector and regulatory function" for peer review at <i>eLife</i>. Your submission has been favorably evaluated by Tadatsugu Taniguchi (Senior editor), and three reviewers, one of whom is a member of our Board of Reviewing Editors.</p><p>The reviewers have discussed the reviews with one another and the Reviewing editor has drafted this decision to help you prepare a revised submission.</p><p>Summary:</p><p>Villarino et al. address specific roles of the STAT5 paralogs on regulatory and helper/effector T cell function, particularly with respect to production of members of cytokine family that signals through receptors bearing common &gamma; chain. Largely through genomic analysis of a range of allelic combinations of <i>Stat5a</i> and <i>Stat5b</i> mutant mice via disruption of either single or dual alleles (combinations of homozygous or heterozygous <i>Stat5a</i> and <i>Stat5b</i> mutant mice), the authors find that the two paralogs are highly redundant in their functions, but because STAT5B is expressed at higher levels, it has a dominant role in regulating immune cell development and function. The studies are generally well-performed and address an issue that has not been clearly resolved for immune cells.</p><p>Essential revisions:</p><p>1) Although the authors have used state-of-art genomic analysis to arrive at the conclusion that a dose-dependent effect of the STAT5 paralog is critical to drive the STAT5 inducible genes optimally, most of their conclusions are correlative rather serving as a direct biological 'cause-effect' relation. Additional studies would be needed to provide direct biological evidence to validate their claim of dosage effect of paralog on T cell-mediated immune response. The studies shown in <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9</span> should be extended to demonstrate that the aberrant T cell responses exhibited by STAT5B-deficieny (e.g., increased frequency of CD44high effector populations, lower Tbet induction in response to IL-2, lower extent of IL-2Ra and FoxP3 induction during Treg development and greater loss of suppressive ability of Tregs) can be reversed by retroviral expression of STAT5A in STAT5B-deficient cells. This could be achieved by transducing naive T cells ex vivo and then analyzing their ability to differentiate into distinct T cell lineages.</p><p>2) This paper involves comparisons between 8 different mouse genotypes yet according to the Methods section these were all compared using single t-tests which is inappropriate. The analysis of data in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1B-E</span>, <span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2A-B</span>, Figure 2&mdash;figure supplement 1, <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B, D, F</span>, <span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4B</span>, <span class="jrnlFigRef" data-citation-string=" F4-S1 ">Figure 4&mdash;figure supplement 1</span>, <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5B</span>, <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6E</span>, and <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9D</span> should be performed using ANOVA or some other comparable test to account for multiple sample comparison.</p><p>3) Replicates for ChIP-seq experiments should be performed if not done so already. Without replicates a determination of FDR (or IDR using Encode methods for combining replicates) cannot be performed. This is important given that the question is whether or not STAT5A and STAT5B bind differentially to any regions of the genome. A Table summarizing read depth is needed to allow assessment of data quality. An anonymous link to a browser session is recommended. The RNA-Seq and ChIP-seq data needs to be deposited in a public repository.</p><p>4) Because of the global nature of the <i>Stat5</i> knockouts, cell types in addition to T cells could contribute autoimmune kidney pathology, particularly macrophages. These experiments would be strengthened by blocking cytokine signaling pathways of T cells that are dysregulated in the absence of STAT5B. At the least, the potential contributions of additional cell types to the renal pathology needs to be discussed.</p><p>5) A major point of the manuscript is that STAT5B is dominant because it is expressed at higher levels. This point is not adequately documented. The only figure panel that directly shows differential expression at the level of protein is <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9C</span>, which uses a pan-STAT Ab to visualize STAT5 in WT and 5A or 5B KO cells. The labeling of this figure is ambiguous, but a large decrease in expression is observed for 5A (5A-deficient) cells and no expression is seen in 5B (5B-deficient) cells. These differences seem to be much greater than the ~2-fold differences in RNA expression shown in <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span> for Treg and Nv cells and RNA expression is nearly equal in Tef cells. The authors show evidence for cell-specific enhancers in <span class="jrnlFigRef" data-citation-string=" F9-S2 ">Figure 9&mdash;figure supplement 2</span>, which could explain differences in STAT5 expression across cell types, but these do not necessarily explain differences in STAT5A vs. STAT5B expression within a particular cell type, raising the possibility that differences in post transcriptional regulation of STAT5A/B expression are important. Further documentation of different levels of STAT5B and STAT5A protein expression is needed in wild type cells. Any further insights into the basis of differential expression would also strengthen the manuscript.</p></div></div><div class="sub-article" data-editable="true" article-type="reply" id="sa2"><h1 class="sub-article-title">Author response</h1><div class="sub-body"><p><i>Essential revisions:</i>
</p><p><i><br/> 1) Although the authors have used state-of-art genomic analysis to arrive at the conclusion that a dose-dependent effect of the STAT5 paralog is critical to drive the STAT5 inducible genes optimally, most of their conclusions are correlative rather serving as a direct biological 'cause-effect' relation. Additional studies would be needed to provide direct biological evidence to validate their claim of dosage effect of paralog on T cell-mediated immune response. The studies shown in <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9</span> should be extended to demonstrate that the aberrant T cell responses exhibited by STAT5B-deficieny (e.g., increased frequency of CD44high effector populations, lower Tbet induction in response to IL-2, lower extent of IL-2Ra and FoxP3 induction during Treg development and greater loss of suppressive ability of Tregs) can be reversed by retroviral expression of STAT5A in STAT5B-deficient cells. This could be achieved by transducing naive T cells ex vivo and then analyzing their ability to differentiate into distinct T cell lineages. <br/><br/></i>
</p><p>A principal finding of our work is that STAT5B is dominant over STAT5A because it is more abundant, not because it is endowed with unique functional properties. This conclusion is supported in the original manuscript by gain-of-function studies showing that ectopic STAT5A can rescue gene expression in <i>Stat5b</i><i>-</i>deficient T cells but, as noted by the reviewers, these did not address whether it can also rescue cellular differentiation. To explore the latter possibility, we transduced <i>Stat5b</i>-deficient T cells under Treg or Th17 polarizing conditions, then measured FoxP3 and IL-17 to gauge lineage commitment. As expected, ectopic STAT5A restored their capacity to become FoxP3<sup>+</sup> Treg cells while, at the same time, limiting their capacity to become IL-17-producing effectors. Ectopic STAT5A even drove FoxP3 expression in WT and heterozygous cells (i.e. <i>Stat5a</i><sup>+/-</sup><i>Stat5b</i><sup>+/-</sup>), suggesting that availability of STAT5 is actually rate limiting for Treg differentiation. Furthermore, there was a clear, linear correlation between STAT5 protein levels, whether endogenous or ectopic, and FoxP3 induction in all genotypes, suggesting that high concentrations predispose towards the Treg lineage. Collectively, these data support our hypothesis that STAT5 paralog dose bears major influence on T cell differentiation and imply that a certain STAT5 threshold must be achieved to execute the Treg program.</p><p>All ectopic STAT5A experiments have been compiled in the new <span class="jrnlFigRef" data-citation-string=" F10 ">Figure 10</span> and <span class="jrnlFigRef" data-citation-string=" F10-S1 ">Figure 10&mdash;figure supplement 1</span>. They are described in the Results section (last two paragraphs).</p><p><i>2) This paper involves comparisons between 8 different mouse genotypes yet according to the Methods section these were all compared using single t-tests which is inappropriate. The analysis of data in <span class="jrnlFigRef" data-citation-string=" F1 ">Figure 1B-E</span>, <span class="jrnlFigRef" data-citation-string=" F2 ">Figure 2A-B</span>, Figure 2&mdash;figure supplement 1, <span class="jrnlFigRef" data-citation-string=" F3 ">Figure 3B, D, F</span>, <span class="jrnlFigRef" data-citation-string=" F4 ">Figure 4B</span>, <span class="jrnlFigRef" data-citation-string=" F4-S1 ">Figure 4&mdash;figure supplement 1</span>, <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5B</span>, <span class="jrnlFigRef" data-citation-string=" F6 ">Figure 6E</span>, and <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9D</span> should be performed using ANOVA or some other comparable test to account for multiple sample comparison. <br/><br/></i>
</p><p>Unpaired ANOVA is now used for all analyses involving three or more experimental groups. This methodology clearly improved the statistical power of our multi-group comparisons &ndash; in some cases revealing differences that were missed by single t-tests &ndash; but did not affect our interpretation of the results.</p><p><i><br/> 3) Replicates for ChIP-seq experiments should be performed if not done so already. Without replicates a determination of FDR (or IDR using Encode methods for combining replicates) cannot be performed. This is important given that the question is whether or not STAT5A and STAT5B bind differentially to any regions of the genome. A Table summarizing read depth is needed to allow assessment of data quality. An anonymous link to a browser session is recommended. The RNA-Seq and ChIP-seq data needs to be deposited in a public repository. <br/><br/></i>
</p><p>Two independent ChIP-seq experiments have been performed and Spearman's rank coefficients indicate good correlation between biological replicates (WT=0.82, <i>Stat5a/b</i><sup>het</sup>=0.83, <i>Stat5a</i>-deficient=0.82, <i>Stat5b</i>-deficient=0.81). Positive false discovery rates, or q-values, were calculated empirically for each peak. Due to stringent peak calling criteria &ndash; only peaks with p values &lt;0.00005 were considered &ndash; all were more than 3-fold enriched over background with FDR values below 0.2%. Each of these points are now clarified in the Methods section (subsection &ldquo;Chromatin immuno-precipitation and DNA sequencing&rdquo;).</p><p>QC-passing read counts for all RNA-seq and ChIP-seq replicates are now provided in <span class="jrnlSupplRef" data-type="ruleset" data-citation-string=" SP1 ">Supplementary file 1</span>.</p><p>Although a dedicated UCSC genome browser session would facilitate access to our data, we are currently unable to allocate the server space necessary to host it in perpetuity. However, the raw data have been deposited to the Gene Expression Omnibus and, thus, will be freely available upon publication (accession number GSE77656; <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=mvyzwwievhcbvgr&amp;acc=GSE77656" data-mce-href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=mvyzwwievhcbvgr&amp;acc=GSE77656">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=mvyzwwievhcbvgr&amp;acc=GSE77656</a>).</p><p><i>4) Because of the global nature of the Stat5 knockouts, cell types in addition to T cells could contribute autoimmune kidney pathology, particularly macrophages. These experiments would be strengthened by blocking cytokine signaling pathways of T cells that are dysregulated in the absence of STAT5B. At the least, the potential contributions of additional cell types to the renal pathology needs to be discussed. <br/><br/></i>
</p><p>Given the importance of helper T cells for maintaining and/or breaking immunological tolerance, and extensive data (including our own) on T cell intrinsic functions of STAT5, we conclude that helper T cell are the principal instigators of autoimmunity in <i>Stat5b</i>-deficient mice. However, as noted by the reviewers, STAT5 is known to impact many other cell types that may influence pathogenesis in these germline mutant animals. This important point is now raised in the Discussion section:</p><p>"These findings strongly implicate Tfh cells in the pathogenesis of Stat5b-deficient mice but, since these are germline &lsquo;knockouts&rsquo;, we must also consider the (likely) possibility that intrinsic defects in other cell types contribute to their autoimmune phenotype. For instance, multiple dendritic cell subsets are known to be dysregulated in mice lacking STAT5 or upstream kinase Jak3, and its influence on non-immune cells, particularly downstream of hormone receptors, cannot be overlooked."</p><p><i>5) A major point of the manuscript is that STAT5B is dominant because it is expressed at higher levels. This point is not adequately documented. The only figure panel that directly shows differential expression at the level of protein is <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9C</span>, which uses a pan-STAT Ab to visualize STAT5 in WT and 5A or 5B KO cells. The labeling of this figure is ambiguous, but a large decrease in expression is observed for 5A (5A-deficient) cells and no expression is seen in 5B (5B-deficient) cells. These differences seem to be much greater than the ~2-fold differences in RNA expression shown in <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span> for Treg and Nv cells and RNA expression is nearly equal in Tef cells. The authors show evidence for cell-specific enhancers in <span class="jrnlFigRef" data-citation-string=" F9-S2 ">Figure 9&mdash;figure supplement 2</span>, which could explain differences in STAT5 expression across cell types, but these do not necessarily explain differences in STAT5A vs. STAT5B expression within a particular cell type, raising the possibility that differences in post transcriptional regulation of STAT5A/B expression are important. Further documentation of different levels of STAT5B and STAT5A protein expression is needed in wild type cells. Any further insights into the basis of differential expression would also strengthen the manuscript.</i>
</p><p>The idea that STAT5B is more abundant than STAT5A is central to our thesis on why it is the dominant paralog in helper T cells. To better support this point, we employed two independent strategies. First, we mined published transcriptome datasets to confirm that STAT5B is more abundant at the mRNA level (<span class="jrnlFigRef" data-citation-string=" F9-S1 ">Figure 9&mdash;figure supplement 1</span>). Second, we refined our pan-STAT5 flow cytometry assay to better illustrate the disparity at the protein level (<span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span>).</p><p>Due to inevitable differences in affinity and/or avidity, we believe it is impossible to compare cellular concentrations of two proteins using two separate antibodies. Therefore, we have chosen to calculate relative abundance based on the performance of a single pan-STAT5 antibody, which presumably recognizes a single homologous epitope, across a range of STAT5 allele combinations. Using this methodology, which has been refined such that we can resolve individual subsets (i.e. naive, Treg, Tfh) across 6 genotypes (or more), we confirmed that STAT5B makes the greater contribution to the total STAT5 pool. The asymmetry was most obvious in cells lacking one allele &ndash; deleting a single copy of <i>Stat5b</i> was enough to yield a phenotype while deleting a single copy <i>Stat5a</i> was not &ndash; but was also evident in those lacking two; <i>Stat5b</i>-deficient cells had far less STAT5 protein than <i>Stat5a</i>-deficient cells (<span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span>). We also found that STAT5 levels were consistently lower in cells from 'one allele' Stat5a-deficient mice (<i>Stat5a</i><sup>-/-</sup><i>Stat5b</i><sup>+/-</sup>) than in those from double heterozygotes (<i>Stat5a</i><sup>+/-</sup><i>Stat5b</i><sup>+/-</sup>), suggesting that, while <i>Stat5b</i> may be the dominant input, the <i>Stat5a</i> locus does make an appreciable, albeit more limited, contribution (<span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span>).</p><p>All STAT5 mRNA and protein measurements have been compiled in the new <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9</span> and <span class="jrnlFigRef" data-citation-string=" F9-S1 ">Figure 9&mdash;figure supplement 1</span>. They are described in the Results section, subsection &ldquo;Asymmetric expression of STAT5 paralogs in helper T cells&rdquo; and in the Methods section, subsection &ldquo;Flow cytometry&rdquo;.</p><p>Overall, there was good accord between our mRNA and protein data in demonstrating that STAT5B is from 1.5 to 3 times more abundant than STAT5A. These ratios are in line with our original flow cytometry studies (Original <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9B</span>) but less dramatic than what we had observed by western blotting (Original <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9C</span>), likely due to differences in sensitivity between the two assays. Given this inconsistency, and the fact that our original flow cytometry studies did not distinguish individual T cell subsets, we have chosen to include only our latest flow cytometry data in the revised manuscript.</p><p>One surprise that emerged from our RNA-seq studies is that paralog asymmetry appeared to be lost after in vitro activation (see 'effector T cells' in the Original <span class="jrnlFigRef" data-citation-string=" F9 ">Figure 9A</span>). Upon inspection, we found that this was evident only when cells were cultured for 72 hours, not when they were cultured for 96 hours (Data not shown) or under iTreg polarizing conditions (data from <span class="jrnlFigRef" data-citation-string=" F5 ">Figure 5</span>). We also found no evidence for such an effect in published datasets (Data not shown), nor could we detect it at the protein level (Data not shown). Thus, while it remains possible that acute stimuli (i.e. TCR engagement, co-stimulation) temporarily alter the balance between STAT5 paralogs, we are unsure about the kinetics and/or robustness of this phenomenon and, thus, have decided to omit all mention of it from the revised manuscript.</p></div></div></div></div><div id="queryDivNode"><p>Queries are deleted.</p></div>